{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "execution_count": 37,
      "metadata": {
        "id": "IALr7__a9XpV"
      },
      "outputs": [],
      "source": [
        "import pandas as pd\n",
        "import os\n",
        "import xml.etree.ElementTree as ET\n",
        "from sklearn.model_selection import train_test_split\n",
        "from sklearn.feature_extraction.text import TfidfVectorizer\n",
        "from sklearn.svm import SVC\n",
        "from sklearn.metrics import accuracy_score, classification_report\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Baca file train_labels.csv\n",
        "df = pd.read_csv('/content/train_labels.csv')\n",
        "df.head()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "XD-_rARCDsSB",
        "outputId": "693d86d7-17ae-4f94-aec3-e648901518f7"
      },
      "execution_count": 38,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "               article_id                                 dataset_id     type\n",
              "0    10.1002_2017jc013030             https://doi.org/10.17882/49388  Primary\n",
              "1  10.1002_anie.201916483  https://doi.org/10.5517/ccdc.csd.cc1npvt0  Missing\n",
              "2  10.1002_anie.202005531  https://doi.org/10.5517/ccdc.csd.cc24wxqp  Missing\n",
              "3  10.1002_anie.202007717  https://doi.org/10.5517/ccdc.csd.cc24rrb0  Missing\n",
              "4  10.1002_chem.201902131  https://doi.org/10.5517/ccdc.csd.cc221dk3  Missing"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-58d1b04c-e5bf-4fff-896c-1cbf307a4953\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>article_id</th>\n",
              "      <th>dataset_id</th>\n",
              "      <th>type</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>10.1002_2017jc013030</td>\n",
              "      <td>https://doi.org/10.17882/49388</td>\n",
              "      <td>Primary</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>10.1002_anie.201916483</td>\n",
              "      <td>https://doi.org/10.5517/ccdc.csd.cc1npvt0</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>10.1002_anie.202005531</td>\n",
              "      <td>https://doi.org/10.5517/ccdc.csd.cc24wxqp</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>10.1002_anie.202007717</td>\n",
              "      <td>https://doi.org/10.5517/ccdc.csd.cc24rrb0</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>10.1002_chem.201902131</td>\n",
              "      <td>https://doi.org/10.5517/ccdc.csd.cc221dk3</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-58d1b04c-e5bf-4fff-896c-1cbf307a4953')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-58d1b04c-e5bf-4fff-896c-1cbf307a4953 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-58d1b04c-e5bf-4fff-896c-1cbf307a4953');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-ba3c25d5-4a0b-43e7-af0a-47e1fa5ed93e\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-ba3c25d5-4a0b-43e7-af0a-47e1fa5ed93e')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-ba3c25d5-4a0b-43e7-af0a-47e1fa5ed93e button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "df",
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1066,\n  \"fields\": [\n    {\n      \"column\": \"article_id\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 523,\n        \"samples\": [\n          \"10.7717_peerj.12422\",\n          \"10.1371_journal.pone.0198382\",\n          \"10.1002_chem.202000235\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"dataset_id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1066,\n        \"samples\": [\n          \"https://doi.org/10.7937/tcia.2020.6c7y-gq39\",\n          \"https://doi.org/10.6084/m9.figshare.12027189.v1\",\n          \"K02404\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"type\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 3,\n        \"samples\": [\n          \"Primary\",\n          \"Missing\",\n          \"Secondary\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 38
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "texts = []\n",
        "labels = []\n",
        "\n",
        "for idx, row in df.iterrows():\n",
        "    article_id = row['article_id']\n",
        "    label = row['type']\n",
        "    file_path = f'/content/{article_id}.xml'  # File XML harus sesuai nama article_id\n",
        "\n",
        "    if os.path.exists(file_path):\n",
        "        try:\n",
        "            tree = ET.parse(file_path)\n",
        "            root = tree.getroot()\n",
        "            # Gabungkan semua isi teks dalam elemen XML\n",
        "            sentences = [elem.text.strip() for elem in root.iter() if elem.text]\n",
        "            full_text = \" \".join(sentences)\n",
        "\n",
        "            texts.append(full_text)\n",
        "            labels.append(label)\n",
        "        except:\n",
        "            print(f\"⚠️ Gagal parsing file: {file_path}\")\n",
        "    else:\n",
        "        print(f\"❌ File tidak ditemukan: {file_path}\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RYZ884cMDtIS",
        "outputId": "f64e722f-b56d-40da-96b7-3820436ae035"
      },
      "execution_count": 39,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "❌ File tidak ditemukan: /content/10.1002_ecs2.1280.xml\n",
            "❌ File tidak ditemukan: /content/10.1002_ejoc.202000139.xml\n",
            "❌ File tidak ditemukan: /content/10.1002_esp.5058.xml\n",
            "❌ File tidak ditemukan: /content/10.1002_nafm.10870.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_jhep07(2018)134.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_jhep07(2018)134.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_jhep11(2018)113.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_jhep11(2018)113.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_jhep11(2018)115.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_jhep12(2018)117.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_s00382-012-1636-1.xml\n",
            "❌ File tidak ditemukan: /content/10.1007_s10904-014-0054-8.xml\n",
            "❌ File tidak ditemukan: /content/10.1016_j.fuel.2022.125768.xml\n",
            "❌ File tidak ditemukan: /content/10.1016_j.fuel.2022.125768.xml\n",
            "❌ File tidak ditemukan: /content/10.1016_j.jlp.2022.104761.xml\n",
            "❌ File tidak ditemukan: /content/10.1016_j.jlp.2022.104761.xml\n",
            "❌ File tidak ditemukan: /content/10.1016_j.jsames.2022.103824.xml\n",
            "❌ File tidak ditemukan: /content/10.1016_j.websem.2024.100815.xml\n",
            "❌ File tidak ditemukan: /content/10.1017_rdc.2022.19.xml\n",
            "❌ File tidak ditemukan: /content/10.1021_acs.jcim.9b01185.xml\n",
            "❌ File tidak ditemukan: /content/10.1029_2019jg005297.xml\n",
            "❌ File tidak ditemukan: /content/10.1029_2019jg005297.xml\n",
            "❌ File tidak ditemukan: /content/10.1029_2019pa003774.xml\n",
            "❌ File tidak ditemukan: /content/10.1029_2021ea001703.xml\n",
            "❌ File tidak ditemukan: /content/10.1029_2021pa004379.xml\n",
            "❌ File tidak ditemukan: /content/10.1038_hdy.2013.74.xml\n",
            "❌ File tidak ditemukan: /content/10.1038_hdy.2014.75.xml\n",
            "❌ File tidak ditemukan: /content/10.1038_s41893-020-0559-9.xml\n",
            "❌ File tidak ditemukan: /content/10.1039_c9ce01020c.xml\n",
            "❌ File tidak ditemukan: /content/10.1039_d0dt02130j.xml\n",
            "❌ File tidak ditemukan: /content/10.1039_d0gc00363h.xml\n",
            "❌ File tidak ditemukan: /content/10.1051_e3sconf_202017220004.xml\n",
            "❌ File tidak ditemukan: /content/10.1073_pnas.1323607111.xml\n",
            "❌ File tidak ditemukan: /content/10.1073_pnas.1705601114.xml\n",
            "❌ File tidak ditemukan: /content/10.1073_pnas.1711872115.xml\n",
            "❌ File tidak ditemukan: /content/10.1073_pnas.1711872115.xml\n",
            "❌ File tidak ditemukan: /content/10.1073_pnas.1915395117.xml\n",
            "❌ File tidak ditemukan: /content/10.1073_pnas.1915395117.xml\n",
            "❌ File tidak ditemukan: /content/10.1073_pnas.1915395117.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_01621459.2019.1573732.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_02713683.2019.1607392.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_0284186x.2019.1669817.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_0284186x.2020.1714721.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_07350015.2020.1766469.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_13501763.2019.1630470.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1080_19475683.2023.2226196.xml\n",
            "❌ File tidak ditemukan: /content/10.1098_rspb.2015.2764.xml\n",
            "❌ File tidak ditemukan: /content/10.1101_096297.xml\n",
            "❌ File tidak ditemukan: /content/10.1107_s0108270106019688.xml\n",
            "❌ File tidak ditemukan: /content/10.1107_s2414314616000523.xml\n",
            "❌ File tidak ditemukan: /content/10.1107_s2414314616007033.xml\n",
            "❌ File tidak ditemukan: /content/10.1107_s2414314616007227.xml\n",
            "❌ File tidak ditemukan: /content/10.1107_s2414314619012446.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_1365-2435.13087.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_1365-2656.12501.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_1365-2656.12594.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_1365-2664.13128.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_1365-2664.13168.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_1365-2664.13446.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_1365-2745.13449.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_evo.13972.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_mec.16743.xml\n",
            "❌ File tidak ditemukan: /content/10.1111_mec.16743.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-018-6033-4.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-018-6033-4.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-019-6583-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-019-6583-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-019-6583-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-019-6583-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-019-6583-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-019-6583-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjc_s10052-019-6583-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1140_epjds_s13688-018-0132-1.xml\n",
            "❌ File tidak ditemukan: /content/10.1186_s13617-019-0084-6.xml\n",
            "❌ File tidak ditemukan: /content/10.1186_s13750-020-0184-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1186_s13750-020-0184-0.xml\n",
            "❌ File tidak ditemukan: /content/10.1186_s40657-018-0125-6.xml\n",
            "❌ File tidak ditemukan: /content/10.1186_s40657-020-00194-w.xml\n",
            "❌ File tidak ditemukan: /content/10.1242_jeb.211466.xml\n",
            "❌ File tidak ditemukan: /content/10.1364_oe.25.001985.xml\n",
            "❌ File tidak ditemukan: /content/10.14358_pers.22-00039r2.xml\n",
            "❌ File tidak ditemukan: /content/10.14379_iodp.proc.390393.208.2024.xml\n",
            "❌ File tidak ditemukan: /content/10.14379_iodp.proc.390393.208.2024.xml\n",
            "❌ File tidak ditemukan: /content/10.1534_genetics.120.303190.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0001-3765201920180768.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0001-3765201920190822.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0034-7167-2018-0065.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0034-737x201966050002.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0047-2085000000239.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0102-778632120150164.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0102-77863220007.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0102.3772e35417.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0103-6351_4070.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0103-6440201802861.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0104-4060.59642.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0104-530x4900-19.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_0104-6632.20190362s20180340.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1517-869220192505126869.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1519-6984.192126.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1678-4162-10032.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1678-4162-10752.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1678-4499.20190067.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1679-78255326.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1679-78255753.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1806-90882019000500004.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1806-93042019000400002.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1806-9479.2019.185555.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1807-3107bor-2019.vol33.0095.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1809-2950_19003927012020.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1809-2950_19008627012020.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1809-4430-eng.agric.v40n1p24-33_2020.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1809-6891v21e-43578.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1809-9823.2007.10034.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1980-5373-mr-2018-0766.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1980-5373-mr-2018-0921.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1980-5373-mr-2019-0338.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1980-5373-mr-2019-0338.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1980-5918.032.ao27.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1980-5918.033.ao15.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1981-22562019022.190037.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1981-52712015v43n3rb20180090.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1981-6723.06017.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1981-86372019000243506.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1982-0216_20192158519.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_1983-21252019v32n324rc.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_2236-3459_83528.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_2237-6089-2017-0134.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_fst.33717.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_fst.36617.xml\n",
            "❌ File tidak ditemukan: /content/10.1590_s1678-86212019000400340.xml\n",
            "❌ File tidak ditemukan: /content/10.17815_jlsrf-2-58.xml\n",
            "❌ File tidak ditemukan: /content/10.18438_eblip29674.xml\n",
            "❌ File tidak ditemukan: /content/10.21203_rs.3.rs-2814013_v1.xml\n",
            "❌ File tidak ditemukan: /content/10.3389_fevo.2023.1112519.xml\n",
            "❌ File tidak ditemukan: /content/10.3389_fevo.2023.1112519.xml\n",
            "❌ File tidak ditemukan: /content/10.3389_fevo.2023.1112519.xml\n",
            "❌ File tidak ditemukan: /content/10.3390_cryst2031058.xml\n",
            "❌ File tidak ditemukan: /content/10.3390_cryst9060301.xml\n",
            "❌ File tidak ditemukan: /content/10.3390_rs12121957.xml\n",
            "❌ File tidak ditemukan: /content/10.3847_1538-4357_aae92c.xml\n",
            "❌ File tidak ditemukan: /content/10.3897_jhr.96.111550.xml\n",
            "❌ File tidak ditemukan: /content/10.3897_jhr.96.111550.xml\n",
            "❌ File tidak ditemukan: /content/10.4025_actasciagron.v40i1.39392.xml\n",
            "❌ File tidak ditemukan: /content/10.5194_essd-8-663-2016.xml\n",
            "❌ File tidak ditemukan: /content/10.5194_se-13-1541-2022.xml\n",
            "❌ File tidak ditemukan: /content/10.5194_tc-17-3617-2023.xml\n",
            "❌ File tidak ditemukan: /content/10.5194_tc-17-3617-2023.xml\n",
            "❌ File tidak ditemukan: /content/10.5194_tc-17-3617-2023.xml\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Gabungkan teks dan label jadi satu DataFrame\n",
        "data = pd.DataFrame({'text': texts, 'label': labels})\n",
        "data.head()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "P9dmM2_2D1zS",
        "outputId": "1d986652-72ed-4cbf-838a-44f85932bf05"
      },
      "execution_count": 40,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                text    label\n",
              "0      Assessing the variability in the relations...  Primary\n",
              "1  Angew Chem Int Ed Engl Angew. Chem. Int. Ed. E...  Missing\n",
              "2  BioC-API 20250509 collection.key PMC7540508 CC...  Missing\n",
              "3  Angew Chem Int Ed Engl Angew Chem Int Ed Engl ...  Missing\n",
              "4  Chemistry Chemistry 10.1002/(ISSN)1521-3765 CH...  Missing"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-f20506c3-0143-416a-ab35-f49515d95e7e\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>text</th>\n",
              "      <th>label</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Assessing the variability in the relations...</td>\n",
              "      <td>Primary</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>Angew Chem Int Ed Engl Angew. Chem. Int. Ed. E...</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>BioC-API 20250509 collection.key PMC7540508 CC...</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>Angew Chem Int Ed Engl Angew Chem Int Ed Engl ...</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>Chemistry Chemistry 10.1002/(ISSN)1521-3765 CH...</td>\n",
              "      <td>Missing</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-f20506c3-0143-416a-ab35-f49515d95e7e')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-f20506c3-0143-416a-ab35-f49515d95e7e button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-f20506c3-0143-416a-ab35-f49515d95e7e');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-082adf9a-b120-4be4-a1c2-12354148221e\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-082adf9a-b120-4be4-a1c2-12354148221e')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-082adf9a-b120-4be4-a1c2-12354148221e button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "data",
              "summary": "{\n  \"name\": \"data\",\n  \"rows\": 902,\n  \"fields\": [\n    {\n      \"column\": \"text\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 399,\n        \"samples\": [\n          \"BMC Plant Biol BMC Plant Biol BMC Plant Biology 1471-2229 BioMed Central London 6670228 1889 10.1186/s12870-019-1889-5 Research Article Outlook for coeliac disease patients: towards bread wheat with hypoimmunogenic gluten by gene editing of \\u03b1- and \\u03b3-gliadin gene families Jouanin Aur\\u00e9lie aurelie.jouanin@gmail.com 1 2 Schaart Jan G. jan.schaart@wur.nl 1 Boyd Lesley A. lesley.boyd@niab.com 2 Cockram James james.cockram@niab.com 2 Leigh Fiona J. fiona.leigh@niab.com 2 Bates Ruth ruth.bates@niab.com 2 Wallington Emma J. emma.wallington@niab.com 2 Visser Richard G. F. Richard.Visser@wur.nl 1 http://orcid.org/0000-0002-8549-6046 Smulders Marinus J. M. rene.smulders@wur.nl 1 1 0000 0001 0791 5666 grid.4818.5 Wageningen University and Research, Plant Breeding, 2 0000 0004 0383 6532 grid.17595.3f The John Bingham Laboratory, NIAB, 1 8 2019 1 8 2019 2019 19 333 16 10 2018 17 6 2019 \\u00a9 The Author(s). 2019 Open Access http://creativecommons.org/licenses/by/4.0/ http://creativecommons.org/publicdomain/zero/1.0/ Background Wheat grains contain gluten proteins, which harbour immunogenic epitopes that trigger Coeliac disease in 1\\u20132% of the human population. Wheat varieties or accessions containing only safe gluten have not been identified and conventional breeding alone struggles to achieve such a goal, as the epitopes occur in gluten proteins encoded by five multigene families, these genes are partly located in tandem arrays, and bread wheat is allohexaploid. Gluten immunogenicity can be reduced by modification or deletion of epitopes. Mutagenesis technologies, including CRISPR/Cas9, provide a route to obtain bread wheat containing gluten proteins with fewer immunogenic epitopes. Results In this study, we analysed the genetic diversity of over 600 \\u03b1- and \\u03b3-gliadin gene sequences to design six sgRNA sequences on relatively conserved domains that we identified near coeliac disease epitopes. They were combined in four CRISPR/Cas9 constructs to target the \\u03b1- or \\u03b3-gliadins, or both simultaneously, in the hexaploid bread wheat cultivar Fielder. We compared the results with those obtained with random mutagenesis in cultivar Paragon by \\u03b3-irradiation. For this, Acid-PAGE was used to identify T1 grains with altered gliadin protein profiles compared to the wild-type endosperm. We first optimised the interpretation of Acid-PAGE gels using Chinese Spring deletion lines. We then analysed the changes generated in 360 Paragon \\u03b3-irradiated lines and in 117 Fielder CRISPR/Cas9 lines. Similar gliadin profile alterations, with missing protein bands, could be observed in grains produced by both methods. Conclusions The results demonstrate the feasibility and efficacy of using CRISPR/Cas9 to simultaneously edit multiple genes in the large \\u03b1- and \\u03b3-gliadin gene families in polyploid bread wheat. Additional methods, generating genomics and proteomics data, will be necessary to determine the exact nature of the mutations generated with both methods. Electronic supplementary material The online version of this article (10.1186/s12870-019-1889-5) contains supplementary material, which is available to authorized users. Keywords Gene editing CRISPR/Cas9 Mutation breeding \\u03b3-Irradiation Wheat Polyploid Gluten \\u03b1-Gliadin \\u03b3-Gliadin Coeliac disease http://dx.doi.org/10.13039/100011264 FP7 People: Marie-Curie Actions FP7-PEOPLE-2013_ITN-607178 issue-copyright-statement \\u00a9 The Author(s) 2019 Background Bread wheat ( Triticum aestivum 1 Wheat can cause allergies and/or intolerances after consumption in susceptible individuals [ 2 3 4 8 9 12 13 16 Bread wheat is hexaploid, so each locus is present on the homoeologous chromosome pairs of the three different sub-genomes, A, B and D. Immunogenic epitopes occur in \\u03b1- \\u03b3- and \\u03c9- gliadins and to a lesser extent in low molecular weight (LMW) glutenins while high molecular weight (HMW) glutenins are mostly safe for CD patients [ 17 1 18 Gli-2 19 20 Gli-1 Gli-3 21 22 18 23 25 18 26 13 16 27 Alternatively, RNAi targeting all three gliadin families has successfully decreased 97% of gliadin expression in bread wheat grain and the gluten extract did not stimulate CD patient T cells while dough quality was barely affected [ 28 29 30 31 32 15 33 34 16 In this study, we have transformed the hexaploid bread wheat cultivar Fielder with constructs containing Cas9 and combinations of single guide RNAs (sgRNA) targeting specific sites upstream of, or within, the CD epitopes in the \\u03b1-gliadin as well as the \\u03b3-gliadin gene family. These constructs would potentially create nucleotide changes and small indels in the epitope regions, or deletion of the epitope regions, either in, or out of frame. In the ideal scenario, small in-frame mutations would be generated, producing non-immunogenic gliadins which retained the desired rheological properties. Small out of frame mutations upstream the epitope region, would generate a knock out of the gliadin that should prevent the immunogenicity but may affect bread dough quality. However, the transcription of the truncated protein could prevent the often-reported compensation by other gliadin genes [ 35 36 Due to the high sequence complexity, variability and number of targeted gliadin genes, a simple PCR on wild type plants would give numerous bands. A CAPS assay to reveal mutations by identifying restriction site loss, would not be possible since not all sequences would carry the site. In addition, regular gene cloning and sequencing would not be an efficient pre-screening method, given the complexity of the genes, gene families and genomic loci under study. Indeed, the cloning of the 50 to 100 single different gene copies present in a plant cannot be guaranteed and would require deep sequencing for every single Fielder-CRISPR plant generated. In addition, as many gliadin genes are pseudogenes [ 18 26 We compared the changes that occurred in our Fielder-CRISPR gene-edited lines with those we found in selected lines from a \\u03b3-irradiated mutant collection in variety Paragon [ 37 38 39 Results Sequence alignment and sgRNA design Genomic sequences of 1273 \\u03b1-gliadin and 442 \\u03b3-gliadin genes from several wheat species and varieties were translated into protein sequences and manually aligned to infer position of epitope and most conserved regions, and to superimpose this information on the DNA sequence. The alignment was optimised by hand, especially around the repetitive parts of \\u03b1- and \\u03b3-gliadin protein sequences as presented in Additional\\u00a0file 1 2 For \\u03b1-gliadins, five protein subgroups exist based on sequence patterns, two associated to genome A, two to genome B, and one that presents a unique amino-acid variant specific for genome A or D. Some CD epitopes were present or absent in specific protein subgroups. As a consequence, some CD epitopes are specific for one homoeologous genome (Additional file 1 40 In \\u03b3-gliadins, six protein subgroups exist based on sequence patterns, two associated with genome A, one with genome B and the three others associated to genomes B and/or D (Additional file 2 2 41 Six sgRNAs were designed, three of which target \\u03b1-gliadins (Fig. 1 1 Fig. 1 Alignment of \\u03b1- and \\u03b3-gliadin protein sequences with sgRNA position and potential sites of gene editing. A representation of the protein sequence alignments of \\u03b1-gliadins ( a b 1 2 1 2 Our alignments show that as a result of the existing sequence differences, largely within the DQ2.5 epitope region (Additional file 1 2 1 1 2 Multiplexing sgRNAs in clustered gene families can generate many different types of mutation (Fig. 2 Fig. 2 Representation of one \\u03b1-gliadin Gli-2 Gli-2 Generation of CRISPR/Cas9_sgRNA constructs and transgenic fielder wheat plants Four different CRISPR/Cas9_sgRNA binary constructs were created containing Cas9 plus different combinations of the sgRNAs (Table 1 A. tumefaciens Cas9 1 nptII Table 1 Summary of T0 Fielder-CRISPR regenerated plants expressing Cas9 and their nptII copy number T0 Fielder wheat plants Regenerated Expressing Cas9 1 or 2 transgene copies Construct \\u03b11_sgRNA sgRNA_\\u03b187 40 38 (95%) 21 (55%) Construct \\u03b12_sgRNA sgRNA_\\u03b1213 + sgRNA_\\u03b1324 32 12 (38%) 4 (33%) Construct \\u03b33_sgRNA sgRNA_\\u03b386 + sgRNA_\\u03b3272 + sgRNA_\\u03b3603 36 35 (97%) 26 (74%) Construct \\u03b12\\u03b33_sgRNA sgRNA_\\u03b1213 + sgRNA_\\u03b1324 + sgRNA_\\u03b386 + sgRNA_\\u03b3272 + sgRNA_\\u03b3603 42 32* (76%) 12 (38%) TOTAL 150 117 (78%) 63 (54%) For each construct, between 32 and 42 plants were regenerated. The * indicates that 34 plants expressed Cas9 but 2 were discarded since they did not contain all 5 sgRNAs. Overall, 78% of plants expressed the Cas9, among which 54% had 1 or 2 copies of the nptII selectable marker gene whereas the remainder contained additional copies of nptII. A total of 150 T0 Fielder-CRISPR plantlets were regenerated, using the four constructs, among which 31 plants did not express the Cas9, had prematurely aborting grains, or died. Among the plants carrying the T-DNA, only two plants transformed with the \\u03b12\\u03b33_sgRNA construct did not carry all the sgRNAs. Generally, T0 wheat plants generated 4 to 6 ears and 80\\u2013250\\u2009T1 grains. A subset of T1 grain samples was analysed using Acid-PAGE. Acid-PAGE analysis of gliadin proteins Acid-Polyacrylamide Gel Electrophoresis has been used for decades to differentiate and identify wheat varieties based on their characteristic gliadin protein profile [ 42 Optimizing acid-PAGE analysis using Chinese spring deletion lines Chinese Spring (CS), a model variety for hexaploid bread wheat, plus the CS nullisomic/tetrasomic and CS Kansas deletion lines, identified as either missing and/or having substitutions of gliadin genes on known homoeologous chromosomes arms, were used to set up and optimise the gliadin protein profile screening method using Acid-PAGE. These deletion and substitution lines revealed differences in gliadin protein patterns compared with to CS. Various bands were missing or shifted on the Acid-PAGE, depending on which chromosomes (Chr1 \\u03b3- and \\u03c9-gliadins or Chr6 \\u03b1-gliadins) and which homoeologous genome (A, B or D) were changed (Fig. 3 3 3 Fig. 3 Acid-PAGE of Chinese Spring deletion lines showing altered endosperm gliadin protein profiles. Gliadin extracts from grains of Chinese Spring nullisomic/tetrasomic lines and Kansas deletion lines were run on non-denaturing Acid-PAGE alongside a gliadin extract from Chinese Spring (CS WT). The lanes displayed next to each other have been run alongside each other on the same gel but each panel represents a different gel. Each sample was always run alongside CS WT as a control. The black and grey arrows point at the changes in the protein groups from the deleted chromosome arms and in others respectively. CS gliadin profile in absence of a b c d e f In addition to optimising the method, the use of the CS deletion lines gave indications on the position of bands from \\u03b1-, \\u03b3- and \\u03c9-gliadins which are specifically associated with gliadin proteins originating from A, B or D homoeologous genomes [ 43 Analysis of the paragon \\u03b3-irradiated population Wheat grains from various Paragon \\u03b3-irradiated lines were screened using Acid-PAGE. Differences in gliadin proteins profiles were observed between Paragon and M4 grains in 14 out of 360 \\u03b3-irradiated lines tested (3.88%). Three lines showed differences in only the \\u03b1-gliadins (Fig. 4 4 4 2 Fig. 4 Acid-PAGE of selected Paragon \\u03b3-irradiated mutant lines that showed changes in gliadin protein profiles. Gliadin extracts from grains of the M4 generation of Paragon \\u03b3-irradiated mutant population were run on non-denaturing Acid-PAGE alongside a gliadin extract from Paragon (Paragon WT). Each panel represent a different gel. The lanes displayed next to each other have been run alongside each other. The black arrows point at the changes observed in the irradiated lines. a b c d e f Table 2 \\u03b3-irradiated Paragon lines and the inferred chromosome arms underlying the gliadin protein profile variations Altered protein profile \\u03b1 \\u03b3 \\u03c9 P3-75 6AS - - P4-34 - 1BS - P4-84 - 1BS 1BS P4-86 - 1BS 1BS P5-20 - 1AS 1AS P5-53 - - 1BS P6-02 - 1AS 1AS P6-43 6DS 1AS - P6-57 6AS - - P6-59 6DS - - P6-60 - - 1AS P6-65 - - 1AS P6-74 - - 1AS P10-73 - 1AS 1AS Paragon \\u03b3-irradiated lines and the gliadin families in which altered protein profiles are seen on Acid-PAGE. The chromosome arms inferred to be altered by the \\u03b3-irradiation and to cause the protein profiling changes are indicated in the table. In total, four lines showed changes in the \\u03b1-gliadins, seven lines had differences in the \\u03b3-gliadins and nine lines displayed variations in the \\u03c9-gliadins. P6\\u201357 had large changes in \\u03b1-gliadins from, most likely, 6AS. By comparing the Paragon mutant gliadin protein profiles with those obtained from CS deletion lines and CS nullisomic/tetrasomic lines, it was possible to infer the homoeologous chromosome location of the gliadin genes altered by \\u03b3-irradiation mutagenesis in the identified Paragon lines (Table 2 Analysis of the fielder CRISPR-Cas9 plants For all of the 117 T0 plants expressing the Cas9 mRNA and carrying the full complement of expected sgRNAs, 8 or more randomly selected T1 grains per plant were screened using Acid-PAGE. Differences in the gliadin protein profile were observed between Fielder wild type and T1 grains harvested from some T0 CRISPR/Cas9 plants (Fig. 5 3 Fig. 5 Acid-PAGE of T1 grains showing altered gliadin protein profiles. Gliadin extracts from Fielder-CRISPR T1 grain from each of the 4 constructs were run on non-denaturing Acid-PAGE alongside the gliadin extract from Fielder wild type. Each panel represent a different gel. The lanes displayed next to each other have been run alongside each other. The start of the sample names refers to the constructs with the sgRNAs they include (\\u03b11, \\u03b12, \\u03b33 or \\u03b12\\u03b33), followed by the T0 plant line and grain number. The black and grey arrows point respectively at the changes intended or unintended by the construct present in the plant that set the grains. a b c d Table 3 Summary of T1 Fielder-CRISPR grains, and associated T0 plants, with modified gliadin protein profile CRISPR/Cas9 constructs T0 plants expressing Cas9 T1 grains tested on Acid-PAGE T0 plants giving T1 mutant grains with clear changes T1 grains with clear changes in gliadin profile T0 plants giving T1 mutant grains with any changes T1 grains with any changes in gliadin profile Construct \\u03b11_sgRNA sgRNA_\\u03b187 38 325 3 (7.9%) 4 (1.2%) 3 (7.9%) 8 (2.5%) Construct \\u03b12_sgRNA sgRNA_\\u03b1213 + sgRNA_\\u03b1324 12 288 7 (58.3%) 13 (4.5%) 7 (58.3%) 19 (6.6%) Construct \\u03b33_sgRNA sgRNA_\\u03b386 + sgRNA_\\u03b3272 + sgRNA_\\u03b3603 35 280 10 (28.6%) 10 (3.6%) 15 (42.9%) 23 (8.2%) Construct \\u03b12\\u03b33_sgRNA sgRNA_\\u03b1213 + sgRNA_\\u03b1324 + sgRNA_\\u03b386 + sgRNA_\\u03b3272 + sgRNA_\\u03b3603 32 256 3 (9.4%) 11 (4.3%) 11 (34.4%) 20 (7.8%) TOTAL 117 1149 23 (19.6%) 38 (3.3%) 36 (30.8%) 70 (6.1%) The plants and associated T1 grains tested on Acid-PAGE, as well as the number of T0 plants giving T1 mutant grains with \\u201cclear changes\\u201d or \\u201cany changes\\u201d in Acid-PAGE profiles, and the number of T1 mutant grains showing \\u201cclear changes\\u201d and \\u201cany changes\\u201d. For each construct mutations were observed. Mutations occurring in both the targeted and non-targeted gliadin families were included in \\u201cany changes\\u201d here. Modified profiles were found for each of the 4 CRISPR/Cas9_sgRNA constructs used (Fig. 5 5 5 5 5 5 5 The constructs differed significantly in mutation efficiency for any changes (Chi square, P P 3 3 It was observed that construct \\u03b33_sgRNA and construct \\u03b12\\u03b33_sgRNA produced grains which relatively often had differences in \\u03c9-gliadin expression although these constructs do not target this gliadin family. This phenomenon was rarely observed with construct \\u03b12_sgRNA and seen only once with construct \\u03b11_sgRNA. No correlation was found between transgene copy number present in a plant and the number of grains presenting an altered gliadin protein profile, nor in the number of mutations seen per grain. Since the Cas9 expression was observed by RT-PCR, but was not quantified, no correlation can be made between its expression level and the number of gliadin protein changes observed using Acid-PAGE. Discussion In this study, CRISPR/Cas9 was deployed in hexaploid bread wheat to simultaneously target multiple gliadin genes. The Cas9-induced mutations were designed to remove gliadin gene copies or disrupt their immunogenic epitopes in gliadins, so preventing them from triggering the human immune system and causing coeliac disease upon wheat gluten consumption. At the start of this study, CRISPR/Cas9 had not been used to target large clusters of repeated gene families, such as the gliadins, in polyploid crops such as hexaploid wheat. Therefore, the challenge was not only to generate such mutant plants but also to develop screening methods to identify them and characterise their mutations further. Gliadin alignments, sgRNA design and generation of fielder-CRISPR lines generation In the absence of \\u03b1- and \\u03b3-gliadin genomic or proteomic sequences from the hexaploid wheat cultivar Fielder used for gliadin gene editing experiments, and considering the time and resources necessary to fully clone and sequence these gene families, an alternative approach was used mainly based on publicly available data. Only seven \\u03b1-gliadins from Fielder were cloned, sequenced and merged with all \\u03b1-gliadin genomic sequences from GenBank/ENA. The Fielder sequences were different to one another but similar to some sequences already present in GenBank/ENA. Deduced protein sequences were manually aligned to enable inferring gliadin epitope positions and conserved regions in gliadin gene sequences across cultivars. Using this approach, it was possible to design sgRNA targeting as many gliadin genes as possible near their epitope region, not only in Fielder but also in many other different cultivars. Alignment of gliadin sequences also revealed the presence of groups based on sequences patterns, some associated to specific homoeologous genomes, similarly to what was observed by Ozuna et al. [ 20 The sequences downloaded from GenBank/ENA originated from different hexaploid varieties. When sequences from one cultivar are uploaded into the database, there is no guarantee that the set of sequence variants per cultivar is complete. Furthermore, gliadin genes are present in each homoeologous genome and clustered at specific loci as multiple copy repeats, with identical or variable sequences. A gene sequence found multiple times in one cultivar, is uploaded only once as new sequence to avoid redundancy. However, if an identical sequence was found in different cultivars or in the same cultivar in different studies by different groups, it will be present several times in the database. Therefore, the frequency with which a sequence is present in the alignments generated for this study does not reflect the frequency at which a gene sequence is present in the genome of a cultivar such as Fielder. As a result, the proportion of sequences targeted by the different sgRNA designed does not reflect the real percentages of matching sequences present in Fielder, especially since the evaluation was made on 100% match while in reality, sgRNA are known to also target sequences with slightly lower similarity [ 44 Four CRISPR/Cas9_sgRNA constructs, containing 1 to 5 different sgRNAs, were designed and used to target \\u03b1- and/or \\u03b3-gliadin gene families, in hexaploid wheat. The constructs were stably transformed into bread wheat cultivar Fielder. A total of 117 regenerated lines expressed Cas9, regardless of which construct they carried. The T1 grains generated by these plants potentially contain mutations at a number of target sites which are segregating independently. However, it is possible that the Cas9-induced mutation is heterozygous in a cell. It is also possible that Cas9 did not cut the same genes in different cells of the T0 plant, thus generating chimerical plants or even that Cas9 remained active in the gametes after meiosis. This means that, following segregations events, each grain generated by a T0 plant could have a unique assortment of gliadin mutations. For this reason, each grain was cut into three parts, the embryo and two identical pieces of endosperm, to run further complementary types of analysis (sequencing and advanced proteomics) on the identical fractions of endosperm isolated from the same T1 or T2 grain. Pre-screening for wheat grains with mutated gliadins Due to the high complexity of the large gliadin gene families that are only expressed in wheat grains, the traditional screening methods such as restriction site loss or sequencing are not appropriate for pre-screening and identification of potential mutants. Instead, these methods would be relevant for in depth study and characterisation of the mutations occurring in interesting mutants identified using different pre-screening methods. Acid-PAGE was successful for identifying gliadin protein profile differences [ 42 The optimised Acid-PAGE protocol was then used to identify 14 lines out of 360 (3.9%) Paragon \\u03b3-irradiated lines that showed gliadin proteins expression changes compared to wild type Paragon. Lines showed changes in one or two gliadin families but never in all three families simultaneously. Irradiation mutation is known to trigger large deletions, up to several mega-bases, which can explain the deletion of a complete gliadin gene locus. The nature of the homoeologous chromosomes altered by \\u03b3-radiation were inferred based on the results obtained using the CS deletion line resources, since similarly large deletions are expected in both line sets. The percentage of mutations observed in Paragon \\u03b3-irradiated germplasm using on Acid-PAGE was around 4%, for all visible changes in the three large gliadin gene families, each counting more than dozens of members. Similarly, Acid-PAGE allowed pre-screening of T1 germplasm generated via CRISPR/Cas9 gene editing. Changed gliadin profiles were observed with each construct, implying that each construct contained at least one sgRNA that successfully generated mutations in some copies of the targeted \\u03b1- or \\u03b3-gliadin gene family. More importantly, it indicated that CRISPR/Cas9 can edit a sufficient number of genes within large family in polyploid plants to actually generate a different phenotype in the progeny. The Acid-PAGE analysis of the Fielder-CRISPR lines revealed differences in gliadin profiles in 70 T1 grains (6.1% of the total number of grains tested) harvested from 36 T0 plants (30.8% of the total number of T0 plants expressing CRISPR/Cas9 constructs) across all four CRISPR/Cas9_sgRNA constructs used. Therefore, a single T0 plant that produced T1 mutant grains, gave on average 2 mutant grains out of 8 grains tested. Constructs targeting \\u03b3-gliadins sometimes triggered a shift of protein bands in the untargeted \\u03c9-gliadins. This phenomenon could be explained by the deletion of \\u03c9-gliadins located between targeted \\u03b3-gliadins since both gene families are suspected to have some overlap on the short arm of chromosome 1 [ 45 35 36 Variation in gene editing efficiency could be observed between the constructs, with construct \\u03b11_sgRNA being significantly less efficient. This could be explained by the presence of only one sgRNA in this construct while several are present in the other constructs used. It could also be due to a presence of secondary structure in the sgRNA \\u2013 identified after use, using RNAfold software - that could decrease the target binding efficiency. The number of sgRNA in a construct appears to slightly increase the ratio of mutant grains obtained, but no direct correlation was observed. Indeed, construct \\u03b12_sgRNA and construct \\u03b33_sgRNA generated 19 and 23 mutant grains respectively while construct \\u03b12\\u03b33_sgRNA \\u2013 combining both previous sgRNA guides in a single construct - gave only 20 mutant grains, which does not show a cumulative efficiency. However, estimation of the Cas9 expression level in different plants as well as the actual number of mutations generated at the DNA level would be necessary to give a robust answer regarding the variation of efficiency of the different constructs. Comparison with other groups also using CRISPR/Cas9 to target gluten genes in hexaploid wheat S\\u00e1nchez-Le\\u00f3n et al. [ 16 16 46 47 47 16 46 Comparison of CRISPR/Cas9 targeted mutations with random mutagenesis using \\u03b3-irradiation Using either CRISPR/Cas9 or \\u03b3-irradiation, the mutation efficiency revealed by the Acid-PAGE is comparable. However, the actual mutation rate obtained by using CRISPR is probably much higher. Fielder-CRISPR lines tested correspond to T1 generation grain potentially heterozygous for mutations while Paragon \\u03b3-irradiated lines represented M4 generation grain where most mutations are homozygous and easier to visualise. In addition, Acid-PAGE will probably not reveal amino-acid substitutions or small in-frame indels potentially generated by CRISPR/Cas9 since proteins with different sequences but having similar molecular weight/charge ratio can be represented by a single band on the gel [ 48 Interestingly, some T1 CRISPR-Fielder grains showed an altered gliadin protein profile similar to some M4 Paragon \\u03b3-irradiated grains (Fig. 5 4 14 Conclusion This pilot study aimed to mutate the large \\u03b1- and \\u03b3-gliadin gene families in hexaploid wheat to decrease gluten immunogenicity for coeliac patients. We succeeded in generating wheat grains that contained gliadin protein profiles altered for the targeted gene family in the bread wheat cultivar Fielder by targeted mutagenesis using CRISPR/Cas9 and in identifying them in Paragon germplasm induced by random mutagenesis using \\u03b3-irradiation. As many gliadin genes appear likely not to have been altered, optimisation via use of different promoter and different Cas9 genes or new Base Editor systems is required to obtain wheat plants which are safe for CD patients. The pre-screening and identification of the mutations was performed using Acid-PAGE, which identifies mostly the non-sense mutations and large deletions. However, the final purpose is to modify the epitopes into safe versions without knocking out the complete gliadin genes, in order to avoid compensation by other gliadins and to maintain baking quality. Therefore, reliable high throughput methods will be important for small in-frame mutation detection in the known CD epitopes [ 49 50 Segregating out the CRISPR/Cas9 construct from promising lines and subsequently self-pollinating these wheat lines to make all mutations homozygous would be the next steps. Potentially interesting lines would then need extensive investigation including immunological tests using monoclonal antibodies to determine their immunogenicity level and rheological studies to evaluate the bread dough quality obtained using these generated \\u201chypoimmunogenic-gluten\\u201d wheat lines. Methods Gliadin sequence alignment Over 438 \\u03b1-gliadin gene sequences from 30 wheat accessions and 187 \\u03b3-gliadin gene sequences from hexaploid wheat were downloaded from GenBank/ENA in September 2014 and July 2015 respectively, translated into amino acid sequences, manually aligned using Mega_6 and clustered based on their combination of known CD epitopes [ 49 Triticum monococcum Triticum urartu Aegilops speltoides Aegilops tauschii Triticum turgidum 25 sgRNA protospacer design Six sgRNA protospacers were designed on conserved regions that were identified based on the sequence alignments and were present in the sequences of the hexaploid wheat cultivar Fielder. Of these six sgRNAs, three targeted \\u03b1-gliadins and the other three targeted \\u03b3-gliadins (Fig. 2 sgRNA_\\u03b187: 5\\u2032-GATTTTGTGGCTGCAATTG-3\\u2032 targets \\u03b1-gliadins downstream the signal peptide, P87. sgRNA_\\u03b1213: 5\\u2032-ATGGTTGTTGTGATGGAAA-3\\u2032 targets \\u03b1-gliadins upstream the epitope region, P213. sgRNA_\\u03b1324: 5\\u2032-GTTGTGGTCGAAATGGTTG-3\\u2032 targets \\u03b1-gliadins downstream the epitope region, P324. sgRNA_\\u03b386: 5\\u2032-TTGTTGTGGCCATTGTACT-3\\u2032 targets \\u03b3-gliadins downstream the signal peptide, P86. sgRNA_\\u03b3272: 5\\u2032-AATGGTTGTTGTGGTTGCTG-3\\u2032 targets \\u03b3-gliadins within the epitope region, P274. sgRNA_\\u03b3603: 5\\u2032-TGCTGGGGGAATGATTGTTG-3\\u2019 targets \\u03b3-gliadins downstream the epitope region, P603. These sgRNA protospacers where tested in silico for the absence of off-targets using BLAST in the Ensembl! plant wheat database, for absence of secondary structures using RNAfold web server and for absence of cross-dimers between multiplexed sgRNAs using ThermoFisher Scientific Primer Analyzer. Each sgRNA, including a wheat-optimised U6 promoter, gliadin-specific protospacer and sgRNA scaffold sequence, flanked by multiple unique restriction sites, was individually synthesised by GenScript. CRISPR/Cas9-sgRNA constructs Type II-A Streptococcus pyogenes 51 52 1 Four T-DNA constructs were produced by combining different sgRNAs, using multiple unique restriction sites: Construct 1\\u03b1_sgRNA\\u2009=\\u2009sgRNA_\\u03b187. Construct 2\\u03b1_sgRNA\\u2009=\\u2009sgRNA_\\u03b1213\\u2009+\\u2009sgRNA_\\u03b1324. Construct 3\\u03b3_sgRNA\\u2009=\\u2009sgRNA_\\u03b386\\u2009+\\u2009sgRNA_\\u03b3272\\u2009+\\u2009sgRNA_\\u03b3603. Construct 2\\u03b13\\u03b3_sgRNA\\u2009=\\u2009sgRNA_\\u03b1213\\u2009+\\u2009sgRNA_\\u03b1324\\u2009+\\u2009sgRNA_\\u03b386\\u2009+\\u2009sgRNA_\\u03b3272\\u2009+\\u2009sgRNA_\\u03b3603 (i.e., it combines Construct 2\\u03b1_sgRNA and Construct 3\\u03b3_sgRNA). Construct 2\\u03b1_sgRNA is shown as an example in Fig. 6 Fig. 6 CRISPR/Cas9 T-DNA construct 2\\u03b1 _sgRNA. Construct 2\\u03b1_sgRNA that contains sgRNA_ \\u03b1213 and sgRNA_ \\u03b1324, as an example of the four T-DNA constructs generated. They are similar, only the number and nature of sgRNA integrated are different. Stable transformation and generation of fielder-CRISPR plants and derived grains The CRISPR/Cas9-sgRNA binary vectors were transferred to A. tumefaciens 53 54 nptII nptII 54 The individual T1 grains were first cut transversally, alongside the embryo, so that this could be germinated later. The resulting endosperm section was then cut in half longitudinally. One half was used for Acid-PAGE. Selected T1 embryos were germinated in Petri dishes containing filter paper soaked with water. Seedlings were potted in compost after sufficient root development and transferred to growth chambers until grain set. Ears containing T2 grains were harvested individually. Other plant materials Chinese Spring (CS) wild type (WT) and selected CS Kansas deletion lines [ 39 55 1 38 1 The Paragon \\u03b3-irradiated population [ 37 4 Table 4 List of Triticum aestivum  Triticum aestivum Lines Description Cultivar 'Chinese Spring' (CS) WT Control 1AS-1 Kansas Deletion lines missing part of Chr 1 short arm that contain Gli-1 Gli-3 1AS-3 1BS-9 1BS-10 1DS-1 1DS-5 6AS-1 Kansas Deletion lines missing part of Chr 6 short arm that contain Gli-2 6BS-4/5BS-2 6DS-4 6DS-4/1BS-19 N1AT1D = 1DDB Nullisomic-tetrasomic lines with one homoeologous pair of Chr 1 that contain Gli-1 Gli-3 N1AT1B = 1BBD N1DT1A = 1AAB N1DT1B = 1BBA N1BT1D = 1DDA N1BT1A = 1AAD N6AT6D = 6DDB Nullisomic-tetrasomic lines with one homoeologous pair of Chr 6 that contain Gli-2 N6AT6B = 6BBD N6DT6A = 6AAB N6DT6B = 6BBA N6BT6D = 6DDA Cultivar 'Paragon' WT Control Line P3-47 to P6-79 and 360 \\u03b3-irradiated lines M4 generation Line P10-19 to P10-96 Acid-polyacrylamide gel electrophoresis (acid-PAGE) For each Fielder-CRISPR plant, a minimum of 8 T1 grains were screened individually, alongside Fielder WT. The grain samples were loaded in duplicate. As Fielder exhibit some heterogeneity in its storage protein profile on Acid-PAGE, grains from multiple Fielder plants were loaded alongside on the gel. For T2, four grain samples from four different ears were loaded in duplicates. A similar procedure was applied for screening of CS and Paragon, with the exception that only two grains were tested as they are homogeneous for the mutations due to several generations of self-pollination. Each half endosperm sample was crushed into fine powder, and the gliadin fraction extracted overnight at 4\\u2009\\u00b0C, in 150\\u2009\\u03bcl of 25% dichloroethanol solution containing 0.05% Pyronin Y. Duplicate 30\\u2009\\u03bcl samples were loaded in wide slots on 13% polyacrylamide gels (acrylamide-bis 19:1) and run at 180\\u2009V, for 4\\u2009h at room temperature (adapted from [ 41 Additional files  Additional file 1: Figure S1. 3 5 Additional file 2: Figure S2. 4 6 Additional file 3: \\u03b1-gliadin protein sequences alignment. (FAS 3386 kb) Additional file 4: \\u03b3-gliadin protein sequences alignment. (FAS 1278 kb) Additional file 5: \\u03b1-gliadin protein sequences alignment. (DOCX 2533 kb) Additional file 6: \\u03b3-gliadin protein sequences alignment. (DOCX 2663 kb) Abbreviations Acid-PAGE Acid-Polyacrylamide Gel Electrophoresis CD Coeliac disease CRISPR clustered regularly interspaced short palindromic repeats GF Gluten-free gRNA Guide RNA HMW glutenins High molecular weight glutenins LMW glutenins Low molecular weight glutenins Publisher\\u2019s Note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Acknowledgements Melanie Craze and Sarah Bowden from the NIAB Crop Transformation team are thanked for creating the validated CRIPSR/Cas9 transgenic wheat lines. Helen Appleyard\\u2019s Analytical Services team, especially Sofia Alves and Pravina Solanki, are thanked for technical assistance with Acid-PAGE gels. The Paragon \\u03b3-irradiated population and the Chinese Spring nulli-tetrasomic lines were kindly provided by Dr. Clare Lister and Dr. Simon Griffiths of JIC, Norwich, UK, respectively. The Cas9 endonuclease and the sgRNA sequences used were kindly provided by Professor Caixia Gao, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, and described in Shan et al., 2013 [ 56 Authors\\u2019 contributions AJ performed the multi-gene alignments, designed and made the constructs, screened the \\u03b3-irradiated population, and performed the half-seed analysis with Acid-PAGE. EJW was responsible for wheat transformation and RB created validated transgenic plants from the multi-guide constructs. FJL coordinated research at NIAB with assistance from JC. LAB wrote the proposal and was PI on the Max-Crop project. JS assisted in construct development. MJMS and RGFV coordinated research at WUR. AJ wrote an initial draft of the paper, with advice and suggestions from JS and EJW. MJMS, RGFV, JS, EJW, JC, FJL and LAB reviewed the manuscript. AJ and MJMS edited the final version of the text. All authors approved the final version. Authors for correspondence: AJ, MJMS. Funding AJ was funded by FP7-PEOPLE-2013_ITN-607178. This manuscript does not necessarily reflect the Commission\\u2019s views and does not anticipate the European Commission\\u2019s future policy in this area. The funding body was not involved in the design of the study, in collection, analysis, and interpretation of data, nor in writing the manuscript. Availability of data and materials All sequence data analysed during this study are included in this published article and its Supplementary files. Ethics approval and consent to participate Not applicable. Consent for publication Not applicable. Competing interests The authors declare that they have no competing interests. References 1. Shewry PR D'Ovidio R Lafiandra D Jenkins JA Mills ENC Bekes F Wheat grain proteins. In K Khan, PR Shewry (Eds.), Wheat: Chemistry and Technology (4th edn.) 2009 St. Paul A.A.C.C 223 298 2. Fasano A Systemic autoimmune disorders in celiac disease Curr Opin Gastroenterol 2006 22 674 679 10.1097/01.mog.0000245543.72537.9e 17053448 3. Gilissen LJWJ Van der Meer IM Smulders MJM Reducing the incidence of allergy and intolerance to cereals J Cereal Sci 2014 59 337 353 10.1016/j.jcs.2014.01.005 4. Dicke WK Weijers HA Van de Kamer JH Coeliac disease. II. The presence in wheat of a factor having a deleterious effect in cases of coeliac disease Acta Paediatrica 1953 42 34 42 10.1111/j.1651-2227.1953.tb05563.x 13050382 5. M\\u00e4ki M Mustalahti K Kokkonen J Kulmala P Haapalahti M Karttunen T Ilonen J Laurila K Dahlbom I Hansson T H\\u00f6pfl P Prevalence of celiac disease among children in Finland New Engl J Med 2003 348 2517 2524 10.1056/NEJMoa021687 12815137 6. Rewers M Epidemiology of celiac disease: what are the prevalence, incidence, and progression of celiac disease? Gastroenterology. 2005 128 S47 S51 10.1053/j.gastro.2005.02.030 15825126 7. Catassi C Fasano A Coeliac disease: The debate on coeliac disease screening - are we there yet? Nat Rev Gastroenterol Hepatol 2014 11 457 458 10.1038/nrgastro.2014.119 25001972 8. Vriezinga SL Schweizer JJ Koning F Mearin ML Coeliac disease and gluten-related disorders in childhood Nat Rev Gastroenterol Hepatol 2015 12 527 536 10.1038/nrgastro.2015.98 26100369 9. Caponio F Summo C Clodoveo ML Pasqualone A Evaluation of the nutritional quality of the lipid fraction of gluten-free biscuits Eur Food Res Technol 2008 227 135 139 10.1007/s00217-007-0702-0 10. Capriles VD Ar\\u00eaas JAG Novel approaches in gluten-free bread making: interface between food science, nutrition, and health Compr Rev Food Sci Food Saf 2014 13 871 890 10.1111/1541-4337.12091 11. Belz MCE Reduction of salt in yeasted wheat bread: impact on bread quality and solutions using sourdough fermented by functional lactic acid bacteria strains 2016 12. Horstmann SW Belz MCE Heitmann M Zannini E Arendt EA Fundamental study on the impact of gluten-free starches on the quality of gluten-free model breads Foods 2016 5 30 10.3390/foods5020030 13. Jouanin Aur\\u00e9lie Gilissen Luud J.W.J. Boyd Lesley A. Cockram James Leigh Fiona J. Wallington Emma J. van den Broeck Hetty C. van der Meer Ingrid M. Schaart Jan G. Visser Richard G.F. Smulders Marinus J.M. Food processing and breeding strategies for coeliac-safe and healthy wheat products Food Research International 2018 110 11 21 10.1016/j.foodres.2017.04.025 30029701 14. Jouanin A Boyd LA Visser RGF Smulders MJM Development of wheat with hypoimmunogenic gluten obstructed by the gene editing policy in Europe Frontiers in Plant Science 2018 9 1523 10.3389/fpls.2018.01523 30405661 15. Ribeiro M Nunes FM Rodriguez-Quijano M Carrillo JM Branlard G Igrejas G Next-generation therapies for celiac disease: the gluten-targeted approaches Trends Food Sci Technol 2018 75 56 71 10.1016/j.tifs.2018.02.021 16. S\\u00e1nchez-Le\\u00f3n S Gil-Humanes J Ozuna CV Gim\\u00e9nez MJ Sousa C Voytas DF Barro F Low-gluten, nontransgenic wheat engineered with CRISPR/Cas9 Plant Biotechnol J 2018 16 902 910 10.1111/pbi.12837 28921815 17. Molberg \\u00d8 Fl\\u00e6te NS Jensen T Lundin KEA Arentz-Hansen H Anderson OD Intestinal T-cell responses to high-molecular-weight glutenins in celiac disease Gastroenterology 2003 125 337 344 10.1016/S0016-5085(03)00890-4 12891534 18. Van Herpen TWJM Goryunova SV Van der Schoot J Mitreva M Salentijn EMJ Vorst O Alpha-gliadin genes from the a, B and D genomes of wheat contain different sets of celiac disease epitopes BMC Genomics 2006 7 1 10.1186/1471-2164-7-1 16403227 19. Anderson O Litts J Greene F The \\u03b1-gliadin gene family. I. Characterization of ten new wheat \\u03b1-gliadin genomic clones, evidence for limited sequence conservation of flanking DNA, and southern analysis of the gene family Theor Appl Genet 1997 95 50 58 10.1007/s001220050531 20. Ozuna CV Iehisa JCM Gim\\u00e9nez MJ Alvarez JB Sousa C Barro F Diversification of the celiac disease \\u03b1-gliadin complex in wheat: a 33-mer peptide with six overlapping epitopes, evolved following polyploidization Plant J 2015 82 794 805 10.1111/tpj.12851 25864460 21. Sabelli PA Shewry PR Characterization and organization of gene families at the Gli-1 loci of bread and durum wheats by restriction fragment analysis Theor Appl Genet 1991 83 209 216 10.1007/BF00226253 24202360 22. Mitea C Salentijn EMJ Van Veelen P Goryunova SV Van der Meer IM Van den Broeck HC A universal approach to eliminate antigenic properties of alpha-gliadin peptides in celiac disease PLoS One 2010 5 e15637 10.1371/journal.pone.0015637 21179575 23. Van den Broeck HC De Jong HC Salentijn EMJ Dekking L Bosch D Hamer RJ Presence of celiac disease epitopes in modern and old hexaploid wheat varieties: wheat breeding may have contributed to increased prevalence of celiac disease Theor Appl Genet 2010 121 1527 1539 10.1007/s00122-010-1408-4 20664999 24. Van den Broeck HC Hongbing C Lacaze X Dusautoir JC Gilissen LJWJ Smulders MJM In search of tetraploid wheat accessions reduced in celiac disease-related gluten epitopes Mol BioSyst 2010 6 2206 2213 10.1039/c0mb00046a 20714643 25. Salentijn EMJ Esselink DG Goryunova SV Van der Meer IM Gilissen LJWJ Smulders MJM Quantitative and qualitative differences in celiac disease epitopes among durum wheat varieties identified through deep RNA-amplicon sequencing BMC Genomics 2013 14 905 10.1186/1471-2164-14-905 24354426 26. Salentijn EMJ Mitea DC Goryunova SV Van der Meer IM Padioleau I Gilissen LJWJ Celiac disease T cell epitopes from \\u03b3-gliadins: immunoreactivity depends on the genome of origin, transcript frequency, and flanking protein variation BMC Genomics 2012 13 277 10.1186/1471-2164-13-277 22726570 27. Smulders MJM Jouanin A Schaart J Visser RGF Cockram J Leigh F Koehler P Development of wheat varieties with reduced contents of celiac-immunogenic epitopes through conventional and GM strategies 2015 Nantes Proceedings of the 28th meeting of the working group on Prolamin analysis and toxicity, 25\\u201327 September 2014 47 56 28. Gil-Humanes J Pist\\u00f3n F Tollefsen S Sollid LM Barro F Effective shutdown in the expression of celiac disease-related wheat gliadin T-cell epitopes by RNA interference Proc Natl Acad Sci U S A 2010 107 17023 17028 10.1073/pnas.1007773107 20829492 29. Gil-Humanes J Pist\\u00f3n F Altamirano-Fortoul R Real A Comino I Reduced-gliadin wheat bread: an alternative to the gluten-free diet for consumers suffering gluten-related pathologies PLoS One 2014 9 e90898 10.1371/journal.pone.0090898 24621595 30. Becker D Wieser H Koehler P Folck A M\\u00fchling KH Z\\u00f6rb C Protein composition and techno-functional properties of transgenic wheat with reduced alpha-gliadin content obtained by RNA interference J Appl Bot Food Qual 2012 85 23 33 31. Wen S Wen N Pang J Langen G Brew-Appiah RAT Mejias JH Structural genes of wheat and barley 5-methylcytosine DNA glycosylases and their potential applications for human health Proc Natl Acad Sci U S A 2012 109 20543 20548 10.1073/pnas.1217927109 23184965 32. Tanner GJ Blundell MJ Colgrave ML Howitt CA Creation of the first ultra-low gluten barley ( Hordeum vulgare Plant Biotechnology Journal 2016 14 1139 1150 10.1111/pbi.12482 26427614 33. Zhang Y Liang Z Zong Y Wang Y Liu J Chen K Qiu J-L Gao C Efficient and transgene-free genome editing in wheat through transient expression of CRISPR/Cas9 DNA or RNA Nature Communications 2016 7 12617 10.1038/ncomms12617 34. Gil-Humanes J Wang Y Liang Z Shan Q Ozuna CV S\\u00e1nchez-Le\\u00f3n S High-efficiency gene targeting in hexaploid wheat using DNA replicons and CRISPR/Cas9 The Plant Journal 2017 89 1251 1262 10.1111/tpj.13446 27943461 35. Galili G Levy AA Feldman M Gene-dosage compensation of endosperm proteins in hexaploid wheat Triticum aestivum Proceedings of the National Academy of Sciences of the United States of America 1986 83 6524 6528 10.1073/pnas.83.17.6524 16593753 36. Pist\\u00f3n F Gil-Humanes J Rodr\\u00edguez-Quijano M Barro F Down-regulating \\u03b3-gliadins in bread wheat leads to non-specific increases in other gluten proteins and has no major effect on dough gluten strength PLoS One 2011 6 e24754 10.1371/journal.pone.0024754 21935456 37. Shaw LM Turner AS Herry L Griffiths S Laurie DA Mutant alleles of Photoperiod-1 Triticum aestivum PLoS One 2013 8 e79459 10.1371/journal.pone.0079459 24244507 38. Sears E. R. Nullisomic-Tetrasomic Combinations in Hexaploid Wheat Chromosome Manipulations and Plant Genetics 1966 Boston, MA Springer US 29 45 39. Endo TR Induction of chromosomal structural changes by a chromosome of Aegilops cylindrica J Hered 1988 79 366 370 10.1093/oxfordjournals.jhered.a110529 40. Anderson OD Characterization of members of a pseudogene subfamily of the wheat \\u03b1-gliadin storage protein genes Plant Mol Biol 1991 16 335 337 10.1007/BF00020565 1893105 41. Goryunova SV Salentijn EMJ Chikida NN Kochieva EZ Van der Meer IM Gilissen LJWJ Smulders MJM Expansion of the \\u03b3-gliadin gene family in Aegilops Triticum BMC Evol Biol 2012 12 215 10.1186/1471-2148-12-215 23137212 42. Cooke RJ The classification of wheat cultivars using a standard reference electrophoresis method J Natl Inst Agric Bot 1987 17 273 281 43. Van den Broeck HC Van Herpen TWJM Schuit C Salentijn EMJ Dekking L Bosch D Hamer RJ Smulders MJM Gilissen LJWJ Van der Meer IM Removing celiac disease-related gluten proteins from bread wheat while retaining technological properties: a study with Chinese spring deletion lines BMC Plant Biology 2009 9 41 10.1186/1471-2229-9-41 19351412 44. Endo M Mikami M Toki S Multigene knockout utilizing off-target mutations of the CRISPR/Cas9 system in rice Plant Cell Physiol 2015 56 41 47 10.1093/pcp/pcu154 25392068 45. Dong L Huo N Wang Y Deal K Wang D Hu T Dvorak J Anderson OD Luo MC Gu YQ Rapid evolutionary dynamics in a 2.8-Mb chromosomal region containing multiple prolamin and resistance gene families in Aegilops tauschii Plant J 2016 87 495 506 10.1111/tpj.13214 27228577 46. Ma X Zhang Q Zhu Q Liu W Chen Y Qiu R Wang B Yang Z A robust CRISPR/Cas9 system for convenient, high-efficiency multiplex genome editing in monocot and dicot plants Mol Plant 2015 8 1274 1284 10.1016/j.molp.2015.04.007 25917172 47. Wang W, Akhunova A, Chao S, Akhunov E. Optimizing multiplex CRISPR/Cas9-based genome editing for wheat. BioRxiv. 2016. 10.1101/051342. 48. Anderson OD Greene FC The \\u03b1-gliadin gene family. II. DNA and protein sequence variation, subfamily structure, and origins of pseudogenes Theor Appl Genet 1997 95 59 65 10.1007/s001220050532 49. Sollid LM Qiao SW Anderson RP Gianfrani C Koning F Nomenclature and listing of celiac disease relevant gluten T-cell epitopes restricted by HLA-DQ molecules Immunogenetics. 2012 64 455 460 10.1007/s00251-012-0599-z 22322673 50. Jouanin Aur\\u00e9lie Borm Theo Boyd Lesley A. Cockram James Leigh Fiona Santos Bruno A.C.M. Visser Richard G.F. Smulders Marinus J.M. Development of the GlutEnSeq capture system for sequencing gluten gene families in hexaploid bread wheat with deletions or mutations induced by \\u03b3-irradiation or CRISPR/Cas9 Journal of Cereal Science 2019 88 157 166 10.1016/j.jcs.2019.04.008 51. Perochon A Jianguang J Kahla A Arunachalam C Scofield S Bowden B Wallington E Doohan F TaFROG Fusarium graminearum Plant Physiol 2015 169 2895 2906 26508775 52. Segovia V Hubbard A Craze M Bowden S Wallington EJ Bryant R Greenland A Bayles R Uauy C Yr36 Puccinia striiformis Phytopathology 2014 104 871 878 10.1094/PHYTO-10-13-0295-R 24601983 53. Howells RM Craze M Bowden S Wallington EJ Efficient generation of stable, heritable gene edits in wheat using CRISPR/Cas9 BMC Plant Biol 2018 18 215 10.1186/s12870-018-1433-z 30285624 54. Milner MJ Howells RM Craze M Bowden S Graham N Wallington EJ A PSTOL-like gene in wheat, TaPSTOL BMC Plant Biology 2018 18 115 10.1186/s12870-018-1331-4 29884124 55. Endo TR Gill BS The deletion stocks of common wheat The Journal of Heredity 1996 87 295 307 10.1093/oxfordjournals.jhered.a023003 56. Shan Q Wang Y Li J Zhang Y Chen K Liang Z Zhang K Liu J Xi JJ Qiu J-L Gao C Targeted genome modification of crop plants using a CRISPR-Cas system Nature Biotechnology 2013 31 686 688 10.1038/nbt.2650\",\n          \"Front Cell Dev Biol Front Cell Dev Biol Front. Cell Dev. Biol. Frontiers in Cell and Developmental Biology 2296-634X Frontiers Media S.A. 7872100 10.3389/fcell.2020.618552 Cell and Developmental Biology Original Research A Semi-automated Organoid Screening Method Demonstrates Epigenetic Control of Intestinal Epithelial Differentiation Ostrop Jenny 1 * Zwiggelaar Rosalie T. 1 Terndrup Pedersen Marianne 2 3 Gerbe Fran\\u00e7ois 4 B\\u00f6sl Korbinian 5 Lindholm H\\u00e5vard T. 1 D\\u00edez-S\\u00e1nchez Alberto 1 Parmar Naveen 1 Radetzki Silke 6 von Kries Jens Peter 6 Jay Philippe 4 Jensen Kim B. 2 3 Arrowsmith Cheryl 7 8 9 Oudhoff Menno J. 1 * 1 Centre of Molecular Inflammation Research (CEMIR), Department of Clinical and Molecular Medicine (IKOM), NTNU - Norwegian University of Science and Technology Trondheim Norway 2 BRIC - Biotech Research and Innovation Centre, University of Copenhagen Copenhagen Denmark 3 Novo Nordisk Foundation Center for Stem Cell Biology, Faculty of Health and Medical Sciences, University of Copenhagen Copenhagen Denmark 4 Cancer Biology Department, Institute of Functional Genomics, University of Montpellier Montpellier France 5 Department of Bioinformatics, Computational Biological Unit, University of Bergen Bergen Norway 6 Screening Unit, Leibniz-Forschungsinstitut f\\u00fcr Molekulare Pharmakologie Berlin Germany 7 Structural Genomics Consortium, University of Toronto Toronto, ON Canada 8 Princess Margaret Cancer Centre, University Health Network Toronto, ON Canada 9 Department of Medical Biophysics, University of Toronto Toronto, ON Canada Edited by: Trygve Tollefsbol, University of Alabama at Birmingham, United States Reviewed by: Vanesa Muncan, University of Amsterdam, Netherlands; Charles Giardina, University of Connecticut, United States *Correspondence: Jenny Ostrop jenny.ostrop@ntnu.no Menno J. Oudhoff menno.oudhoff@ntnu.no This article was submitted to Epigenomics and Epigenetics, a section of the journal Frontiers in Cell and Developmental Biology 21 1 2021 2020 8 618552 19 10 2020 16 12 2020 Copyright \\u00a9 2021 Ostrop, Zwiggelaar, Terndrup Pedersen, Gerbe, B\\u00f6sl, Lindholm, D\\u00edez-S\\u00e1nchez, Parmar, Radetzki, von Kries, Jay, Jensen, Arrowsmith and Oudhoff. 2021 Ostrop, Zwiggelaar, Terndrup Pedersen, Gerbe, B\\u00f6sl, Lindholm, D\\u00edez-S\\u00e1nchez, Parmar, Radetzki, von Kries, Jay, Jensen, Arrowsmith and Oudhoff This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. Intestinal organoids are an excellent model to study epithelial biology. Yet, the selection of analytical tools to accurately quantify heterogeneous organoid cultures remains limited. Here, we developed a semi-automated organoid screening method, which we applied to a library of highly specific chemical probes to identify epigenetic regulators of intestinal epithelial biology. The role of epigenetic modifiers in adult stem cell systems, such as the intestinal epithelium, is still undefined. Based on this resource dataset, we identified several targets that affected epithelial cell differentiation, including HDACs, EP300/CREBBP, LSD1, and type I PRMTs, which were verified by complementary methods. For example, we show that inhibiting type I PRMTs, which leads enhanced epithelial differentiation, blocks the growth of adenoma but not normal organoid cultures. Thus, epigenetic probes are powerful tools to study intestinal epithelial biology and may have therapeutic potential. organoids epigenetic modifiers intestinal stem cell biology bioimage analysis PRMT1 EP300 CREBBP Introduction The intestinal epithelium, a single layer of cells, faces the challenge of both providing a barrier against pathogens while also being responsible for the uptake of nutrients and water. One of the hallmarks of intestinal epithelium is the rapid turnover of 3\\u20135 days, which is driven by LGR5+ intestinal stem cells (ISCs) that reside at the bottom of crypts. ISCs are continuously dividing and give rise to progenitor cells, which differentiate into specialized intestinal epithelial cell (IEC) lineages such as absorptive enterocytes and secretory lineages such as mucus-producing goblet cells, antimicrobial-producing Paneth cells, hormone-secreting enteroendocrine cells, and chemosensory tuft cells (Gehart and Clevers, 2019 2017 2017 The balance between ISC proliferation and IEC differentiation is controlled by pathways including WNT, BMP, and NOTCH (Gehart and Clevers, 2019 2001 2017 2019 2019 2017 2015 2015 2014 2013 2014 2017 2017 2012 2016b 2016 2019 2017 2019 Heterogeneous organoid cultures are quite sensitive to subtle changes in handling and culture conditions. Therefore, development of quantitative analysis methods for reproducible quantification of a whole organoid population instead of relying on representative example data points is crucial (Huch et al., 2017 2019 2017 2018 2020 Here, we provide a semi-automated organoid quantification method suitable for screening experiments and designed to be used in laboratories with a standard infrastructure. To widely investigate the role of epigenetic modifiers for adult intestinal epithelial homeostasis, we combined this toolbox with a chemical probe library consisting of 39 inhibitors that target epigenetic modification enzymes with high selectivity and specificity (Scheer et al., 2019 Results Development of a Toolbox to Quantify Intestinal Organoid Growth and Cellular Composition A decade after its establishment by Sato et al. ( 2009 Figure 1A Supplementary Figure 1a Figure 1B Figure 1C Supplementary Figures 1b,c Figure 1 Quantification of intestinal organoid growth and cellular composition. (A) Supplementary Figure 1a (B) Supplementary Figures 1b,c (C) (D) (E) hi Supplementary Figures 1d,e (F) 2014 (G) hi Supplementary Figures 1d,f\\u2013h In addition to determining organoid size, the cellular composition is of critical interest. We therefore selected transcripts that are specific for individual IEC lineages (Haber et al., 2017 Figure 1D Alpi Figure 1D 2019 Figure 1E Supplementary Figures 1d,e Figure 1E Ulex europaeus bright Figure 1E 2014 2014 Figure 1F Figure 1F Figure 1G Supplementary Figures 1d,f,g hi 2011 Lyz1 Figure 1F Supplementary Figure 1h Organoid Screen of Epigenetic Modifier Inhibitors Identifies Established Drugs Targeting Cancer Growth Next, we applied our organoid toolbox for screening of a chemical probe library that targets epigenetic modifiers to modulate the epigenome (Scheer et al., 2019 Figures 2A,B 2017 Figure 2A Figures 2C,D Supplementary Figures 2a\\u2013c 3A Lgr5 Figure 2E Lgr5 Figures 2C\\u2013E Supplementary Figures 2b,c 2004 2007 2019 Lgr5 2016a Lgr5 Lgr5 Figure 2F Supplementary Figure 2d Apc Figure 2G Figure 2 Organoid screen of epigenetics probes identifies established cancer drugs. (A) (B) Supplementary Table 1 (C) Supplementary Figure 2a (D) (E) Lgr5 (F) Lgr5 hi t Supplementary Figure 2d (G) Apc Figure 3 Inhibition of EP300/CREBBP enhances organoid size and Lgr5 (A) (B) Supplementary Figure 3c (C) Lgr5 hi t Supplementary Figure 3d (D) (E) 2012 Inhibition of EP300/CREBBP Enhances Organoid Size and Lgr5 We next focused on probes that increased organoid size ( Figures 2B\\u2013D Supplementary Figure 3a Figure 3A Supplementary Figure 3b 2000 2010 2018 2014 2015 2017 Figure 3B Supplementary Figure 3c Lgr5 Figure 3C Supplementary Figures 3d,e Supplementary Figures 3f,g Figure 3D Supplementary Figure 3h Lgr5 Figure 3D 2012 Figure 3E Egr1 Figure 3D 2009 Muc4 Ccl6 Figures 3B,D Supplementary Figure 3i Supplementary Figures 3j,k Lgr5, Egr1 GSK-LSD1 Broadly Affects IEC Composition So far, we have used organoid size as a probe selection criteria. Additionally, we performed qRT-PCR on 12 genes associated with specific cell lineages (Haber et al., 2017 Figure 4A Supplementary Figures 4a,f\\u2013k Supplementary Figures 4b,c Supplementary Figures 4d,e Gfi1b Figure 4B 2020 2020 hi 2011 Lyz1 Defa22 Figures 4B,C lo hi Figure 4C Villin Lsd1 f/f Lsd1 Figure 4D 2011 Neurod1 Neurog3 Villin Lsd1 f/f 2020 hi Lsd1 Figure 4E Villin Lsd1 f/f Figure 4F Supplementary Figure 4l in vivo Figure 4 GSK-LSD1 broadly affects IEC composition. (A) Defa22 Defa22 Supplementary Figure 4a (B) (C) hi lo hi t (D) Villin Lsd1 fl/fl Lsd1 hi lo hi (E) (F) t BET Inhibition Reduces Relative Abundance of Tuft Cells Secretory cell lineage differentiation, such as goblet and Paneth cells, is well-studied and is generally thought to involve NOTCH-mediated lateral inhibition. Tuft cell differentiation, however, is less defined. Therefore, we next focused on the BRD/BET inhibitors (+)-JQ1 and bromosporine in our marker gene expression dataset ( Figure 4A Dclk1, Trpm5 Gfi1b Figure 5A Supplementary Figure 5a Figure 4A Supplementary Figure 4k Supplementary Figures 2a 5B Figure 5B 2014 Hpgds2 Figures 5B,C Supplementary Figure 5c Supplementary Figure 5d Figure 5 BET inhibition reduces relative abundance of tuft cells. (A) (B) Hpgds (C) Hpgds Inhibition of Type I PRMTs Results in Higher Relative Abundance of Secretory Cells and Prevents Growth of Tumor Organoids So far, we focused on inhibitors that caused reduced IEC differentiation. However, two probes stood out because they increased the expression of genes associated with Paneth-, goblet-, and enteroendocrine cells ( Figure 4A Figures 2C\\u2013E 2016 Figure 6A Supplementary Figure 6a Lgr5 Lgr5 Supplementary Figures 6b\\u2013d hi Figure 6B Supplementary Figure 4c Figure 6C Neurog3 Figure 6D Figure 6E Figure 6F Figure 6F Supplementary Figures 6e,f 2019 Supplementary Figures 6e,g Supplementary Figure 6h 2008 2019 Prmt1 2017 Supplementary Figure 6i Prmt1 Apc Figure 6G Supplementary Figure 6j Figure 6H Supplementary Figures 6k,l Figure 6 Inhibition of type I PRMTs leads to more mature organoids and prevents adenoma growth. (A) (B) hi Supplementary Figure 4c (C) t (D) Neurog3 t (E) (F) 2017 (G) Apc Supplementary Figure 6j (H) Supplementary Figure 6k Discussion Working with heterogeneous organoid cultures is challenging with respect to reproducibility and quantification. Our toolkit, which we present in this article enables reproducible results across biological replicates using standard equipment and is suitable for screening setups. We established a quantification workflow that is based on the open source tools ImageJ and Ilastik, which is a simple yet robust alternative to recent stand-alone software options (Borten et al., 2018 2019 Probes targeting EP300/CREBBP were designed as cancer therapeutics (Attar and Kurdistani, 2017 Figure 3 2000 2010 2013 2018 2018 Figure 3D Ctnnb1 Supplementary Figure 3h 2018 plt6 2013 2010 2018 2014 2017 1997 2009 2019 2019 We recently demonstrated a central role of LSD1 in Paneth and goblet cell differentiation and maturation (Parmar et al., 2020 2020 high low Figure 4 Figure 5 2016 2016 in vivo 2014 2016 2016 2019 Type I PRMT inhibition with MS023 led to a more differentiated intestinal epithelium without major loss of LGR5+ stem cells ( Figure 6 Supplementary Figure 6c Supplementary Figures 6h,i Xenopus 2010 2011 2017 2017 Figures 6A\\u2013F 2005 2020 2016 2020 2019 2019 2018 Apc Figures 6G,H Prmt1 2019 Villin Apc min Highly permissive chromatin and transcriptional control of IEC fate, as well as gene regulation by differential chromatin states, have been discussed as opposing models of intestinal epithelial differentiation regulation (Elliott and Kaestner, 2015 Figure 2 2020 2005 Sox9 2007 2019 To summarize, we developed a resource that allows to compare the requirement of various epigenetic modifiers for intestinal epithelial renewal and IEC differentiation. Our results indicate that some epigenetic modifiers with the capacity to both mediate histone modifications and act as transcriptional co-regulators control the balance between an undifferentiated/differentiated epithelial state. Thereby, they lay basis for a fine-tuned transcriptional regulation and rapid adjustment upon injury or pathogenic challenge. Methods Epigenetic Modifier Inhibitors The epigenetic modifier inhibitors in the screen experiment were part of the Structural Genomics Consortium Epigenetic Chemical Probes Collection as of March 2016. Probes were reconstituted in DMSO and used at the recommended concentration as listed in Supplementary Table 1 Mice C57BL/6JRj wild type (Janvier labs), Lgr5 Villin 2004 Lsd1 f/f 2013 Apc 15lox Hpgds 2018 Neurog3 2015 Small Intestinal Crypt Isolation Small intestinal crypts were isolated as described previously (Sato and Clevers, 2012 Organoid Culture Organoids were generated by seeding ca. 250\\u2013500 small intestinal crypts in a 50 \\u03bcl droplet of cold Matrigel (Corning #734-1101) into the middle of a pre-warmed 24-well plate. Matrigel was solidified by incubation at 37\\u00b0C for 5\\u201315 min and 500 \\u03bcl culture medium added. Basal culture medium (\\u201cENR\\u201d) consisted of advanced DMEM F12 (Gibco) supplemented with 1x Penicillin-Streptomycin (Sigma-Aldrich), 10 mM HEPES, 2 mM Glutamax, 1x B-27 supplement, 1x N2 supplement, (all Gibco) 500 mM N-Acetylcysteine (Sigma-Aldrich), 50 ng/ml recombinant EGF (Thermo Fisher Scientific), 10% conditioned medium from a cell line producing Noggin (kind gift from Hans Clevers), and 20% conditioned medium from a cell line producing R-Spondin-1 (kind gift from Calvin Kuo). ENR culture medium was replaced every 2\\u20133 days. Organoids were passaged at 1:3\\u20131:4 ratio by disruption with rigorous pipetting almost to single cells. Organoid fragments were centrifuged at 300 \\u00d7 g, resuspended in 40\\u201350 \\u03bcl cold Matrigel per well, and plated on pre-warmed 24-well plates. Organoids derived from different mice or a repetition at least one passage apart are considered biological replicates. Technical replicates, i.e., separate wells, were carried out in some experiments and were pooled for analysis. Altering IEC Lineage Composition in Organoids Protocols to alter the IEC composition in organoids have been described previously (Yin et al., 2014 2017 2018 Organoid Screen With Epigenetic Chemical Probes Library Organoids of four biological replicates were passaged to nearly single cells at 1:4 ratio as described above and seeded in 40 \\u03bcl Matrigel droplets in 24-well plates. 250 \\u03bcl/well ENR were added immediately after solidification and 250 \\u03bcl/well ENR + probes at 2x working concentration (see Supplementary Table 1 Reporter Organoids Lgr5 Lgr5 Lgr5 Hpgds Hpgds Hpgds Neurog3 Neurog3 Neurog3 Modified Organoid Growth Conditions With EP300/CREBBP Inhibition Lgr5 Splitting of Organoids After Type I PRMT Inhibition Organoids were treated with DMSO or MS023 for 96 h, passaged to nearly single cells as described above, and cultured in ENR for additional 96 h. Media was replaced every 48 h. Generation of APC-Deficient Adenomas Eight week old Apc 15lox Lgr5 Organoid Growth Quantification Organoid bright-field images were acquired on an EVOS2 microscope (Thermo Fisher Scientific) with 2x magnification. At the starting point of the experiment, for each plate an automation setup was generated to acquire z-stacks with 50 \\u03bcm spacing either of a single position or 2\\u20134 tiled images covering height and most area of the Matrigel dome for each well. This automation setup was reused at consecutive timepoints. A custom ImageJ/Fiji v1.52n (Schindelin et al., 2012 2015 2019 RNA Isolation, Quantitative RT-PCR, and Analysis To harvest RNA, organoids in the Matrigel dome were dissolved in 250 \\u03bcl RNA-solv reagent (Omega Bio-Tek). RNA was isolated using Direct-zol-96 RNA or Direct-zol MiniPrep kit (Zymo Research) according to the manufacturer's instructions, including DNAse digestion. cDNA was transcribed using High-Capacity RNA-to-cDNA Kit (Applied Biosystems) according to the manufacturer's instructions. RNA quality and concentration was assessed on an NanoDrop-1000 instrument (NanoDrop). Samples were handled in 96-well plates and transferred with multichannel pipettes. qRT-PCR was carried out in technical duplicates in 384-well plates on a QuantStudio 5 instrument (Thermo Fisher Scientific) using 2x Perfecta ROX,UNG Fast Mix (Quanta Biosciences) and 5 ng cDNA per reaction in a total volume of 12 \\u03bcl. Primer-probe combinations were selected based on the Universal Probe Library System (Roche) and are listed in Supplementary Table 2 Hprt \\u0394\\u0394CT Defa22 Supplementary Material Flow Cytometry To obtain single cells, Matrigel in 1\\u20133 wells was disrupted by pipetting, well content was transferred to an Eppendorf tube, centrifuged at 300\\u00d7 g, and supernatant removed. Then, organoids were incubated with 300 \\u03bcl TrypLE express (Thermo Fisher Scientific) for 37\\u00b0C for 30 min and pipetted up/down with a 1,000 \\u03bcl pipet tip prior to and after the incubation. Single cells were stained with Zombie Aqua (Biolegend, 1:1,000 in PBS) for 15 min at room temperature (RT) for live-dead exclusion. If DAPI instead of Zombie Aqua staining was used for live-dead exclusion, it was added it during the last washing step (1:1,000). Samples were incubated with antibody conjugates against CD326-BV605, CD24-PerCp-Cy5.5 or AF647, CD44-BV785, CD117-PE-Cy7 [all Biolegend, see Supplementary Table 3 2018 Flow Cytometry Screening For the flow cytometry screening experiment, organoids were grown in 96 well glass-bottom plates (Cellvis) that were pre-cooled and held on ice during seeding. Organoid fragments in Matrigel (50 \\u03bcl/well) were distributed in pre-cooled plates with an automated pipette, then plates were transferred to a rotary plate shaker for 30 s at 150 rpm, before the Matrigel was solidified at 37\\u00b0C. With help of a Viaflo 96-channel pipette (Integra Biosciences) 200 \\u03bcl/well ENR without or with inhibitors were added of which 100 \\u03bcl were replaced daily during the 96 h time course. To obtain single cells, culture media was removed, 100 \\u03bcl/well TrypLE express (Thermo Fisher Scientific) added and Matrigel disrupted by repeated pipetting with a multichannel pipette. Staining with Zombie Aqua, CD326-BV421, CD24 -PerCp-Cy5.5, CD44-AF647, CD117-PE-Cy7 (all Biolegend, see Supplementary Table 3 2018 Confocal Microscopy and Quantification For immunofluorescence staining, organoids were grown in 30 \\u03bcl/well Matrigel droplets in a 8-well microscopy chamber (Ibidi) that was pre-warmed for seeding. After 96 h incubation, the organoids were fixed in 4% paraformaldehyde and 2% sucrose for 30 min at RT, washed, and permeabilized with 0.2% Triton-X100 in PBS. Free aldehyde groups were blocked using 100 mM glycine, followed by blocking buffer (1% BSA, 2% NGS diluted in 0.2% Triton-X100 in PBS) for 1 h at RT. The organoids were incubated overnight at 4\\u00b0C with a primary antibody against KI67 (Invitrogen #MA5-14520; 1:200) or MUC2 (Santa Cruz #sc-15334; 1:200) in blocking buffer, followed by three washes with slight agitation. Next, the organoids were incubated with Goat anti-Rabbit IgG-AF488 (Invitrogen, 1:500), UEA1-Rhodamine (Vector Laboratories #RL-1062-2, 2 \\u03bcg/ml), and Hoechst 33342 overnight at 4\\u00b0C. After washing, the organoids were mounted using Fluoromount G (Thermo Fisher Scientific), and visualized using a LSM880 confocal microscope (Zeiss). UEA1/MUC2-positive cells were manually counted for \\u22655 organoids per biological replicate in a middle plane of a z-stack. Cell numbers are reported relative to the area of the z-stack projection of each organoid, determined by nuclear staining. Immunohistochemistry and Immunofluorescence Staining of Paraffin-Embedded Tissue and Quantification Immediately after euthanizing mice, the intestinal tissues were removed, washed with PBS, fixed in 4% formaldehyde for 48\\u201372 h at RT, and embedded in paraffin wax. Staining was carried out on 4 \\u03bcm paraffin sections. The sections were rehydrated and treated with 3% hydrogen peroxide for 10 min at RT. Antigens were retrieved by boiling the slides in citrate buffer (pH6) in a microwave for 15 min. For immunohistochemistry staining of duodenum sections, the sections were incubated overnight at 4\\u00b0C with anti-DCLK1 (Abcam #ab31704; 1:1,500) in TBS + 0.025% Tween 20 + 1% BSA. Specific binding was detected with Envision-HRP (Dako) and DAB (Dako) and images were acquired on a EVOS2 microscope (Thermo Fisher Scientific) with 10x magnification. DCLK1+ cells were quantified for \\u226530 crypt-villus pairs per mouse. Representative images were acquired on a Eclipse Ci-L microscope (Nikon) with 20x magnification. For immunofluorescence staining of colon sections, slides were blocked with PBS + 1% BSA + 2% goat serum + 0.2% Triton X-100 for 1h at RT and incubated overnight at 4\\u00b0C with anti-DCLK1 antibody (Abcam #ab31704; 1:250) in PBS + 1% BSA +1% goat serum + 0.05% Tween 20. Specific binding was detected with Goat anti-Rabbit IgG-AF488 (Invitrogen, 1:1,000) for 1h at 37\\u00b0C while nuclei were stained with DAPI (1:1,000). Slides were mounted with Fluoromount G (Thermo Fisher Scientific) and images were acquired on a LSM880 confocal microscope (Zeiss) with 20x magnification. DCLK1+ cells were quantified for \\u226550 crypts per mouse. mRNA Sequencing Organoid RNA was isolated and treated with DNAse with Quick-RNA Micro prep kit (Zymo Research) according to manufacturer's instructions. RNA integrity numbers were found to be \\u22657. For the I-CBP112 inhibitor study, library preparation was done using the Illumina TruSeq Stranded protocol. Library concentrations were quantified with the Qubit Fluorometric Quantitation system (Life Technologies) and the size distribution was assessed using a 2100 Bioanalyzer automated electrophoresis system (Agilent). For sequencing, samples were diluted and pooled into NGS libraries in equimolar amounts and sequenced at 75 bp single-read chemistry on an Illumina NS500 MO flow-cell on a Ilumina NextSeq 500 instrument (Illumina) by the Genomics core facility (GCF, NTNU). For the MS023 study, library preparation was done using the NEB Next Ultra RNA Library Prep Kit with poly(A) mRNA enrichment and samples were sequenced at 150X2 bp paired-end chemistry on a Illumina NovaSeq 6000 instrument by Novogene (UK) Co. mRNA Sequencing Analysis Read quality was assessed using FastQC v0.11.8, reads were aligned with STAR v2.7.3a to the Mus musculus genome build mm10, and MultiQC v1.7 was used to summarize logs from STAR and FastQC (Dobin et al., 2013 2013 2016 2014 2019 2014 2019 2012 2017 2012 2019 2018 2005 2019 Data Processing and Statistical Analysis Data was processed and statistical analysis was carried out with GNU R v3.6.3 using the packages tidyverse and ggpubr (Wickham et al., 2019 Code Availability The ImageJ script used for organoid segmentation is available at: https://github.com/jennyostrop/Fiji_organoid_brightfield_processing https://doi.org/10.5281/zenodo.3951126 Data Availability Statement The Imaging data from the initial screen was deposited to the Image Data Resource (Williams et al., 2017 https://idr.openmicroscopy.org https://doi.org/10.5281/zenodo.4311473 2018 https://www.ebi.ac.uk/biostudies 2019 https://www.ebi.ac.uk/arrayexpress Ethics Statement The animal study was reviewed and approved by the Norwegian Food Safety Authority (FOTS ID: 15888). Author Contributions JO and MO designed the study. JO, RZ, MT, FG, HL, AD-S, NP, SR, and MO performed the experiments. JO, RZ, MT, FG, KB, and HL analyzed the data. JO, KB, and HL curated the data. CA provided critical materials. PJ, KJ, CA, and MO supervised or provided critical insight. JO, RZ, and MO wrote the manuscript with subsequent input from other authors. All authors contributed to the article and approved the submitted version. Conflict of Interest The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. We thank the imaging (CMIC), animal care (CoMed), and genomics core (GCF) facilities at NTNU for assistance in this work. We thank Dr. Sneha Pinto for critical reading of the manuscript. This manuscript has been released as a pre-print at BioRxiv (Ostrop et al., 2020 Funding. Supplementary Material The Supplementary Material for this article can be found online at: https://www.frontiersin.org/articles/10.3389/fcell.2020.618552/full#supplementary-material Supplementary Figure 1 (a) Figure 1C Figure 1A (b) (c) (d) (e) hi (f) hi (g) (h) hi Click here for additional data file. Supplementary Figure 2 (a) (b) Lgr5 t (c) Lgr5 t (d) Lgr5 t Click here for additional data file. Supplementary Figure 3 (a) Lgr5 t (b) t (c) Figure 2B (d) Lgr5 t (e) Lgr5 (f) (g) (h) p (i) (j) (k) Click here for additional data file. Supplementary Figure 4 (a) Defa22 Figure 4A (b) (c) hi (d) (e) (f\\u2013k) Alpi, Lgr5, Lyz1, Defa22, Muc2, Agr2, Chga, Chgb, Cpe, Dclk1, Trpm5 Gfi1b t p (l) f f Lsd1 t Click here for additional data file. Supplementary Figure 5 (a) (b) (c) 2020 (d) Click here for additional data file. Supplementary Figure 6 (a) (b) Lgr5 t (c) Lgr5 t (d) (e) (f) (g) (h) 2020 (i) (j) Apc (k) (l) Click here for additional data file. Supplementary Table 1 Inhibitors and concentrations used. Click here for additional data file. Supplementary Table 2 qRT-PCR primers and probes used. Click here for additional data file. Supplementary Table 3 Materials, reagents, and software used. Click here for additional data file. References Ackloo S. Brown P. J. M\\u00fcller S. 2017 Chemical probes targeting epigenetic proteins: applications beyond oncology Epigenetics 12 378 400 10.1080/15592294.2017.1279371 28080202 Altan B. Yokobori T. Ide M. Mochiki E. Toyomasu Y. Kogure N. 2016 Nuclear PRMT1 expression is associated with poor prognosis and chemosensitivity in gastric cancer patients Gastr. Cancer 19 789 797 10.1007/s10120-015-0551-7 \\u00c1lvarez-Errico D. Vento-Tormo R. Sieweke M. Ballestar E. 2015 Epigenetic control of myeloid cell differentiation, identity and function Nat. Rev. Immunol 15 7 17 10.1038/nri3777 25534619 Arrowsmith C. H. Bountra C. Fish P. V. Lee K. Schapira M. 2012 Epigenetic protein families: a new frontier for drug discovery Nat. Rev. Drug Discov 11 384 400 10.1038/nrd3674 22498752 Athar A. F\\u00fcllgrabe A. George N. Iqbal H. Huerta L. Ali A. 2019 ArrayExpress update \\u2013from bulk to single-cell expression data Nucl. Acids Res 47 D711 D715 10.1093/nar/gky964 30357387 Atlasi Y. Stunnenberg H. G. 2017 The interplay of epigenetic marks during stem cell differentiation and development Nat. Rev. Genet 18 643 658 10.1038/nrg.2017.57 28804139 Attar N. Kurdistani S. K. 2017 Exploitation of EP300 and CREBBP lysine acetyltransferases by cancer Cold Spring Harb. Perspect. Med 7 10.1101/cshperspect.a026534 27881443 Bao X. Siprashvili Z. Zarnegar B. J. Shenoy R. M. Rios E. J. Nady N. 2017 CSNK1a1 regulates PRMT1 to maintain the progenitor state in self-renewing somatic tissue Dev. Cell 43 227 239.e5 10.1016/j.devcel.2017.08.021 28943242 Bastide P. Darido C. Pannequin J. Kist R. Robine S. Marty-Double C. 2007 Sox9 regulates cell proliferation and is required for Paneth cell differentiation in the intestinal epithelium J. Cell Biol 178 635 648 10.1083/jcb.200704152 17698607 Beckers T. Burkhardt C. Wieland H. Gimmnich P. Ciossek T. Maier T. 2007 Distinct pharmacological properties of second generation HDAC inhibitors with the benzamide or hydroxamate head group Int. J. Cancer 121 1138 1148 10.1002/ijc.22751 17455259 Berg S. Kutra D. Kroeger T. Straehle C. N. Kausler B. X. Haubold C. 2019 Ilastik: Interactive machine learning for (bio)image analysis Nat. Methods 16 1226 1232 10.1038/s41592-019-0582-9 31570887 Biton M. Haber A. L. Rogel N. Burgin G. Beyaz S. Schnell A. 2018 T helper cell cytokines modulate intestinal stem cell renewal and differentiation Cell 175 1307 1320.e22 10.1016/j.cell.2018.10.008 30392957 Bolden J. E. Tasdemir N. Dow L. E. van Es J. H. Wilkinson J. E. Zhao Z. 2014 Inducible in vivo Cell Rep 8 1919 1929 10.1016/j.celrep.2014.08.025 25242322 Bornstein C. Nevo S. Giladi A. Kadouri N. Pouzolles M. Gerbe F. 2018 Single-cell mapping of the thymic stroma identifies IL-25-producing tuft epithelial cells Nature 559 622 626 10.1038/s41586-018-0346-1 30022162 Borten M. A. Bajikar S. S. Sasaki N. Clevers H. Janes K. A. 2018 Automated brightfield morphometry of 3D organoid populations by OrganoSeg Sci. Rep 8 5319 10.1038/s41598-017-18815-8 29593296 Brandenberg N. Hoehnel S. Kuttler F. Homicsko K. Ceroni C. Ringel T. 2020 High-throughput automated organoid culture via stem-cell aggregation in microcavity arrays Nat. Biomed. Eng 1 12 10.1038/s41551-020-0565-2 31937945 Choi S. Jeong H.-J. Kim H. Choi D. Cho S.-C. Seong J. K. 2019 Skeletal muscle-specific Prmt1 deletion causes muscle atrophy via deregulation of the PRMT6-FOXO3 axis Autophagy 15 1069 1081 10.1080/15548627.2019.1569931 30653406 Creyghton M. P. Cheng A. W. Welstead G. G. Kooistra T. Carey B. W. Steine E. J. 2010 Histone H3K27ac separates active from poised enhancers and predicts developmental state Proc. Natl. Acad. Sci. U.S.A 107 21931 21936 10.1073/pnas.1016071107 21106759 Dobin A. Davis C. A. Schlesinger F. Drenkow J. Zaleski C. Jha S. 2013 STAR: Ultrafast universal RNA-seq aligner Bioinformatics 29 15 21 10.1093/bioinformatics/bts635 23104886 Durinck S. Moreau Y. Kasprzyk A. Davis S. De Moor B. Brazma A. 2005 BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis Bioinformatics 21 3439 3440 10.1093/bioinformatics/bti525 16082012 Ebrahimi A. Sevin\\u00e7 K. G\\u00fcrhan Sevin\\u00e7 G. Cribbs A. P. Philpott M. Uyulur F. 2019 Bromodomain inhibition of the coactivators CBP/EP300 facilitate cellular reprogramming Nat. Chem. Biol 15 519 528 10.1038/s41589-019-0264-z 30962627 el Marjou F. Janssen K.-P. Chang B. H.-J. Li M. Hindie V. Chan L. 2004 Tissue-specific and inducible Cre-mediated recombination in the gut epithelium Genesis 39 186 193 10.1002/gene.20042 15282745 Elliott E. N. Kaestner K. H. 2015 Epigenetic regulation of the intestinal epithelium Cell. Mol. Life Sci 72 4139 4156 10.1007/s00018-015-1997-9 26220502 Eram M. S. Shen Y. Szewczyk M. Wu H. Senisterra G. Li F. 2016 A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases ACS Chem. Biol 11 772 781 10.1021/acschembio.5b00839 26598975 Ewels P. Magnusson M. Lundin S. K\\u00e4ller M. 2016 MultiQC: Summarize analysis results for multiple tools and samples in a single report Bioinformatics 32 3047 3048 10.1093/bioinformatics/btw354 27312411 Frankish A. Diekhans M. Ferreira A.-M. Johnson R. Jungreis I. Loveland J. 2019 GENCODE reference annotation for the human and mouse genomes Nucleic Acids Res 47 D766 D773 10.1093/nar/gky955 30357393 Furumatsu T. Tsuda M. Yoshida K. Taniguchi N. Ito T. Hashimoto M. 2005 Sox9 and p300 cooperatively regulate chromatin-mediated transcription J. Biol. Chem 280 35203 35208 10.1074/jbc.M502409200 16109717 Gehart H. Clevers H. 2019 Tales from the crypt: new insights into intestinal stem cells Nat. Rev. Gastroenterol. Hepatol 16 19 34 10.1038/s41575-018-0081-y 30429586 Gerbe F. Sidot E. Smyth D. J. Ohmoto M. Matsumoto I. Dardalhon V. 2016 Intestinal epithelial tuft cells initiate type 2 mucosal immunity to helminth parasites Nature 529 226 230 10.1038/nature16527 26762460 Gitenay D. Baron V. T. 2009 Is EGR1 a potential target for prostate cancer therapy? Future Oncol 5 993 1003 10.2217/fon.09.67 19792968 Gonneaud A. Jones C. Turgeon N. L\\u00e9vesque D. Asselin C. Boudreau F. 2016a A SILAC-based method for quantitative proteomic analysis of intestinal organoids Sci. Rep 6 38195 10.1038/srep38195 27901089 Gonneaud A. Turgeon N. Boudreau F. Perreault N. Rivard N. Asselin C. 2016b Distinct roles for intestinal epithelial cell-specific Hdac1 and Hdac2 in the regulation of murine intestinal homeostasis J. Cell. Physiol 231 436 448 10.1002/jcp.25090 26174178 Goodman R. H. Smolik S. 2000 CBP/p300 in cell growth, transformation, and development Genes Dev 14 1553 1577 10.1101/gad.14.13.1553 10887150 Guccione E. Richard S. 2019 The regulation, functions and clinical relevance of arginine methylation Nat. Rev. Mol. Cell Biol 20 642 657 10.1038/s41580-019-0155-x 31350521 Haber A. L. Biton M. Rogel N. Herbst R. H. Shekhar K. Smillie C. 2017 A single-cell survey of the small intestinal epithelium Nature 551 333 339 10.1038/nature24489 29144463 Han H. Cho J.-W. Lee S. Yun A. Kim H. Bae D. 2018 TRRUST v2: an expanded reference database of human and mouse transcriptional regulatory interactions Nucleic Acids Res 46 D380 D386 10.1093/nar/gkx1013 29087512 Hay D. A. Fedorov O. Martin S. Singleton D. C. Tallant C. Wells C. 2014 Discovery and optimization of small-molecule ligands for the CBP/p300 bromodomains J. Am. Chem. Soc 136 9308 9319 10.1021/ja412434f 24946055 Huang S. Litt M. Felsenfeld G. 2005 Methylation of histone H4 by arginine methyltransferase PRMT1 is essential in vivo Genes Dev 19 1885 1893 10.1101/gad.1333905 16103216 Huch M. Knoblich J. A. Lutolf M. P. Martinez-Arias A. 2017 The hope and the hype of organoid research Development 144 938 941 10.1242/dev.150201 28292837 Huijbregts L. Petersen M. B. K. Berthault C. Hansson M. Aiello V. Rachdi L. 2019 Bromodomain and extra terminal protein inhibitors promote pancreatic endocrine cell fate Diabetes 68 761 773 10.2337/db18-0224 30655386 Ishizuya-Oka A. Shi Y.-B. 2011 Evolutionary insights into postembryonic development of adult intestinal stem cells Cell Biosci 1 37 10.1186/2045-3701-1-37 22087560 Jadhav U. Cavazza A. Banerjee K. K. Xie H. O'Neill N. K. Saenz-Vash V. 2019 Extensive recovery of embryonic enhancer and gene memory stored in hypomethylated enhancer DNA Mol. Cell 74 542 554.e5 10.1016/j.molcel.2019.02.024 30905509 Jadhav U. Saxena M. O'Neill N. K. Saadatpour A. Yuan G.-C. Herbert Z. 2017 Dynamic reorganization of chromatin accessibility signatures during dedifferentiation of secretory precursors into Lgr5+ intestinal stem cells Cell Stem Cell 21 65 77.e5 10.1016/j.stem.2017.05.001 28648363 Jarrold J. Davies C. C. 2019 PRMTs and arginine methylation: cancer's best-kept secret? Trends Mol. Med 25 993 1009 10.1016/j.molmed.2019.05.007 31230909 Kaaij L. T. van de Wetering M. Fang F. Decato B. Molaro A. van de Werken H. J. 2013 DNA methylation dynamics during intestinal stem cell differentiation reveals enhancers driving gene expression in the villus Genome Biol 14 R50 10.1186/gb-2013-14-5-r50 23714178 Kassis T. Hernandez-Gordillo V. Langer R. Griffith L. G. 2019 OrgaQuant: human intestinal organoid localization and quantification using deep convolutional neural networks Sci. Rep 9 12479 10.1038/s41598-019-48874-y 31462669 Kazakevych J. Sayols S. Messner B. Krienke C. Soshnikova N. 2017 Dynamic changes in chromatin states during specification and differentiation of adult intestinal stem cells Nucleic Acids Res 45 5770 5784 10.1093/nar/gkx167 28334816 Kerenyi M. A. Shao Z. Hsu Y.-J. Guo G. Luc S. O'Brien K. 2013 Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation eLife 2 e00633 10.7554/eLife.00633 23795291 Kim T.-H. Li F. Ferreiro-Neira I. Ho L.-L. Luyten A. Nalapareddy K. 2014 Broadly permissive intestinal chromatin underlies lateral inhibition and cell plasticity Nature 506 511 515 10.1038/nature12903 24413398 Kim Y. H. Larsen H. L. Ru\\u00e9 P. Lemaire L. A. Ferrer J. Grapin-Botton A. 2015 Cell cycle-dependent differentiation dynamics balances growth and endocrine differentiation in the pancreas PLoS Biol 13 e1002111 10.1371/journal.pbio.1002111 25786211 Kishida K. Pearce S. C. Yu S. Gao N. Ferraris R. P. 2017 Nutrient sensing by absorptive and secretory progenies of small intestinal stem cells Am. J. Physiol. Gastroint. Liver Physiol 312 G592 G605 10.1152/ajpgi.00416.2016 28336548 Koppens M. A. J. Bounova G. Gargiulo G. Tanger E. Janssen H. Cornelissen-Steijger P. 2016 Deletion of polycomb repressive complex 2 from mouse intestine causes loss of stem cells Gastroenterology 151 684 697.e12 10.1053/j.gastro.2016.06.020 27342214 Kraiczy J. Zilbauer M. 2019 Intestinal epithelial organoids as tools to study epigenetics in gut health and disease Stem Cells Int 2019 7242415 10.1155/2019/7242415 30809264 Lee B.-K. Jang Y. Kim M. LeBlanc L. Rhee C. Lee J. 2019 Super-enhancer-guided mapping of regulatory networks controlling mouse trophoblast stem cells Nat. Commun 10 4749 10.1038/s41467-019-12720-6 31628347 Leggett R. M. Ramirez-Gonzalez R. H. Clavijo B. Waite D. Davey R. P. 2013 Sequencing quality assessment tools to enable data-driven informatics for high throughput genomics Front. Genet 4 288 10.3389/fgene.2013.00288 24381581 Lehman S. M. Chen H. Burgos E. S. Maron M. Gayatri S. Nieves E. 2020 Transcriptomic and proteomic regulation through abundant, dynamic, and independent arginine methylation by Type I and Type II PRMTs bioRxiv [Preprint] 10.1101/2020.06.23.167601 Liao Y. Smyth G. K. Shi W. 2014 featureCounts: an efficient general purpose program for assigning sequence reads to genomic features Bioinformatics 30 923 930 10.1093/bioinformatics/btt656 24227677 Love M. I. Huber W. Anders S. 2014 Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2 Genome Biol 15 550 10.1186/s13059-014-0550-8 25516281 Luu L. Matthews Z. J. Armstrong S. D. Powell P. P. Wileman T. Wastling J. M. 2018 Proteomic profiling of enteroid cultures skewed toward development of specific epithelial lineages Proteomics 18 1800132 10.1002/pmic.201800132 29952134 Mateo J. Lord C. J. Serra V. Tutt A. Balma na J. Castroviejo-Bermejo M. 2019 A decade of clinical development of PARP inhibitors in perspective Ann. Oncol 30 1437 1447 10.1093/annonc/mdz192 31218365 Mathioudaki K. Papadokostopoulou A. Scorilas A. Xynopoulos D. Agnanti N. Talieri M. 2008 The PRMT1 gene expression pattern in colon cancer Br. J. Cancer 99 2094 2099 10.1038/sj.bjc.6604807 19078953 Matsuda H. Shi Y.-B. 2010 An essential and evolutionarily conserved role of protein arginine methyltransferase 1 for adult intestinal stem cells during postembryonic development Stem Cells 28 2073 2083 10.1002/stem.529 20872846 Morgan R. Mortensson E. Williams A. 2018 Targeting LGR5 in colorectal cancer: therapeutic gold or too plastic? Br. J. Cancer 118 1410 1418 10.1038/s41416-018-0118-6 29844449 Mu noz J. Stange D. E. Schepers A. G. van de Wetering M. Koo B.-K. Itzkovitz S. 2012 The Lgr5 intestinal stem cell signature: Robust expression of proposed quiescent '+4' cell markers EMBO J 31 3079 3091 10.1038/emboj.2012.166 22692129 Musiani D. Giambruno R. Massignani E. Ippolito M. R. Maniaci M. Jammula S. 2020 PRMT1 Is recruited via DNA-PK to chromatin where it sustains the senescence-associated secretory phenotype in response to cisplatin Cell Rep 30 1208 1222.e9 10.1016/j.celrep.2019.12.061 31995759 Nakagawa A. Adams C. E. Huang Y. Hamarneh S. R. Liu W. Von Alt K. N. 2016 Selective and reversible suppression of intestinal stem cell differentiation by pharmacological inhibition of BET bromodomains Sci. Rep 6 20390 10.1038/srep20390 26856877 Namwanje M. Liu L. Chan M. Aaron N. Kraakman M. J. Qiang L. 2019 The depot-specific and essential roles of CBP/p300 in regulating adipose plasticity J. Endocrinol 240 257 269 10.1530/JOE-18-0361 30530904 Ortega E. Rengachari S. Ibrahim Z. Hoghoughi N. Gaucher J. Holehouse A. S. 2018 Transcription factor dimerization activates the p300 acetyltransferase Nature 562 538 544 10.1038/s41586-018-0621-1 30323286 Ostrop J. Zwiggelaar R. Pedersen M. T. Gerbe F. B\\u00f6sl K. Lindholm H. T. 2020 A semi-automated organoid screening method demonstrates epigenetic control of intestinal epithelial maturation bioRxiv [Preprint] 10.1101/2020.07.23.217414 Parmar N. Burrows K. Lindholm H. T. Zwiggelaar R. T. Mart\\u00edn-Alonso M. Fosslie M. 2020 Intestinal-epithelial LSD1 controls cytoskeletal-mediated cell identity including goblet cell effector responses required for gut inflammatory and infectious diseases bioRxiv [Preprint] 10.1101/2020.07.09.186114 Picaud S. Fedorov O. Thanasopoulou A. Leonards K. Jones K. Meier J. 2015 Generation of a selective small molecule inhibitor of the CBP/p300 bromodomain for leukemia therapy Cancer Res 75 5106 5119 10.1158/0008-5472.CAN-15-0236 26552700 Puri P. L. Avantaggiati M. L. Balsano C. Sang N. Graessmann A. Giordano A. 1997 P300 is required for MyoD-dependent cell cycle arrest and muscle-specific gene transcription EMBO J 16 369 383 10.1093/emboj/16.2.369 9029156 Raab J. R. Tulasi D. Y. Wager K. E. Morowitz J. M. Magness S. T. Gracz A. D. 2020 Quantitative classification of chromatin dynamics reveals regulators of intestinal stem cell differentiation Development 147 10.1242/dev.181966 31862843 Raisner R. Kharbanda S. Jin L. Jeng E. Chan E. Merchant M. 2018 Enhancer activity requires CBP/P300 bromodomain-dependent histone H3K27 acetylation Cell Rep 24 1722 1729 10.1016/j.celrep.2018.07.041 30110629 Ramos Y. F. M. Hestand M. S. Verlaan M. Krabbendam E. Ariyurek Y. van Galen M. 2010 Genome-wide assessment of differential roles for p300 and CBP in transcription regulation Nucleic Acids Res 38 5396 5408 10.1093/nar/gkq184 20435671 Ray G. Longworth M. S. 2019 Epigenetics, DNA organization, and inflammatory bowel disease. Inflamm Bowel Dis 25 235 247 10.1093/ibd/izy330 30407525 Sampurno S. Bijenhof A. Cheasley D. Xu H. Robine S. Hilton D. 2013 The Myb-p300-CREB axis modulates intestine homeostasis, radiosensitivity and tumorigenesis Cell Death Dis 4 e605 10.1038/cddis.2013.119 23618903 Sarkans U. Gostev M. Athar A. Behrangi E. Melnichuk O. Ali A. 2018 The BioStudies database\\u2014one stop shop for all data supporting a life sciences study Nucleic Acids Res 46 D1266 D1270 10.1093/nar/gkx965 29069414 Sato T. Clevers H. 2012 \\u201cPrimary mouse small intestinal epithelial cell cultures,\\u201d Epithelial Cell Culture Protocols. Methods in Molecular Biology (Methods and Protocols) Vol. 945 Totowa, NJ Humana Press 10.1007/978-1-62703-125-7_19 23097115 Sato T. van Es J. H. Snippert H. J. Stange D. E. Vries R. G. van den Born M. 2011 Paneth cells constitute the niche for Lgr5 stem cells in intestinal crypts Nature 469 415 418 10.1038/nature09637 21113151 Sato T. Vries R. G. Snippert H. J. van de Wetering M. Barker N. Stange D. E. 2009 Single Lgr5 stem cells build crypt-villus structures in vitro without a mesenchymal niche Nature 459 262 265 10.1038/nature07935 19329995 Scheer S. Ackloo S. Medina T. S. Schapira M. Li F. Ward J. A. 2019 A chemical biology toolbox to study protein methyltransferases and epigenetic signaling Nat. Commun 10 1 14 10.1038/s41467-018-07905-4 30602773 Schindelin J. Arganda-Carreras I. Frise E. Kaynig V. Longair M. Pietzsch T. 2012 Fiji: An open-source platform for biological-image analysis Nat. Methods 9 676 682 10.1038/nmeth.2019 22743772 Schindelin J. Rueden C. T. Hiner M. C. Eliceiri K. W. 2015 The ImageJ ecosystem: an open platform for biomedical image analysis Mol. Reprod. Dev 82 518 529 10.1002/mrd.22489 26153368 Serra D. Mayr U. Boni A. Lukonin I. Rempfler M. Challet Meylan L. 2019 Self-organization and symmetry breaking in intestinal organoid development Nature 569 66 72 10.1038/s41586-019-1146-y 31019299 Sheaffer K. L. Kim R. Aoki R. Elliott E. N. Schug J. Burger L. 2014 DNA methylation is required for the control of stem cell differentiation in the small intestine Genes Dev 28 652 664 10.1101/gad.230318.113 24637118 Szklarczyk D. Gable A. L. Lyon D. Junge A. Wyder S. Huerta-Cepas J. 2019 STRING v11: Protein\\u2013protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets Nucleic Acids Res 47 D607 D613 10.1093/nar/gky1131 30476243 The Gene Ontology Consortium 2019 The Gene Ontology resource: 20 years and still GOing strong Nucleic Acids Res 47 D330 D338 10.1093/nar/gky1055 30395331 Van P. Jiang W. Gottardo R. Finak G. 2018 ggCyto: Next generation open-source visualization software for cytometry Bioinformatics 34 3951 3953 10.1093/bioinformatics/bty441 29868771 von Moltke J. Ji M. Liang H.-E. Locksley R. M. 2016 Tuft-cell-derived IL-25 regulates an intestinal ILC2-epithelial response circuit Nature 529 221 225 10.1038/nature16161 26675736 Wang F. Marshall C. B. Ikura M. 2013 Transcriptional/epigenetic regulator CBP/p300 in tumorigenesis: structural and functional versatility in target recognition Cell. Mol. Life Sci 70 3989 4008 10.1007/s00018-012-1254-4 23307074 Weinert B. T. Narita T. Satpathy S. Srinivasan B. Hansen B. K. Sch\\u00f6lz C. 2018 Time-resolved analysis reveals rapid dynamics and broad scope of the CBP/p300 acetylome Cell 174 231 244.e12 10.1016/j.cell.2018.04.033 29804834 Weisberg E. Catley L. Kujawa J. Atadja P. Remiszewski S. Fuerst P. 2004 in vitro in vivo. Leukemia 18 1951 1963 10.1038/sj.leu.2403519 15496979 Wickham H. Averick M. Bryan J. Chang W. McGowan L. D. Fran\\u00e7ois R. 2019 Welcome to the Tidyverse J. Open Source Softw 4 1686 10.21105/joss.01686 Williams E. Moore J. Li S. W. Rustici G. Tarkowska A. Chessel A. 2017 Image data resource: a bioimage data integration and publication platform Nat. Methods 14 775 781 10.1038/nmeth.4326 28775673 Yang Q. Bermingham N. A. Finegold M. J. Zoghbi H. Y. 2001 Requirement of Math1 for secretory cell lineage commitment in the mouse intestine Science 294 2155 2158 10.1126/science.1065718 11739954 Yin X. Farin H. F. van Es J. H. Clevers H. Langer R. Karp J. M. 2014 Niche-independent high-purity cultures of Lgr5 + intestinal stem cells and their progeny Nat. Methods 11 106 112 10.1038/nmeth.2737 24292484 Yu G. Lam T. T.-Y. Zhu H. Guan Y. 2018 Two methods for mapping and visualizing associated data on phylogeny using Ggtree Mol. Biol. Evol 35 3041 3043 10.1093/molbev/msy194 30351396 Yu G. Wang L.-G. Han Y. He Q.-Y. 2012 clusterProfiler: An R package for comparing biological themes among gene clusters OMICS 16 284 287 10.1089/omi.2011.0118 22455463 Zhong X. Jin Y. 2009 Critical roles of coactivator p300 in mouse embryonic stem cell differentiation and nanog expression J. Biol. Chem 284 9168 9175 10.1074/jbc.M805562200 19150979 Zhu A. Ibrahim J. G. Love M. I. 2019 Heavy-tailed prior distributions for sequence count data: Removing the noise and preserving large differences Bioinformatics 35 2084 2092 10.1093/bioinformatics/bty895 30395178 Zwiggelaar R. T. Lindholm H. T. Fosslie M. Pedersen M. T. Ohta Y. D\\u00edez-S\\u00e1nchez A. 2020 LSD1 represses a neonatal/reparative gene program in adult intestinal epithelium Sci. Adv 6 eabc0367 10.1126/sciadv.abc0367 32917713\",\n          \"    Overwinter sea-ice characteristics important for Antarctic krill recruitment in the southwest Atlantic  5 July 2021     Devi Veytia devi.veytia@utas.edu.au  Institute for Marine and Antarctic Studies, University of Tasmania, 20 Castray Esp., Hobart, Tasmania 7000, Australia Institute for Marine and Antarctic Studies University of Tasmania  20 Castray Esp 7000 Hobart Tasmania Australia  Sophie Bestley  Institute for Marine and Antarctic Studies, University of Tasmania, 20 Castray Esp., Hobart, Tasmania 7000, Australia Institute for Marine and Antarctic Studies University of Tasmania  20 Castray Esp 7000 Hobart Tasmania Australia  Australian Antarctic Program Partnership, Institute for Marine and Antarctic Studies, University of Tasmania, Hobart, Tasmania 7004, Australia Australian Antarctic Program Partnership Institute for Marine and Antarctic Studies University of Tasmania  7004 Hobart Tasmania Australia  So Kawaguchi  Australian Antarctic Program Partnership, Institute for Marine and Antarctic Studies, University of Tasmania, Hobart, Tasmania 7004, Australia Australian Antarctic Program Partnership Institute for Marine and Antarctic Studies University of Tasmania  7004 Hobart Tasmania Australia  Australian Antarctic Division, Department of Agriculture, Water and the Environment, 203 Channel Highway, Kingston, Tasmania 7050, Australia Australian Antarctic Division Department of Agriculture, Water and the Environment  203 Channel Highway 7050 Kingston Tasmania Australia  Klaus M Meiners  Australian Antarctic Program Partnership, Institute for Marine and Antarctic Studies, University of Tasmania, Hobart, Tasmania 7004, Australia Australian Antarctic Program Partnership Institute for Marine and Antarctic Studies University of Tasmania  7004 Hobart Tasmania Australia  Australian Antarctic Division, Department of Agriculture, Water and the Environment, 203 Channel Highway, Kingston, Tasmania 7050, Australia Australian Antarctic Division Department of Agriculture, Water and the Environment  203 Channel Highway 7050 Kingston Tasmania Australia  Eugene J Murphy  British Antarctic Survey, High Cross, Madingley Road, Cambridge CB3 0ET, UK British Antarctic Survey  High Cross, Madingley Road CB3 0ET Cambridge UK  Alexander D Fraser  Australian Antarctic Program Partnership, Institute for Marine and Antarctic Studies, University of Tasmania, Hobart, Tasmania 7004, Australia Australian Antarctic Program Partnership Institute for Marine and Antarctic Studies University of Tasmania  7004 Hobart Tasmania Australia  Kazuya Kusahara  Japan Agency for Marine Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa 236-0001, Japan Japan Agency for Marine Earth Science and Technology (JAMSTEC)  236-0001 Yokohama Kanagawa Japan  Noriaki Kimura  Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8568, Japan Atmosphere and Ocean Research Institute The University of Tokyo  5-1-5 Kashiwanoha 277-8568 Kashiwa Japan  Stuart Corney  Institute for Marine and Antarctic Studies, University of Tasmania, 20 Castray Esp., Hobart, Tasmania 7000, Australia Institute for Marine and Antarctic Studies University of Tasmania  20 Castray Esp 7000 Hobart Tasmania Australia Overwinter sea-ice characteristics important for Antarctic krill recruitment in the southwest Atlantic   5 July 2021 4754ED99D33BC85772563AE8A7343854 10.1016/j.ecolind.2021.107934 Received 11 March 2021; Received in revised form 12 June 2021; Accepted 25 June 2021    GROBID - A machine learning software for extracting information from scholarly documents   Climate change Sea ice Antarctic krill Euphausia superba Overwinter survival Recruitment  Climate change alters the extent and structure of sea-ice environments, which affects how they function as a habitat for polar species. Identifying sea-ice characteristics that serve as indicators of habitat quality will be crucial to the monitoring and management of climate change impacts. In the Southern Ocean, Antarctic krill is a key prey species and fishery target. Krill larvae depend upon sea-ice habitats to survive the winter and recruit to the population in spring. Existing observations of sea-ice characteristics lack sufficient spatiotemporal coverage to quantify which ones contribute to favourable overwintering habitat, leading to uncertainties in how current and future changes in sea ice affect krill populations. Here, we derive regional-scale indices of annual krill recruitment spanning 35 years across the southwest Atlantic. To develop meaningful indicators of sea-ice habitat, we selected variables from a high-resolution sea-ice model that are hypothesized as relevant for larval habitat use. The resulting correlations between recruitment and sea-ice indicators vary by region and show remote connections to sea ice that correspond with established theories of larval transport. Through an improved representation of sea-ice habitat quality, as compared with using more traditional satellite-derived variables such as sea-ice extent and duration, we highlight plausible regions of overwintering habitat. Our findings improve current understanding of how krill are likely responding to changing sea ice and support emerging views that larval habitat use is complex. Furthermore, regional variation in larval dependence on sea ice may provide pockets of resilience to change for the broader krill population.  Introduction Sea ice is highly sensitive to global warming with profound consequences for polar species and ecosystems (Maksym, 2019; Meredith et al., 2019) As many management strategies rely on the capacity to quickly diagnose ecosystem state, identifying indicators of sea-ice habitat quality is a priority. Developing these sea-ice indicators would require quantifying how variation in sea-ice characteristics determines habitat quality for dependent species. In the Southern Ocean, Antarctic krill (Euphausia superba, hereafter krill) are an integral part of the marine ecosystem and are dependent on sea ice during their early life stages. They are a primary prey source supporting a diverse assemblage of marine predators including whales, seals, penguins and flying seabirds (Murphy et al., 2007a (Murphy et al., , 2016;; Trathan and Hill, 2016) They also play a role in nutrient cycling (Cavan et al., 2019; Schmidt et al., 2011) (Nicol and Foster, 2016) Environmental variation, including sea-ice variability, drives large fluctuations in krill biomass on regional and interannual scales (Murphy et al., 2007b) (Croxall et al., 1999; Trathan and Hill, 2016) Krill recruitment (i.e. the survival of larvae until they are one year old) is likely a significant driver of interannual krill abundance and population dynamics (Murphy et al., 1998; Quetin and Ross, 2003; Siegel and Loeb, 1995) Quantifying the drivers of krill recruitment variability is therefore of key importance to understanding ecosystem-level impacts of climate variability and change, as well as informing conservation measures in the Southern Ocean. Overwinter survival is thought to constrain krill recruitment (Atkinson et al., 2004; Siegel and Loeb, 1995) Larval krill feed primarily on phytoplankton and cannot survive prolonged periods of starvation (Yoshida et al., 2009) (Arrigo and Thomas, 2004; Boyd, 2002) Sea-ice algae, however, can thrive close to the increased light availability at the top of the water column as well as concentrated nutrients (Arrigo, 2014) Observational studies indicate that krill larvae may have evolved an overwintering strategy reliant on sea-ice habitats. Larvae have been observed scraping sea-ice algae from under-ice surfaces (Hamner et al., 1989) (Daly, 1990; Ju and Harvey, 2004; Kohlbach et al., 2017; Meyer et al., 2002; Schmidt et al., 2014) Caverns created under ridged sea ice may also provide refuge from currents (Daly, 1990; Meyer, 2012; Meyer et al., 2009) These findings were synthesized into the general conceptual model of sea ice functioning as an important foraging substrate and refuge to support overwinter larval survival (Nicol, 2006) Emerging studies find regional plasticity in larval reliance on sea ice. In the Weddell Sea, larval diets were found to be composed of heterotrophic and autotrophic components from both the water column and sea ice, and larval growth was maximized in the dynamic marginal ice zone (MIZ) rather than the interior pack ice as previously thought (Meyer et al., 2017) Results from other diet studies show that larvae can opportunistically feed on dietary sources from the water column (both under the sea ice and in the open ocean) to meet their energetic requirements (Daly, 1990 (Daly, , 2004;; Jia et al., 2016; Walsh et al., 2020) From a mechanistic perspective, local-scale studies indicate that seaice habitat quality may be characterized by a range of variables: thickness and snow cover to optimize the growth of sea-ice bottom algae communities (Jeffery et al., 2020; Meiners et al., 2012; Wongpan et al., 2018) deformed sea ice) to provide refuge, and dynamism, as seen in the MIZ (Massom et al., 2006; Meyer et al., 2017) In addition, at large scales larvae are likely advected as passive drifters over winter. The most recent theory regarding larval overwinter transport is that larvae are advected in association with sea ice during the day and with surface currents at night (Meyer et al., 2017) If sea-ice characteristics are indicators of habitat quality, then recruitment measured in the spring/summer would be related to sea ice along the preceding winter's transport pathways (Kohlbach et al., 2017) The environmental conditions that determine larval overwinter habitat therefore operate over large spatiotemporal scales. However, these theories are based on localscale studies and cannot be confidently extrapolated to define largerscale relationships. While mesoscale sea ice sampling methods using autonomous underwater vehicles or under-ice trawls as well as aerial observations provide an important stepping stone to link local studies to broader scales of inquiry (e.g. Castellani et al., 2020; David et al., 2016; Flores et al., 2012; Schaafsma et al., 2016) Until recently, sea-ice concentration (SIC) was the most widely used remotely sensed sea-ice characteristic, likely due to its good daily temporal coverage and validation. More dynamically driven sea-ice characteristics (such as deformations created from ridging, snow-cover, and swell) are difficult to detect remotely with legacy instruments. Largescale maps of variables such as thickness have been interpolated from networks of in-situ measurements (Worby et al., 2008) However, interpolation cannot provide sufficient accuracy and temporal resolution for the purposes of ecological modelling, and these measurements are time-consuming and costly to collect. While technologies to obtain remote measurements of other sea-ice characteristics such as thickness and snow cover are in their nascent stage, the time series of these measurements is not as long. A shortened record of satellite data can reduce statistical power to detect relationships with krill recruitment. Recruitment indices across larger scales are calculated from net tow data, which are compiled opportunistically using the KRILLBASE databases (Atkinson et al., 2017) This results in a noisy time series that is fragmented in space and time. To detect a signal across a noisy time series, increased statistical power and thus length of time series is needed. These restrictions influence the scope of studies quantifying the effects of changing sea ice on recruitment. Past analyses have been based on SIC and SIC-derived variables such as sea-ice extent (SIE), timing of advance (TOA), duration (DUR) and timing of retreat (TOR) (Stammerjohn et al., 2008; 1 From these, declining SIE (Atkinson et al., 2004; Kawaguchi and Satake, 1994; Mackintosh, 1972; Nicol et al., 2000; Quetin et al., 2007; Siegel and Loeb, 1995) (Pi\\u00f1ones and Fedorov, 2016; Quetin et al., 2007) (Atkinson et al., 2019; Saba et al., 2014) (Loeb and Santora, 2015) The mechanisms that underpin the relationships between recruitment and these sea-ice variables remain unclear. This gives us low confidence that these relationships will be maintained into a future driven by climate change (Cox et al., 2018; Saba et al., 2014) For example, extrapolating the relationship between krill density and SIE leads to a grim outlook for long-term krill population trajectories as circumpolar SIE is projected to decrease by ~ 30% by 2100 (Maksym, 2019) However, when sea-ice thickness (SIT) and ridging rate (RR) are accounted for in projections, suitable larval habitat may indeed expand in the future (Melbourne-Thomas et al., 2016) To refine our hypotheses of how krill will respond to these variables in a changing climate, we need to quantify relationships between recruitment variability and seaice variables that reflect a more mechanistic understanding of habitat use and quality. Here, we use a sea-ice model to explore relationships between indicators of sea-ice habitat quality and krill recruitment in the southwest Atlantic. This approach is useful for testing hypotheses on environmental drivers of population dynamics within a quantitative framework. Our motivation is for this study to compliment and inform the application of existing and emerging satellite products. To do this, we first develop a time series of regional-scale krill recruitment indices from krill density and length frequency data from net tows. This time series incorporates data spanning 35 years and resolves the entire southwest Atlanticthe population centre for krill (Atkinson et al., 2004 We also derive a suite of modern sea-ice indicators (Table 2 Since such variables are not available from remote sensing with the requisite temporal coverage, we obtain estimates using output from a high-resolution coupled sea ice-ocean model. Maps of temporal correlations are then evaluated between krill recruitment and the sea-ice indicators, and the results compared with those obtained using more traditional sea-ice variables derived from satellite SIC. Through a more sophisticated representation of sea-ice habitat this study represents the first broad-scale analysis that enables a mechanistic understanding to inform the quantification of sea-ice habitat quality. This provides insight into both contemporary and future environmental drivers of krill recruitment, and hence the status of krill populations and the wider Southern Ocean ecosystem. Methods Study area Our study area (130 W to 25 W and 75 S to 50 S, Fig. 1 The southwest Atlantic contains >50% of the circumpolar krill biomass (Atkinson et al., 2009) (Trathan and Hill, 2016) (Nicol and Foster, 2016) This sector also has the best historical spatiotemporal research sampling coverage of net tows (e.g. KRILLBASE; (Atkinson et al., 2017) Krill recruit density Data description: Krill density and length frequency Both krill density (number of krill under 1 m 2 of sea surface) and length frequency (histogram of length measurements from individual krill) data were used to develop regional-scale recruitment indices for this study. Krill density data were retrieved from the KRILLBASE density database (hereafter \\\"the density database\\\", https://www.bas.ac.uk /project/krillbase/), which is a freely-available database comprising net samples of krill over the Southern Ocean. Krill length frequency data were compiled from two sources. The KRILLBASE length frequency database is a similar data rescue project to the density database and comprises krill length frequency measurements, sampled from scientific and commercial hauls. To augment the KRILLBASE length frequency database, we combined its measurements with length frequency measurements from the Palmer LTER study region spanning 2009 -2016 (Palmer Station Antarctica LTER and Steinberg, 2020) This combined dataset will hereafter be referred to as \\\"the length-frequency database\\\". Since these databases are compiled from uncoordinated scientific cruises spanning a long time period , it is difficult to directly compare measurements that may have been taken with varying sampling methods. To minimize the influence of these differences in our analysis, we use the standardized krill density values which adjust measurements to a standard sampling method (Atkinson et al., 2017) To our knowledge no analogous standardization method exists for the length-frequency data, so no correction is applied. Measurements from both databases were subsequently filtered and binned to nine equal-area hexagonal polygon grid (green hexagons, Fig. 1 For more details on data filtering, preparation, and binning, see Appendix B.i. Deriving recruit density Recruit density is calculated by multiplying the mean krill density for each region and sampling season by the corresponding proportion of 1year-old krill in the population (R 1 ) (Atkinson et al., 2019; De la Mare, 1994) Different methods exist to derive R 1 from pooled length frequency data, from classifying any krill measurements with a length between 15 and 30 mm as recruits (Atkinson et al., 2019; Perry et al., 2019) Bhattacharya, 1967; De la Mare, 1994; Macdonald and Pitcher, 1979) We decided to employ a mixture distribution, to recognize that at the regional scale of our analysis growth rates between our regions may vary and generate distinct length frequency distributions. Of the mixture Table 2 Ecological significance and derivation of modern sea-ice indicator variables. All variables were calculated from model estimates of SIC, SIT or RR (daily timesteps). Variable Name Calculation Method Ecological Significance Jul-Aug max RR The maximum daily value of RR, calculated from all the days in July and August of each year. In July-August light remains too low to support photosynthesis, and therefore it is unlikely that sea ice supports a high enough ice algal bottom community growth to sustain larval krill (Meiners et al., 2012; Meyer, 2012, Fig. 5 This is supported by the low larval growth rates observed during midwinter (Supplementary Table 1 Meyer et al., 2017) Rather, sea ice likely functions as a refuge. This would be characterized by over-rafted ice that provides a three-dimensionally complex environment for krill to shelter from predators and currents ( Meyer et al., 2009) Maximum RR is considered a better indicator of over-rafted ice rather than mean RR, as a large ridging event (e.g. a storm) would create the over-rafted structures that would then persist for some time. Jul-Aug meanmode SIT The mean minus the mode values of SIT, calculated from all the days in July and August of each year. This metric serves as an additional approximation of deformed ice. The higher the difference of mean and mode thickness, the more the thickness distribution of ice has a right skew, indicating a higher degree of deformation and presence of thicker ridged ice (Haas, 2004) In a previous study, larvae were found to be abundant under ice with a high mean-mode (termed ice type 1) in both the pack ice and MIZ (Meyer et al., 2017) While in the previous study mean-mode was derived from spatial measurements, in this study the metric is calculated over time. We hypothesize that this approach is still valid, as ice that is highly deformed over time would likely also be deformed spatially. Sep MIZ coverage The percentage of days in September that a grid cell was classified as MIZ using the thresholds identified in the sensitivity analysis. In September, when there is more light to support ice algal growth (Meiners et al., 2012) Some observational evidence indicates that the MIZ, which is estimated to have some of the largest ice algae concentrations (Arrigo et al., 1997) (Brierley et al., 2002; Meyer et al., 2017) Sep sd RR The sd of all the daily values of September RR, calculated for each year. Similarly to the MIZ, ice that supports higher food availability is likely dynamic, as processes such as grinding may release ice algae into the water column, making it more accessible ( Meyer et al., 2017; Walsh et al., 2020) Sep mean SIT The mean of all the daily values of September SIT, calculated for each year. SIT likely has a complex relationship with krill habitat quality. While excessively thick ice would attenuate the light available to sea ice bottom algal communities, maintaining a certain amount of thickness is likely key to optimizing habitat. Both algal bottom communities and larval grazing substrate would be maximized in thicker, deformed ice ( Meiners et al., 2012; Meyer et al., 2017) distribution methods, the Bhattacharya (1967) However, this requires that the user manually selects from the data where these modes occur and this is not entirely objective. We therefore decided to use the number of modes derived using the Bhattacharya (1967) method, along with their corresponding means and standard deviations, to inform a Gaussian mixture distribution (Fig. 2 A resulting mode was determined to be a recruit if the mean was < 33 mm (Siegel, 2000) For each region and sampling season, the R 1 was the sum of the proportions of all modes classified as recruits. Recruit density for each region and sampling season was then calculated by multiplying the corresponding mean krill density and R 1 values. To validate our new approaches to calculating R 1 , we compared the recruit density derived here with published values. Since existing time series of recruit density are pooled over the southwest Atlantic rather than by region (Fig. S1 Atkinson et al., 2019) An approximate comparison was done by fitting a simple log-linear regression to our regional recruit density by sampling season. For this comparison recruit density values to which Palmer LTER data contributed were removed, as these were not included in the Atkinson et al. (2019) Evaluating and summarising sea-ice model output Here, as an initial exploration, we selected a single sea-ice model that has been shown to accurately reproduce key physical sea-ice properties (Kusahara et al., 2018 (Kusahara et al., , 2017 (Kusahara et al., , 2019)) Multi-model ensemble approaches using Intergovernmental Panel on Climate Change (IPCC)-class models (e.g. Veytia et al., 2020) For this study we require a model, providing daily output for a wide range of sea-ice variables, that has been tuned to represent sea-ice processes in the Southern Ocean. Publicly available IPCC-class models provide monthly output for a range of variables for every region of the Earth. Many of these models do not represent Southern Ocean sea ice accurately (Cavanagh et al., 2017) Therefore, our single-model approach is appropriate to address the specific research questions posed in this paper. Data description: sea-ice model We obtained estimates of sea-ice concentration (SIC), sea-ice thickness (SIT) and ridging rate (RR) (i.e. the rate of sea-ice convergence) from a high-resolution coupled ocean-sea ice model (v4.0, Hasumi, 2006) (Kusahara and Hasumi, 2013) The model outputs are the result of a hindcast simulation run from 1979 to 2016, with atmospheric surface boundary conditions from the global atmospheric reanalysis ERA-Interim (Dee et al., 2011) The model uses an irregular grid with a horizontal resolution of 10-20 km in Antarctic coastal margins. Outputs are available in daily timesteps. Previous evaluations conclude that this model can effectively reproduce trends in Antarctic sea-ice extent (SIE) and SIC (Kusahara et al., 2018 (Kusahara et al., , 2017 (Kusahara et al., , 2019)) (Kusahara et al., 2019) The model output was summarised into two sets of sea-ice variables: \\\"classic\\\" and \\\"modern\\\". Classic variables are traditionally calculated from SIC measurements, such as time of arrival (TOA), time of retreat (TOR) and duration (DUR). Our modern variables reflect sea-ice characteristics hypothesized to be advantageous to larvae (Section 2.3.4). Most of these variables (i.e. those that are calculated from SIT or RR) are still new to remote sensing and thus a better temporal coverage can be achieved using estimates from the model. For compatibility with mapping packages, the daily output was remapped onto a regular grid over the study area, with a resolution similar to that of its native grid (14 km), using the lambert azimuthal equal-area projection. Data description: Remotely sensed sea-ice observations To compare the model output to available observations, we accessed output from two gridded satellite products to calculate: (1) the classic variables using daily 25 km SIC observations from the National Snow and Ice Data Centre (NSIDC, Cavalieri et al., 1996, updated yearly) (2) climatologies of SIT for July-August and September using monthly SIT derived from Envisat and Cryosat-2 with a 50 km smoothing radius, distributed by the Archiving, Validation and Interpretation of Satellite Oceanographic (AVISO) data portal (Guerreiro et al., 2017) The AVISO product is available from 2003 and comprises 3 different sampling Where overlap between two sampling methods occurred, the most recent was used. Evaluating model against observations As an additional verification, we performed evaluations comparing modelled output of sea-ice variables with available satellite products. These included an evaluation of modelled SIC and the marginal ice zone (MIZ). The latter is important for our study, as previous work had hypothesized the importance of the MIZ for krill habitat (Meyer et al., 2017) 2 It is therefore crucial that the model represent the MIZ reasonably. A sensitivity analysis (Appendix C.i) enabled us to provide a realistic MIZ representation in terms of area and location (Fig. 3 The model's representation of SIT was also evaluated against remotely sensed SIT (Appendix C.ii), which revealed that the model represented spatial patterns of SIT well. Although modelled SIT was lower than observed, it was close to the lower confidence limit of observed SIT and therefore within a reasonable range. Additionally, inter-instrument differences in the Antarctic are large (Fig. 3 Kacimi and Kwok, 2020) Modern variables: Achieved through model output We developed a suite of modelled sea-ice variables providing a more mechanistic description of krill sea-ice habitat: we term these \\\"modern\\\" variables. These variables are available through use of sea-ice models, and/or reflect new perceptions of krill overwintering habitat. Recent work suggests that SIC, SIT and RR are likely important for describing larval habitat (Meiners et al., 2012; Melbourne-Thomas et al., 2016; Meyer et al., 2017) In particular, the distribution of these variables was identified as important (Meyer et al., 2017) To characterize the distribution of SIC, SIT and RR, the daily model output for each was summarized into modern variables using the summary statistics. The summary statistics used included mean, standard deviation (sd) and coefficient of variation (cv, sd/mean) for all sea-ice variables, as well as the percentage of time (days) that a cell was classified as MIZ (% MIZ cover). Both the maximum RR and the mean-mode (i.e. mean minus mode) SIT were used as proxies for sea-ice deformation (i.e. under-ice habitat structural complexity). Previous work also hypothesizes that krill may alter their use of sea-ice habitat depending on the stage of the season (Meyer, 2012) Correlations between recruitment and sea ice variables To identify sea-ice drivers of krill recruitment, maps of temporal correlations were calculated between recruitment indices in key regions and the suite of modern sea-ice variables. Of these initial modern variables, five had the strongest magnitude and spatial area of correlations and also had support from literature (Table 2 We therefore focussed the analysis on these five. Using estimated sea-ice variables from the model brought new structure and definition to overwintering habitat composition (Fig. C For comparison, correlations to the classic sea-ice variables (DUR, TOA and TOR) were also calculated. Maps of temporal correlations were generated by calculating the Pearson's correlation between the time series of sea-ice variables within every model grid cell over the entire study area and the time series of log recruit density (excluding zero values) per hexagonal region. This resulted in a map of temporal correlation that had the same grid as the model. Zero values of log recruit density were excluded to avoid zeroinflation and over dispersion that would be caused by a high occurrence of seasons with an R 1 (and thus recruit density) of zero. This raises the question of whether these zeros are \\\"real\\\" (i.e. does failing to sample a small size class truly reflect zero recruitment). It is more likely that these size classes were present but were not sampled. This issue requires a more in-depth analysis into how recruitment is inferred from length frequency data and is beyond the scope of this paper. As an indicator of which correlations are unlikely to have arisen by chance, we performed a permutation test. This involved iteratively shuffling the ordering of the recruitment time series and recalculating the map of temporal correlation to generate a frequency distribution of correlations that would result from the randomized data. This provides a random distribution of correlations against which to compare our observed correlations. If the observed correlations occur at the tail ends of the permuted distribution, they are more likely to be real. The permutation test was performed 100 times for each sea-ice variable and recruitment region. We acknowledge that spatial autocorrelation and multiple testing may increase the chance of a type I error in spatial maps of temporal correlation and permutations. Therefore, we also summarized permutated maps (which have the dimensions of the model grid) using two whole-of-area statistics: the mean of the absolute correlation for all the model grid cells, and the 95% quantile. Since these summarized statistics required considerably less data storage, we were able to run 500 permutations for the whole-of-area statistics. Larval transport Finally, to provide context for the spatial correlations resulting from the above analysis and evaluate the prospect of remote connections, we also considered larval transport. Data description: Sea-ice and surface current velocities Sea-ice velocity fields were used to calculate back-trajectories of krill transport when sea ice was present. Previous analyses (e.g. Meyer et al., 2017; Thorpe et al., 2007) (Tschudi et al., 2019) However recent evaluations of NSIDC velocities against buoy data have found measurement artefacts that can significantly affect analysis results (Szanyi et al., 2016) (Sumata et al., 2014) We therefore used sea-ice velocities calculated from the satellite microwave sensors AMSR-E and AMSR-II images using a method based on maximum cross correlation (Kimura and Wakatsuchi, 2011) This \\\"Kimura\\\" dataset is structured on a 60 km resolution grid in daily timesteps from 2003 to 2016. The Kimura dataset has reproduced seaice velocities in the Arctic with a high degree of accuracy when validated against buoy data (Kimura et al., 2013) The ocean surface current velocity fields were produced by SSALTO/ DUACS and represent altimetry-derived dynamic topography heights and geostrophic velocities. This provides a measure of the mesoscale geostrophic component of flow. The product is available on a 1/4 grid in daily timesteps from 1993. Particle tracking To determine likely transport pathways, 15-20 (depending on the area covered by ocean) equally spaced points were laid in each recruitment region. For each of these points Lagrangian backtrajectories over the winter months were calculated from November 30th to April 1st of the same year (Meyer et al., 2017) Tison et al. (2020) This was done for every year from 2003 to 2016. The trajectories were updated in 12 h timesteps. When sea-ice velocity data was present, one timestep (12 h) of the day was advected with sea ice, and a second timestep was advected with under-ice surface currents. Since sea-ice cover inhibits the remote measurement of under-ice surface currents, they were estimated by scaling the magnitude of the altimetry-derived sea-ice velocity components by 0.43 and applying an offset of -35 to the direction (Meyer et al., 2017) When sea-ice velocity data was absent, the particle was advected with surface currents for both timesteps. The position of the particle was recorded every 10th day of the simulation to thin the data. For more detail on the particle tracking method, see Appendix E. All analyses were performed in the R statistical computing environment (R Core Team, 2018). For packages used see Table A Results Krill recruit density The krill data were processed to provide time series of mean krill density and pooled length frequency for nine polygons, termed \\\"recruitment regions\\\" (green hexagons in Fig. 1 Overall, these time series spanned from 1981 to 2016, but the sampling coverages available from the krill density and length-frequency databases varied by recruitment region (Tables B.1-2). Thus, these data supported calculations of proportional recruitment, krill density, and recruit density with variable temporal coverage (Fig. 4 The interannual trend in log 10 -transformed recruit density calculated across our nine recruitment regions (-0.046 log 10 (recruit density no/ m 2 )/year, Pr>|t| = 8.87x10 -3 , adjusted R 2 = 0.11, see Fig. B P < 0.001, adjusted R 2 = 0.39) (Atkinson et al., 2019) As to be expected there was more noise around our model fit for regional recruit density. Since our model structure did not explicitly account for region, the variation due to region was unexplained and resulted in a lower adjusted R 2 . Linking krill recruitment and sea ice 3.2.1. Correlations with sea ice Our selection of modern variables revealed spatial distributions in sea-ice characteristics that could not have been identified from sea-ice concentration (SIC) observations alone (Fig. 5 Highlighted regions include the Weddell-Scotia Confluence, the southwestern Weddell Sea, coastal regions, and the ice edge. This allowed us to identify new patterns of correlation with recruit density. To present the most rigorous findings we focused our results interpretation on those correlations that were most likely to be real, arising from the three best-sampled recruitment regions, as a robust correlation analysis requires a well-populated time series. These regions were: wAP mid., Bransfield and S. Georgia E. (Figs. 1 4 (Fig. 6, These areas were often not located in the same region from which the recruit density was derived, thus we termed these areas \\\"remote connections\\\". Collated at the wholeof-area scale, permutation results indicated six cases where the modern sea-ice variables are very likely to provide real correlations with our regional-scale recruitment densities (Fig. D These include wAP mid. recruitment correlating with Jul-Aug mean-mode sea-ice thickness (SIT), Jul-Aug maximum (max) ridging rate (RR) and Sep standard deviation (sd) of ridging rate (RR); Bransfield recruitment correlating with Sep marginal ice zone (MIZ) coverage; and S. Georgia E. recruitment correlating with Jul-Aug max RR and Sep MIZ coverage. Therefore, we are most confident in the maps of correlations and permutations that correspond to these particular sea-ice indicators. However these wholeof-area statistics can dilute signals from even fairly large-scale features which the permutation maps indicate to very likely be real (e.g. influence of Sep MIZ coverage for Bransfield recruitment, Figs. D.2 & D.4); their spatial size and coherence indicates a non-random phenomenon, so we have reasonable confidence to focus our presentation of remote connection results to the larger-scale features for our three best-sampled recruitment cells. Among the classic variables (Fig. 6 Remote connections for the classic variables were generally small in area. In the cases of the wAP mid. and Bransfield regions, positive correlations extended eastward Fig. 4 R 1 (proportion of 1-year-old krill in the population, green triangle) and recruit density (blue square) are plotted on their relevant link scale (i.e. logit-link for R 1 and log-link for recruit density). Note the two indices are related as recruit density is R 1 weighted by the mean standardized krill density for that region and season. Seasons where sampling occurred but recruit density was zero are indicated by grey circles at the bottom of the graph. The sea ice correlation analysis focused on three of the best-sampled recruitment regions (outlined in red). (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.) (downstream), while negative correlations extended westward along the MIZ in the Bellingshausen and Amundsen Seas as well as occurring in the southwest corner of the Weddell Sea. For S. Georgia E. the remote connections were mostly positive and more dispersed, extending westward from the Bransfield Strait and throughout part of the Weddell Sea. Remote connections identified using the modern modelled variables generally encompassed a larger area and highlighted different regions than the classic variables. For the wAP mid. these included a region of positive correlation spanning the Bellingshausen/Amundsen Seas, a positive correlation along the eastern peninsula (western Weddell Sea) and a negative correlation reflected in the greater Weddell Sea. A relatively similar pattern was evident in the west (positive) and east (negative) of the study domain for the Bransfield region. S. Georgia E. showed strong negative correlations with sea ice in the western and southern areas of the Weddell Sea. In addition, a thin arc of positive correlation occurred along the northern ice edge extending westward (upstream) from S. Georgia along the South Scotia Ridge. While this appears to be a series of smaller elements, a holistic look at the surrounding correlations reveals it is part of a larger ice-edge feature which spans all three modern sea-ice variables for September (i.e., MIZ coverage, mean SIT and sd RR; Fig. D.4 Albeit with less confidence, the continuity and spatial area suggest that it may indeed represent a real physical habitat feature. Finding remote connections with sea ice supports the theory that krill may associate with and be transported by sea ice. An exploration of sea-ice back trajectories is therefore needed to determine whether this explanation for the remote connection is likely, and if so, which remote connections are likely to have mechanistic drivers. Larval transport The larval transport analysis allows for the remote connections observed in the correlations to be interpreted through a mechanistic lens (Figs. 7 E These results indicate that recruitment in the wAP mid region may be, in part, driven by larval transport with sea ice from the Bellingshausen Sea. Similar inferences can be drawn between recruitment in the Bransfield region and sea ice in the western Weddell Sea, as well as South Georgia and sea ice along the northern ice edge of the Scotia Sea. Fig. 5 These climatologies represent spatial averages of their respective variables, calculated from the model output from 1979 to 2016. For example, the map for Jul-Aug max RR represents an average of the maximum RR that occurred over July and August for each year. Each variable has been scaled from 0 to 1 so that they can be plotted on the same colour scale. Both Jul-Aug max RR and Sep sd RR maps have been capped at their 95% quantiles; this was done so that extremely high values near the coast would not mask the structuring of RR values further offshore. Discussion The temporal correlations between regional recruit density and sea ice reveal remote connections which vary by recruitment region. These remote connections are often driven by multiple spatially overlapping sea-ice variables, but differ substantially depending on whether the seaice variables used were classic descriptors of sea-ice habitat (derived from observed SIC) or the modern variables (derived from sea-ice model output) developed within this study. Assuming the sea-ice variables represent real functional relationships between larvae and sea ice, then the remote connections should follow overwinter transport pathways from source regions to each recruitment region. We find that the remote connections derived from our modern sea-ice variables correspond well with overwinter larval transport, adding value to existing classic sea-ice variables in defining overwinter habitat quality. In S. Georgia E., recruitment studies generally conclude that the population is not self-sustaining, and that recruits are transported from elsewhere (Siegel and Watkins, 2016) Current understanding of krill transport to South Georgia supports the remote connections we found for S. Georgia E.. Transport studies have identified source populations for South Georgia, finding krill advected from the north-western Weddell Sea by the Weddell Gyre or Weddell-Scotia Confluence, and from the wAP by the Antarctic Circumpolar Current (ACC) (Fach and Klinck, 2006; Hofmann et al., 1998; Mackintosh, 1972; Marr, 1962; Meyer et al., 2017; Murphy et al., 2004) The remote connections identified with classic variables tended to be smaller in area, but the identification of regions near the ice edge in the Bransfield Strait supports earlier findings that changes in SIE and timing may interact with ocean circulation to affect krill transport to South Georgia (Murphy et al., 2004 (Murphy et al., , 1998;; Thorpe et al., 2007) Remote connections with the modern variables were more extensive and displayed a more connected network of possible transport pathways. Most notable was the strong negative correlation in the western and southern Weddell Sea. This was driven mainly by July-August maximum (max) ridging rate (RR) and September mean sea-ice thickness (SIT), indicating that lower values of these variables are related to higher recruitment in South Georgia (Fig. D.4). A lower RR and SIT could indicate that the ice is diverging, which would be linked to higher export by the Weddell Gyre to South Georgia. This remote connection matches where krill likely originate in the Weddell Sea, assuming they arrive in the productive marginal ice zone (MIZ) along the South Scotia Ridge by mid-September (Meyer et al., 2017) Krill would then be capable of reaching South Georgia by December (~76-89 days, Murphy et al., 2004) Larval back-trajectories support these previously studied transport pathways, reinforcing the importance of the South Scotia Ridge. Furthermore, we found a semi-continuous arc of high positive correlation along the South Scotia Ridge with S. Georgia E. recruitment. This may coincide with the eastward drift of the northern ice edge, transported by the ACC through the Drake Passage and across the Scotia Sea towards South Georgia. This is a well-evidenced transport route for krill to South Georgia (Fach and Klinck, 2006; Hofmann et al., 1998; Murphy et al., 2004 Murphy et al., , 1998;; Thorpe et al., 2004) It is feasible that krill arriving in South Georgia by spring-summer are found along this route in mid-late winter. This remote connection also coincides with the MIZ and is evident within all our modern September sea-ice variables, which is when we expect the MIZ to be most important for krill recruitment. By providing a food-rich environment, transport associated with the MIZ may explain the larger krill lengths observed in South Georgia compared to upstream source regions (Fach et al., 2002; Siegel and Watkins, 2016) We also note this feature extends past the tip of the Antarctic Peninsula Fig. 6 New regions of high correlation between recruit density in main cells and modern sea ice variables. These maps compare correlations to (columns left to right): classic variables (DUR, TOA, TOR) calculated using NSIDC SIC observations; classic variables calculated using the modelled SIC output; the modern sea ice variables identified using the model (Table 2 Note that this therefore compares the overlay of 3 variables for the two left columns, and 5 variables for the right column. Coloured regions show where the observed correlation was at the tail ends of the permuted distribution, to the 10% level. For TOA, the sign of the correlations is reversed, as an earlier (i.e. smaller) TOA results in longer DUR. Correlations for each of the variables are overlain with transparency, so each map represents a composite of the positive and negative correlations generated from each set of variables. Correlations are displayed for the three best-sampled recruitment regions. For maps that separate these high correlation areas into their component variables, see (AP), which includes key krill habitat in the South Shetland Islands and Bransfield Strait. Particle tracking using ocean transport has shown January krill may have come from as far as the South Shetland Islands in October (Murphy et al., 2004) Another study using velocities from surface drifting buoys suggested a transport time of 110 days, placing krill at the South Shetlands by mid-September (Ichii and Naganobu, 1996) Our larval transport analysis agrees with these transport times (Fig. 7 This region was particularly highlighted by a positive correlation with July-August mean-mode SIT (Fig. D.5) This indicator of seaice rugosity may suggest that sea ice in the South Shetland Islands and Bransfield Strait functions as a refuge for krill on their way to South Georgia in early winter. Thus, there may be true mechanistic links driving these feature, despite weaker correlations. In the western Antarctic Peninsula (wAP), the presence of the AP as a land barrier to the east dictates that incoming krill originate from the west, and sea-ice dynamics within the Amundsen and Bellingshausen Seas regions have been found to be important for explaining krill recruitment variability (Pi\\u00f1ones et al., 2013; Wiedenmann et al., 2009) The remote connections we identified using classic variables highlighted negative correlations with sea ice especially in the northern Amundsen Sea, which is inconsistent with existing evidence. They therefore seem less likely to represent plausible sea-ice habitat descriptors for krill recruiting in the wAP mid. The modern variables, however, identified a large remote connection (strong positive correlation) spanning both seas. This was located somewhat farther upstream than hypothesized source regions for the wAP (Pi\\u00f1ones et al., 2013) Our larval transport analysis that incorporates sea ice indicates that the remote connections in the western Bellingshausen Sea are indeed feasible source regions for krill in the wAP mid. If longer transport times or an increased association with sea ice (which moves ~ 2x faster than surface currents) were to be considered, it is highly likely the Amundsen Sea would also be included. This agrees with studies showing that considering sea-ice transport can enhance the distance travelled (Meyer et al., 2017; Thorpe et al., 2007) Therefore, recruitment in the wAP mid region may be partially driven by transport through sea ice in the Amundsen and Bellingshausen seas. The positive remote connection spanning the Amundsen and Bellingshausen seas was supported by the overlap of the three September sea-ice indicators (Fig. D.4 Together, these indicators suggest that thicker, more dynamic sea ice in these areas at the end of winter was most influential for high recruitment in the wAP mid. Such icescapes may result from dynamic processes such as storms that would contribute to sea ice thickening and ridging events. This would create over-rafted sea ice for refuge and more complex feeding substrate (Meyer et al., 2017) 1 Sea ice habitat in the Amundsen and Bellingshausen seas may therefore provide important refuge and increased food availability during late winter for larvae en route to the wAP. Despite evidence for local origins of krill within the wAP (summarised in Pi\\u00f1ones et al., 2013) (Fig D.5) This is possibly due to the shallow bathymetry in continental shelf areas that may provide a food-rich habitat which may facilitate diverse feeding strategies and reduce reliance on sea ice (Schmidt et al., 2014; Walsh et al., 2020) Shallow bathymetry has been assumed to limit successful spawning (Hofmann and H\\u00fcsrevoglu, 2003; Thorpe et al., 2019) Our findings may therefore indicate that the environmental conditions that interact to define good habitat within the shelf regions are complex and likely include more than just sea-ice processes. In the Bransfield region, Lagrangian particle tracking has identified diverse source regions, such as the Bellingshausen and western Weddell seas (Capella et al., 1992) Our larval transport analyses are consistent with this. The remote connections derived from classic and modern variables both highlighted some positive correlations upstream in the Bellingshausen Sea, but in general the correlations for Bransfield Fig. 7 Sea ice back-trajectories were calculated from November 30th of each year (2003) (2004) (2005) (2006) (2007) (2008) (2009) (2010) (2011) (2012) (2013) (2014) (2015) (2016) The progressive locations of the particles along the back-trajectories were recorded every 10 days. The resulting locations across all years were binned to a 4 \\u00d7 2 degree grid. Grid cell counts represent the number of binned locations along a back-trajectory occurring in a given month (e.g. to show where particles that end up at a given location were in July, etc.). (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.) recruitment were weaker than those found for wAP mid. or S. Georgia E. (Fig. D.4) Some small negative correlations with September mean SIT and standard deviation (sd) RR in the western Weddell Sea may be linked to increased sea-ice divergence and export of krill into the Bransfield Strait. There were also localized positive correlations around the South Shetland Islands with most sea-ice indicators, particularly September mean SIT (Fig D.5) This could indicate a hotspot of important habitat for larval overwintering around the South Shetland Islands. However, due to the small area of this feature it is difficult to validate this finding from current observations. These less conclusive findings may also reflect the emerging view that krill in this region have flexible overwintering strategies and are less reliant on sea-ice habitats. Alternatively, correlations could be masked by the diverse composition of krill in the Bransfield region, which reflects krill from different source regions and life histories. In addition to remote connections that have likely mechanistic drivers, we observed two predominant signals of climate modes within the remote connections. First was a dipole of positive correlations to the west of the AP and negative correlations to the east amongst the modern sea-ice variables for both the wAP mid. and Bransfield regions. Using inferences based on larval transport, we can attribute the western positive correlations to likely source regions, as well as correlations in the western Weddell Sea for the Bransfield region. We speculate that the remainder of the remote connections found in the Weddell Sea are likely the result of atmospheric correlations. This dipole invokes the patterns of large-scale atmospheric modes that operate in the area, such as the Pacific-South America mode (Mo and Higgins, 1998) (Turner, 2004) (Raphael, 2007) These modes have been shown to have a strong influence on Antarctic sea-ice patterns (Murphy et al., 2014; Yuan and Li, 2008) (Loeb and Santora, 2015; Massom and Stammerjohn, 2010; Massom et al., 2006) These climate modes may also play a role in a second climate-related pattern. This pattern was composed of negative remote connections extending westward from the AP along the MIZ and was formed by classic variables for the wAP mid. and Bransfield regions. This pattern is similar to that observed during a sea-ice compaction event. In these events, a quasi-stationary zonal wave three, along with a weak La Nina (in the 2005 event) or a positive Southern Annular Mode (2001/2 events) were associated with atmospheric anomalies that drove persistent north westerly winds (Massom et al., 2008 (Massom et al., , 2006)) The deepening of the Amundsen Sea Low can drive such winds causing extreme ice compaction towards the coast in the Amundsen and Bellingshausen seas (Raphael et al., 2016) Such events resulted in earlier ice retreat (Massom et al., 2008 (Massom et al., , 2006)) Observations indicate that compaction events have negative impacts on larval growth and recruitment (Massom and Stammerjohn, 2010; Massom et al., 2006) We should then expect to see a positive correlation between the ice edge and krill recruitment (shorter ice coverage = lower recruitment) rather than the negative correlation observed. Yet these compaction events can also result in increased ice melt and primary production (Massom et al., 2006) (Massom et al., 2008 (Massom et al., , 2006)) Thus, while these correlations may indicate an indirect mechanistic connection to krill recruitment and an importance of wAP compaction events, their influence on krill recruitment requires further exploration. Conclusions Overall, the correlations between recruit density and our modern sea-ice variables represent an improved approximation of krill overwinter habitat quality. Early analyses exploring the relationships between krill recruitment and sea ice identified classic sea-ice variables, such as duration, time of arrival, and time of retreat, as important, and the remote connections resulting from these classic variables corroborate many of these early findings. The remote connections from our modern sea-ice variables improve upon this by using a more mechanistic description of the sea-ice habitat to identify more probable source regions and more extensive and connected networks of transport pathways. These include source regions for South Georgia in the western and southern Weddell Sea as well as transport pathways along the South Scotia Ridge; source regions for the wAP in the Amundsen and Bellingshausen seas; and source regions for the Bransfield region in the Bellingshausen and western Weddell seas. These results provide a stepchange forward from exploring correlations between recruitment and environmental parameters to an understanding of the mechanisms that lead to these correlations. Our results have important implications in progressing current paradigms of how krill are responding to climate change. Our results support the emerging view that the impact of changing sea-ice dynamics on krill recruitment is nuanced (Melbourne-Thomas et al., 2016; Meyer et al., 2017; Saba et al., 2014) It is likely that krill respond to changes in sea ice quality, bounded by changes in sea-ice extent and timing. Furthermore, regional variation in dependencies on sea ice, e.g., evidenced by a lack of strong remote connections in the Bransfield region, may provide pockets of resilience to climate-driven changes in sea-ice dynamics. To robustly quantify the impacts of climate change on krill populations, progression beyond correlations to statistical models with mechanistic underpinnings is needed (Murphy et al., 2018; Quetin et al., 2007) Ideally, the most robust assessment method would be to develop a model that simulates the krill life cycle and population dynamics in response to environmental drivers, so that spatial estimates of krill biomass could be achieved. Currently, a lack of quantification in how changes in sea ice quality impact recruitment serves as a main barrier to simulating krill population dynamics. This has been hindered by uncertainties in measuring krill recruitment, as well as limited spatiotemporal coverage in measurements of sea ice physical-biological properties (Meiners et al., 2017) Our study begins to address this important knowledge gap. We identify a new suite of sea-ice variables that improve understanding of the relationship between sea ice and krill. These variables explicitly describe the physical habitat properties of sea ice, while considering its biological properties indirectly. Here our choice of sea-ice model is informed by our aims to compliment emerging satellite products, such as sea-ice thickness and rugosity (Tamura et al., 2015; Toyota et al., 2011) In the future, exploring the biological properties of sea ice explicitly will be vital to further quantifying these promising relationships. This could involve a model that represents sea ice biology, alongside mesoscale in-situ measurements of sea-ice variables related to food availability (e.g. Castellani et al., 2020) Future analyses may also account for the effects of varying larval overwinter back-trajectories. Variations in transport would result in different life histories of exposure to sea ice habitats (Kohlbach et al., 2017) CRediT authorship contribution statement Devi Veytia: Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Visualization, Writing -original draft, Writing -review & editing. Fig. 1 . 1 Fig. 1. Map of study area. This map highlights the recruitment regions in which krill density and length-frequency data were binned (green), as well as commonly referenced oceanographic features (white) and seas (yellow). The bathymetry and coastline raster was retrieved from the Blue Marble: Next Generation dataset, produced by the NASA Earth Observatory. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.) Fig. 2 . 2 Fig. 2. The derivation of R 1 from length frequency data. This method involved using a. the Bhattacharya method to inform b. a Gaussian mixture analysis. In a. an example of the Bhattacharya method mixture analysis is shown for the Bransfield region in 1994. The upper bar graph refers to the length-frequency distribution. The lower plot shows the sample calculation. Here the log of the data can be approximated by a parabola, which can be further simplified to a straight line by differencing. Thus, by plotting the logarithmic difference of each length frequency class against its midpoint, separate components can be identified by drawing straight lines through the data, as shown. In b. varying resulting R 1 values (for cases where high, moderate and zero R 1 values were calculated), are drawn in green over their original lengthfrequency distributions (S. Georgia E. in 1982, Bransfield in 1982 and Weddell N.E. in 2009 are shown as examples from left to right). (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.) Fig. 3 . 3 Fig. 3. Validating COCO the modelled SIE and MIZ against NSIDC observations. Panel a. displays the similarity in SIE and MIZ location, showing SIC between NSIDC and COCO the model for September, averaged over 1979-2016. Red contours outline the MIZ along the 0.15 and 0.80 ice concentration fraction isolines in NSIDC, and the 0.9 < DO YOU MEAN 0.09? 0.9 doesn't make sense!> and 0.92 isolines in COCOthe model. Panel b. validates how well COCO the model can reproduce interannual variation in SIE and MIZ area, using a monthly timeseries of SIE and MIZ area. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.) Fig. D.1. To determine correlations unlikely to have arisen by chance the maps of temporal correlations were interpreted alongside the permutation results (mapped in Fig. D.2, summarised by whole-of-area statistics in Fig. D.3). Fig.6. New regions of high correlation between recruit density in main cells and modern sea ice variables. These maps compare correlations to (columns left to right): classic variables (DUR, TOA, TOR) calculated using NSIDC SIC observations; classic variables calculated using the modelled SIC output; the modern sea ice variables identified using the model (Table2). Note that this therefore compares the overlay of 3 variables for the two left columns, and 5 variables for the right column. Coloured regions show where the observed correlation was at the tail ends of the permuted distribution, to the 10% level. For TOA, the sign of the correlations is reversed, as an earlier (i.e. smaller) TOA results in longer DUR. Correlations for each of the variables are overlain with transparency, so each map represents a composite of the positive and negative correlations generated from each set of variables. Correlations are displayed for the three best-sampled recruitment regions. For maps that separate these high correlation areas into their component variables, see Fig. D.4-5. Table 1 1 Commonly used abbreviations. Category Abbreviation Long Name Sea-ice variables SIE Sea-ice extent MIZ Marginal Ice Zone SIC Sea-ice concentration RR Ridging rate SIT Sea-ice thickness DUR Sea-ice duration (Stammerjohn et al., 2008) TOA Sea-ice time of arrival ( Stammerjohn et al., 2008) TOR Sea-ice time of retreat ( Stammerjohn et al., 2008) sd Standard deviation cv Coefficient of variation max Maximum Geographical and ACC Antarctic Circumpolar Current oceanographic features wAP western Antarctic Peninsula AP Antarctic Peninsula   Acknowledgements The authors acknowledge the contributions of all the data providers of the krill density and length frequency data. D.V. is the recipient of a Tasmania Graduate Research Scholarship provided by the University of Tasmania. S.B. is the recipient of an Australian Research Council Australian Discovery Early Career Award [project DE180100828]. E.M. was supported by the BAS ALI-Science Ecosystems project (developing krill life-cycle models and future projections) and acknowledges BAS colleagues for discussions on the development of analyses, models and projections of Antarctic krill population processes. S. Wotherspoon provided very helpful guidance regarding the permutation analyses. This research was supported by the Australian Government [Antarctic Science Projects 4408 and 4512], and through the Antarctic Science Collaboration Initiative program as part of the Australian Antarctic Program Partnership. The study is also a contribution to the international Integrating Climate and Ecosystems Dynamics Programme (ICED). K. K. was supported by JSPS KEKENHI [grants JP19K12301 and JP17H06323]. The surface velocity data were produced by SSALTO/ DUACS and distributed by Aviso+, with support from Cnes (http:// www.aviso.altimetry.fr/duacs/). The altimetry sea ice thickness data used in this study (doi 10.6096/CTOH_SEAICE_2019_12) were developed, validated by the CTOH/LEGOS, France and distributed by Aviso + .  Data statement KRILLBASE density data are freely available (https://www.bas.ac. uk/project/krillbase/). KRILLBASE length frequency data are available from the British Antarctic Survey by request (contact: Simeon Hill, sih@bas.ac.uk), and the Palmer LTER length frequency data are available online (https://doi.org/10.6073/pasta/be42bb841e696b7bca-d9957aed33db5e). Ocean-sea ice model output are available from K.K. upon request (kazuya.kusahara@gmail.com). For sea-ice velocity fields from the Kimura dataset, contact N.K. (kimura_n@aori.u-tokyo.ac.jp). The remotely-sensed surface velocity and sea-ice thickness products are available on Aviso+ (http://www.aviso.altimetry.fr/).  Author contributions All authors contributed to conceptualization, methodology, formal analysis and writing. S.C., S.B., S.K., K.M. and E.M. contributed to the supervision of D.V. Declaration of Competing Interest The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper. Appendix A. Supplementary data Supplementary data to this article can be found online at https://doi. org/10.1016/j.ecolind.2021.107934.     Sea ice ecosystems  K R Arrigo  Ann. Rev. Mar. Sci  6 2014 Arrigo, K.R., 2014. Sea ice ecosystems. Ann. Rev. Mar. Sci. 6, 439-467.   Large scale importance of sea ice biology in the Southern Ocean  K R Arrigo  D N Thomas  Antarct. Sci  16 471 2004 Arrigo, K.R., Thomas, D.N., 2004. Large scale importance of sea ice biology in the Southern Ocean. Antarct. Sci. 16, 471.   Primary production in Antarctic sea ice  K R Arrigo  D L Worthen  M P Lizotte  P Dixon  G Dieckmann  Science  276 1997 Arrigo, K.R., Worthen, D.L., Lizotte, M.P., Dixon, P., Dieckmann, G., 1997. Primary production in Antarctic sea ice. Science 276, 394-397.   KRILLBASE: a circumpolar database of Antarctic krill and salp numerical densities  A Atkinson  S L Hill  E A Pakhomov  R Anadon  S Chiba  K L Daly  R Downie  P T Fretwell  L Gerrish  G W Hosie  2017. 1926. 2016 Atkinson, A., Hill, S.L., Pakhomov, E.A., Anadon, R., Chiba, S., Daly, K.L., Downie, R., Fretwell, P.T., Gerrish, L., Hosie, G.W., 2017. KRILLBASE: a circumpolar database of Antarctic krill and salp numerical densities, 1926-2016.   Krill (Euphausia superba) distribution contracts southward during rapid regional warming  A Atkinson  S L Hill  E A Pakhomov  V Siegel  C S Reiss  V J Loeb  D K Steinberg  K Schmidt  G A Tarling  L Gerrish  S F Sailley  Nat. Clim. Change  9 2019 Atkinson, A., Hill, S.L., Pakhomov, E.A., Siegel, V., Reiss, C.S., Loeb, V.J., Steinberg, D.K., Schmidt, K., Tarling, G.A., Gerrish, L., Sailley, S.F., 2019. Krill (Euphausia superba) distribution contracts southward during rapid regional warming. Nat. Clim. Change 9, 142-147.   A re-appraisal of the total biomass and annual production of Antarctic krill  A Atkinson  V Siegel  E Pakhomov  M Jessopp  V Loeb  Deep Sea Res. Part I  56 2009 Atkinson, A., Siegel, V., Pakhomov, E., Jessopp, M., Loeb, V., 2009. A re-appraisal of the total biomass and annual production of Antarctic krill. Deep Sea Res. Part I 56, 727-740.   Long-term decline in krill stock and increase in salps within the Southern Ocean  A Atkinson  V Siegel  E Pakhomov  P Rothery  Nature  432 2004 Atkinson, A., Siegel, V., Pakhomov, E., Rothery, P., 2004. Long-term decline in krill stock and increase in salps within the Southern Ocean. Nature 432, 100-103.   A simple method of resolution of a distribution into Gaussian components  C Bhattacharya  Biometrics  1967 Bhattacharya, C., 1967. A simple method of resolution of a distribution into Gaussian components. Biometrics 115-135.   Environmental factors controlling phytoplankton processes in the Southern Ocean1  P W Boyd  J. Phycol  38 2002 Boyd, P.W., 2002. Environmental factors controlling phytoplankton processes in the Southern Ocean1. J. Phycol. 38, 844-861.   Antarctic krill under sea ice: elevated abundance in a narrow band just south of ice edge  A S Brierley  P G Fernandes  M A Brandon  F Armstrong  N W Millard  S D Mcphail  P Stevenson  M Pebody  J Perrett  M Squires  Science  295 2002 Brierley, A.S., Fernandes, P.G., Brandon, M.A., Armstrong, F., Millard, N.W., McPhail, S. D., Stevenson, P., Pebody, M., Perrett, J., Squires, M., 2002. Antarctic krill under sea ice: elevated abundance in a narrow band just south of ice edge. Science 295, 1890-1892.   Models of the early life history of Euphausia superba-Part II. Lagrangian calculations  J E Capella  L B Quetin  E E Hofmann  R M Ross  Deep Sea Res. Part I Oceanogr. Res. Pap  39 1992 Capella, J.E., Quetin, L.B., Hofmann, E.E., Ross, R.M., 1992. Models of the early life history of Euphausia superba-Part II. Lagrangian calculations. Deep Sea Res. Part I Oceanogr. Res. Pap. 39, 1201-1220.   Large-scale variability of physical and biological sea-ice properties in polar oceans  G Castellani  F L Schaafsma  S Arndt  B A Lange  I Peeken  J Ehrlich  C David  R Ricker  T Krumpen  S Hendricks  Front. Mar. Sci  7 536 2020 Castellani, G., Schaafsma, F.L., Arndt, S., Lange, B.A., Peeken, I., Ehrlich, J., David, C., Ricker, R., Krumpen, T., Hendricks, S., 2020. Large-scale variability of physical and biological sea-ice properties in polar oceans. Front. Mar. Sci. 7, 536.   updated yearly. Sea Ice Concentrations from Nimbus-7 SMMR and DMSP SSM/I-SSMIS Passive Microwave Data  D J Cavalieri  C L Parkinson  P Gloersen  H J Zwally  Version 1. NASA National Snow and Ice Data Center Distributed Active Archive Center Boulder, Colorado USA  1996 Cavalieri, D.J., Parkinson, C.L., Gloersen, P., Zwally, H.J., 1996, updated yearly. Sea Ice Concentrations from Nimbus-7 SMMR and DMSP SSM/I-SSMIS Passive Microwave Data, Version 1. NASA National Snow and Ice Data Center Distributed Active Archive Center, Boulder, Colorado USA.   The importance of Antarctic krill in biogeochemical cycles  E L Cavan  A Belcher  A Atkinson  S L Hill  S Kawaguchi  S Mccormack  B Meyer  S Nicol  L Ratnarajah  K Schmidt  D K Steinberg  G A Tarling  P W Boyd  Nat. Commun  10 4742 2019 Cavan, E.L., Belcher, A., Atkinson, A., Hill, S.L., Kawaguchi, S., McCormack, S., Meyer, B., Nicol, S., Ratnarajah, L., Schmidt, K., Steinberg, D.K., Tarling, G.A., Boyd, P.W., 2019. The importance of Antarctic krill in biogeochemical cycles. Nat. Commun. 10, 4742.   A synergistic approach for evaluating climate model output for ecological applications  R D Cavanagh  E J Murphy  T J Bracegirdle  J Turner  C A Knowland  S P Corney  W O Smith Jr  C M Waluda  N M Johnston  R G Bellerby  Front. Mar. Sci  4 308 2017 Cavanagh, R.D., Murphy, E.J., Bracegirdle, T.J., Turner, J., Knowland, C.A., Corney, S.P., Smith Jr, W.O., Waluda, C.M., Johnston, N.M., Bellerby, R.G., 2017. A synergistic approach for evaluating climate model output for ecological applications. Front. Mar. Sci. 4, 308.   No evidence for a decline in the density of Antarctic krill Euphausia superba Dana, 1850, in the Southwest Atlantic sector between 1976 and 2016  M J Cox  S Candy  W K De La Mare  S Nicol  S Kawaguchi  N Gales  J. Crust. Biol  38 2018 Cox, M.J., Candy, S., de la Mare, W.K., Nicol, S., Kawaguchi, S., Gales, N., 2018. No evidence for a decline in the density of Antarctic krill Euphausia superba Dana, 1850, in the Southwest Atlantic sector between 1976 and 2016. J. Crust. Biol. 38, 656-661.   Diet, provisioning and productivity responses of marine predators to differences in availability of Antarctic krill  J P Croxall  K Reid  P A Prince  Mar. Ecol. Prog. Ser  177 1999 Croxall, J.P., Reid, K., Prince, P.A., 1999. Diet, provisioning and productivity responses of marine predators to differences in availability of Antarctic krill. Mar. Ecol. Prog. Ser. 177, 115-131.   Overwintering development, growth, and feeding of larval Euphausia superba in the Antarctic marginal ice zone  K L Daly  Limnol. Oceanogr  35 1990 Daly, K.L., 1990. Overwintering development, growth, and feeding of larval Euphausia superba in the Antarctic marginal ice zone. Limnol. Oceanogr. 35, 1564-1576.   Overwintering growth and development of larval Euphausia superba: an interannual comparison under varying environmental conditions west of the Antarctic Peninsula  K L Daly  Deep Sea Res. Part II  51 2004 Daly, K.L., 2004. Overwintering growth and development of larval Euphausia superba: an interannual comparison under varying environmental conditions west of the Antarctic Peninsula. Deep Sea Res. Part II 51, 2139-2168.   Under-ice distribution of polar cod Boreogadus saida in the central Arctic Ocean and their association with sea-ice habitat properties  C David  B Lange  T Krumpen  F Schaafsma  J A Van Franeker  H Flores  Polar Biol  39 2016 David, C., Lange, B., Krumpen, T., Schaafsma, F., van Franeker, J.A., Flores, H., 2016. Under-ice distribution of polar cod Boreogadus saida in the central Arctic Ocean and their association with sea-ice habitat properties. Polar Biol. 39, 981-994.   Estimating krill recruitment and its variability  W De La Mare  CCAMLR Sci  1 1994 De la Mare, W., 1994. Estimating krill recruitment and its variability. CCAMLR Sci. 1, 55-69.   The ERA-Interim reanalysis: Configuration and performance of the data assimilation system  D P Dee  S M Uppala  A Simmons  P Berrisford  P Poli  S Kobayashi  U Andrae  M Balmaseda  G Balsamo  D P Bauer  Q. J. R. Meteorolog. Soc  137 2011 Dee, D.P., Uppala, S.M., Simmons, A., Berrisford, P., Poli, P., Kobayashi, S., Andrae, U., Balmaseda, M., Balsamo, G., Bauer, D.P., 2011. The ERA-Interim reanalysis: Configuration and performance of the data assimilation system. Q. J. R. Meteorolog. Soc. 137, 553-597.   Modeling studies of Antarctic krill Euphausia superba survival during transport across the Scotia Sea  B A Fach  E E Hofmann  E J Murphy  Mar. Ecol. Prog. Ser  231 2002 Fach, B.A., Hofmann, E.E., Murphy, E.J., 2002. Modeling studies of Antarctic krill Euphausia superba survival during transport across the Scotia Sea. Mar. Ecol. Prog. Ser. 231, 187-203.   Transport of Antarctic krill (Euphausia superba) across the Scotia Sea. Part I: circulation and particle tracking simulations  B A Fach  J M Klinck  Deep Sea Res. Part I  53 2006 Fach, B.A., Klinck, J.M., 2006. Transport of Antarctic krill (Euphausia superba) across the Scotia Sea. Part I: circulation and particle tracking simulations. Deep Sea Res. Part I 53, 987-1010.   The association of Antarctic krill Euphausia superba with the under-ice habitat  H Flores  J A Van Franeker  V Siegel  M Haraldsson  V Strass  E H Meesters  U Bathmann  W J Wolff  PLoS ONE  7 31775 2012 Flores, H., Van Franeker, J.A., Siegel, V., Haraldsson, M., Strass, V., Meesters, E.H., Bathmann, U., Wolff, W.J., 2012. The association of Antarctic krill Euphausia superba with the under-ice habitat. PLoS ONE 7, e31775.   Comparison of CryoSat-2 and ENVISAT radar freeboard over Arctic sea ice: toward an improved Envisat freeboard retrieval  K Guerreiro  S Fleury  E Zakharova  A Kouraev  F R\\u00e9my  P Maisongrande  The Cryosphere  11 2017 Guerreiro, K., Fleury, S., Zakharova, E., Kouraev, A., R\\u00e9my, F., Maisongrande, P., 2017. Comparison of CryoSat-2 and ENVISAT radar freeboard over Arctic sea ice: toward an improved Envisat freeboard retrieval. The Cryosphere 11, 2059-2073.   Late-summer sea ice thickness variability in the Arctic Transpolar Drift 1991-2001 derived from ground-based electromagnetic sounding  C Haas  Geophys. Res, Lett  31 2004 Haas, C., 2004. Late-summer sea ice thickness variability in the Arctic Transpolar Drift 1991-2001 derived from ground-based electromagnetic sounding. Geophys. Res, Lett, p. 31.   Field observations on the ontogeny of schooling of Euphausia superba furciliae and its relationship to ice in Antarctic waters  W Hamner  P Hamner  B Obst  J Carleton  Limnol. Oceanogr  34 1989 Hamner, W., Hamner, P., Obst, B., Carleton, J., 1989. Field observations on the ontogeny of schooling of Euphausia superba furciliae and its relationship to ice in Antarctic waters. Limnol. Oceanogr. 34, 451-456.   CCSR Ocean Component Model (COCO) version 4.0  H Hasumi  2006  University of Tokyo CCSR Report 25 Hasumi, H., 2006. CCSR Ocean Component Model (COCO) version 4.0. CCSR Report 25. University of Tokyo.   A circumpolar modeling study of habitat control of Antarctic krill (Euphausia superba) reproductive success  E E Hofmann  Y S H\\u00fcsrevoglu  Deep Sea Res. Part II  50 2003 Hofmann, E.E., H\\u00fcsrevoglu, Y.S., 2003. A circumpolar modeling study of habitat control of Antarctic krill (Euphausia superba) reproductive success. Deep Sea Res. Part II 50, 3121-3142.   Krill transport in the Scotia Sea and environs  E E Hofmann  J M Klinck  R A Locarnini  B Fach  E Murphy  Antarct. Sci  10 1998 Hofmann, E.E., Klinck, J.M., Locarnini, R.A., Fach, B., Murphy, E., 1998. Krill transport in the Scotia Sea and environs. Antarct. Sci. 10, 406-415.   Surface water circulation in krill fishing areas near the South Shetland Islands  T Ichii  M Naganobu  CCAMLR Sci  3 1996 Ichii, T., Naganobu, M., 1996. Surface water circulation in krill fishing areas near the South Shetland Islands. CCAMLR Sci. 3, 125-136.   Investigating controls on sea ice algal production using E3SMv1. 1-BGC  N Jeffery  M E Maltrud  E C Hunke  S Wang  J Wolfe  A K Turner  S M Burrows  X Shi  W H Lipscomb  W Maslowski  Annals of Glaciology  2020 Jeffery, N., Maltrud, M.E., Hunke, E.C., Wang, S., Wolfe, J., Turner, A.K., Burrows, S.M., Shi, X., Lipscomb, W.H., Maslowski, W., 2020. Investigating controls on sea ice algal production using E3SMv1. 1-BGC. Annals of Glaciology, 1-22.   The zooplankton food web under East Antarctic pack ice -A stable isotope study  Z Jia  K M Swadling  K M Meiners  S Kawaguchi  P Virtue  Deep Sea Res. Part II  131 2016 Jia, Z., Swadling, K.M., Meiners, K.M., Kawaguchi, S., Virtue, P., 2016. The zooplankton food web under East Antarctic pack ice -A stable isotope study. Deep Sea Res. Part II 131, 189-202.   Lipids as markers of nutritional condition and diet in the Antarctic krill Euphausia superba and Euphausia crystallorophias during austral winter  S.-J Ju  H R Harvey  Deep Sea Res. Part II  51 2004 Ju, S.-J., Harvey, H.R., 2004. Lipids as markers of nutritional condition and diet in the Antarctic krill Euphausia superba and Euphausia crystallorophias during austral winter. Deep Sea Res. Part II 51, 2199-2214.   The Antarctic sea ice cover from ICESat-2 and CryoSat-2: freeboard, snow depth, and ice thickness  S Kacimi  R Kwok  The Cryosphere  14 2020 Kacimi, S., Kwok, R., 2020. The Antarctic sea ice cover from ICESat-2 and CryoSat-2: freeboard, snow depth, and ice thickness. The Cryosphere 14, 4453-4474.   Relationship between recruitment of the Antarctic krill and the degree of ice cover near the South Shetland Islands  S Kawaguchi  M Satake  Fish. Sci  60 1994 Kawaguchi, S., Satake, M., 1994. Relationship between recruitment of the Antarctic krill and the degree of ice cover near the South Shetland Islands. Fish. Sci. 60, 123-124.   Influence of winter sea-ice motion on summer ice cover in the Arctic  N Kimura  A Nishimura  Y Tanaka  H Yamaguchi  Polar Res  32 20193 2013 Kimura, N., Nishimura, A., Tanaka, Y., Yamaguchi, H., 2013. Influence of winter sea-ice motion on summer ice cover in the Arctic. Polar Res. 32, 20193.   Large-scale processes governing the seasonal variability of the Antarctic sea ice  N Kimura  M Wakatsuchi  Tellus A: Dynamic Meteorology and Oceanography  63 2011 Kimura, N., Wakatsuchi, M., 2011. Large-scale processes governing the seasonal variability of the Antarctic sea ice. Tellus A: Dynamic Meteorology and Oceanography 63, 828-840.   Ice Algae-Produced Carbon Is Critical for Overwintering of Antarctic Krill Euphausia superba  D Kohlbach  B A Lange  F L Schaafsma  C David  M Vortkamp  M Graeve  J A Van Franeker  T Krumpen  H Flores  Frontiers in Marine Science  4 2017 Kohlbach, D., Lange, B.A., Schaafsma, F.L., David, C., Vortkamp, M., Graeve, M., van Franeker, J.A., Krumpen, T., Flores, H., 2017. Ice Algae-Produced Carbon Is Critical for Overwintering of Antarctic Krill Euphausia superba. Frontiers in Marine Science 4.   Modeling Antarctic ice shelf responses to future climate changes and impacts on the ocean  K Kusahara  H Hasumi  J. Geophys. Res. Oceans  118 2013 Kusahara, K., Hasumi, H., 2013. Modeling Antarctic ice shelf responses to future climate changes and impacts on the ocean. J. Geophys. Res. Oceans 118, 2454-2475.   An ocean-sea ice model study of the unprecedented Antarctic sea ice minimum in 2016  K Kusahara  P Reid  G D Williams  R Massom  H Hasumi  Environ. Res. Lett  13 84020 2018 Kusahara, K., Reid, P., Williams, G.D., Massom, R., Hasumi, H., 2018. An ocean-sea ice model study of the unprecedented Antarctic sea ice minimum in 2016. Environ. Res. Lett. 13, 084020.   Roles of wind stress and thermodynamic forcing in recent trends in Antarctic sea ice and Southern Ocean SST: An ocean-sea ice model study  K Kusahara  G D Williams  R Massom  P Reid  H Hasumi  Global Planet. Change  158 2017 Kusahara, K., Williams, G.D., Massom, R., Reid, P., Hasumi, H., 2017. Roles of wind stress and thermodynamic forcing in recent trends in Antarctic sea ice and Southern Ocean SST: An ocean-sea ice model study. Global Planet. Change 158, 103-118.   Spatiotemporal dependence of Antarctic sea ice variability to dynamic and thermodynamic forcing: A coupled ocean-sea ice model study  K Kusahara  G D Williams  R Massom  P Reid  H Hasumi  Clim. Dyn  52 2019 Kusahara, K., Williams, G.D., Massom, R., Reid, P., Hasumi, H., 2019. Spatiotemporal dependence of Antarctic sea ice variability to dynamic and thermodynamic forcing: A coupled ocean-sea ice model study. Clim. Dyn. 52, 3791-3807.   Climate variability and spatiotemporal dynamics of five Southern Ocean krill species  V J Loeb  J A Santora  Prog. Oceanogr  134 2015 Loeb, V.J., Santora, J.A., 2015. Climate variability and spatiotemporal dynamics of five Southern Ocean krill species. Prog. Oceanogr. 134, 93-122.   Age-groups from size-frequency data: a versatile and efficient method of analyzing distribution mixtures  P Macdonald  T Pitcher  Can. J. Fish Aquat. Sci  36 1979 Macdonald, P., Pitcher, T., 1979. Age-groups from size-frequency data: a versatile and efficient method of analyzing distribution mixtures. Can. J. Fish Aquat. Sci. 36, 987-1001.   Life cycle of Antarctic krill in relation to ice and water conditions  N A Mackintosh  Discovery Rep  36 1972 Mackintosh, N.A., 1972. Life cycle of Antarctic krill in relation to ice and water conditions. Discovery Rep. 36, 1-94.   Arctic and Antarctic sea ice change: contrasts, commonalities, and causes  T Maksym  Ann. Rev. Mar. Sci  11 2019 Maksym, T., 2019. Arctic and Antarctic sea ice change: contrasts, commonalities, and causes. Ann. Rev. Mar. Sci. 11, 187-213.   The natural history and geography of the Antarctic krill (Euphausia superba Dana)  J W S Marr  Discovery Rep  32 1962 Marr, J.W.S., 1962. The natural history and geography of the Antarctic krill (Euphausia superba Dana). Discovery Rep. 32, 33-464.   Antarctic sea ice change and variability-physical and ecological implications  R A Massom  S E Stammerjohn  Polar Sci  4 2010 Massom, R.A., Stammerjohn, S.E., 2010. Antarctic sea ice change and variability-physical and ecological implications. Polar Sci. 4, 149-186.   West Antarctic Peninsula sea ice in 2005: Extreme ice compaction and ice edge retreat due to strong anomaly with respect to climate  R A Massom  S E Stammerjohn  W Lefebvre  S A Harangozo  N Adams  T A Scambos  M J Pook  C Fowler  J. Geophys. Res. Oceans  113 2008 Massom, R.A., Stammerjohn, S.E., Lefebvre, W., Harangozo, S.A., Adams, N., Scambos, T. A., Pook, M.J., Fowler, C., 2008. West Antarctic Peninsula sea ice in 2005: Extreme ice compaction and ice edge retreat due to strong anomaly with respect to climate. J. Geophys. Res. Oceans 113.   Extreme anomalous atmospheric circulation in the West Antarctic Peninsula region in austral spring and summer 2001/02, and its profound impact on sea ice and biota  R A Massom  S E Stammerjohn  R C Smith  M J Pook  R A Iannuzzi  N Adams  D G Martinson  M Vernet  W R Fraser  L B Quetin  J. Clim  19 2006 Massom, R.A., Stammerjohn, S.E., Smith, R.C., Pook, M.J., Iannuzzi, R.A., Adams, N., Martinson, D.G., Vernet, M., Fraser, W.R., Quetin, L.B., 2006. Extreme anomalous atmospheric circulation in the West Antarctic Peninsula region in austral spring and summer 2001/02, and its profound impact on sea ice and biota. J. Clim. 19, 3544-3571.   Antarctic pack ice algal distribution: Floe-scale spatial variability and predictability from physical parameters  K M Meiners  S Arndt  S Bestley  T Krumpen  R Ricker  M Milnes  K Newbery  U Freier  S Jarman  R King  R Proud  S Kawaguchi  B Meyer  Geophys. Res. Lett  44 2017 Meiners, K.M., Arndt, S., Bestley, S., Krumpen, T., Ricker, R., Milnes, M., Newbery, K., Freier, U., Jarman, S., King, R., Proud, R., Kawaguchi, S., Meyer, B., 2017. Antarctic pack ice algal distribution: Floe-scale spatial variability and predictability from physical parameters. Geophys. Res. Lett. 44, 7382-7390.   Chlorophyll a in Antarctic sea ice from historical ice core data  K M Meiners  M Vancoppenolle  S Thanassekos  G S Dieckmann  D N Thomas  J L Tison  K R Arrigo  D L Garrison  A Mcminn  D Lannuzel  P Van Der Merwe  K M Swadling  W O Smith  I Melnikov  B Raymond  Geophys. Res. Lett  39 2012 Meiners, K.M., Vancoppenolle, M., Thanassekos, S., Dieckmann, G.S., Thomas, D.N., Tison, J.L., Arrigo, K.R., Garrison, D.L., McMinn, A., Lannuzel, D., van der Merwe, P., Swadling, K.M., Smith, W.O., Melnikov, I., Raymond, B., 2012. Chlorophyll a in Antarctic sea ice from historical ice core data. Geophys. Res. Lett. 39, n/a-n/a.   Under ice habitats for Antarctic krill larvae: Could less mean more under climate warming?  J Melbourne-Thomas  S Corney  R Trebilco  K Meiners  R Stevens  S Kawaguchi  M Sumner  A Constable  Geophys. Res. Lett  43 2016 Melbourne-Thomas, J., Corney, S., Trebilco, R., Meiners, K., Stevens, R., Kawaguchi, S., Sumner, M., Constable, A., 2016. Under ice habitats for Antarctic krill larvae: Could less mean more under climate warming? Geophys. Res. Lett. 43.   Polar Regions  M Meredith  M Sommerkorn  S Cassotta  C Derksen  A Ekaykin  A Hollowed  G Kofinas  A Mackintosh  J Melbourne-Thomas  M M C Muelbert  G Ottersen  H Pritchard  E A G ; Schuur  P Zhai  M Tignor  E Poloczanska  K Mintenbeck  IPCC Special Report on the Ocean and Cryosphere in a Changing Climate  A Alegr\\u00eda  M Nicolai  A Okem  J Petzold  B Rama  N M Weyer  2019 Meredith, M., Sommerkorn, M., Cassotta, S., Derksen, C., Ekaykin, A., Hollowed, A., Kofinas, G., Mackintosh, A., Melbourne-Thomas, J., Muelbert, M.M.C., Ottersen, G., Pritchard, H., Schuur, E.A.G., 2019. Polar Regions. In: IPCC Special Report on the Ocean and Cryosphere in a Changing Climate [H.-O. P\\u00f6rtner, D.C. Roberts, V. Masson-Delmotte, P. Zhai, M. Tignor, E. Poloczanska, K. Mintenbeck, A. Alegr\\u00eda, M. Nicolai, A. Okem, J. Petzold, B. Rama, N.M. Weyer (eds.)].   The overwintering of Antarctic krill, Euphausia superba, from an ecophysiological perspective  B Meyer  Polar Biol  35 2012 Meyer, B., 2012. The overwintering of Antarctic krill, Euphausia superba, from an ecophysiological perspective. Polar Biol. 35, 15-37.   Feeding and energy budgets of Antarctic krill Euphausia superba at the onset of winter-I. Furcilia III larvae  B Meyer  A Atkinson  D St\\u00f6bing  B Oettl  W Hagen  U Bathmann  Limnol. Oceanogr  47 2002 Meyer, B., Atkinson, A., St\\u00f6bing, D., Oettl, B., Hagen, W., Bathmann, U., 2002. Feeding and energy budgets of Antarctic krill Euphausia superba at the onset of winter-I. Furcilia III larvae. Limnol. Oceanogr. 47, 943-952.   The winter pack-ice zone provides a sheltered but food-poor habitat for larval Antarctic krill  B Meyer  U Freier  V Grimm  J Groeneveld  B P Hunt  S Kerwath  R King  C Klaas  E Pakhomov  K M Meiners  Nat. Ecol. Evol  1 2017 Meyer, B., Freier, U., Grimm, V., Groeneveld, J., Hunt, B.P., Kerwath, S., King, R., Klaas, C., Pakhomov, E., Meiners, K.M., 2017. The winter pack-ice zone provides a sheltered but food-poor habitat for larval Antarctic krill. Nat. Ecol. Evol. 1, 1853-1861.   Physiology, growth, and development of larval krill Euphausia superba in autumn and winter in the Lazarev Sea, Antarctica  B Meyer  V Fuentes  C Guerra  K Schmidt  A Atkinson  S Spahic  B Cisewski  U Freier  A Olariaga  U Bathmanna  Limnol. Oceanogr  54 2009 Meyer, B., Fuentes, V., Guerra, C., Schmidt, K., Atkinson, A., Spahic, S., Cisewski, B., Freier, U., Olariaga, A., Bathmanna, U., 2009. Physiology, growth, and development of larval krill Euphausia superba in autumn and winter in the Lazarev Sea, Antarctica. Limnol. Oceanogr. 54, 1595-1614.   The Pacific-South American modes and tropical convection during the Southern Hemisphere winter  K C Mo  R W Higgins  Mon. Weather Rev  126 1998 Mo, K.C., Higgins, R.W., 1998. The Pacific-South American modes and tropical convection during the Southern Hemisphere winter. Mon. Weather Rev. 126, 1581-1596.   Integrating Climate and Ecosystem Dynamics in the Southern Ocean (ICED) programme  E Murphy  N Johnston  S Corney  K Reid  Report of the ICED-CCAMLR Projections Workshop Hobart, Aus  2018. Apr 2018 SC-CAMLR-XXXVII Murphy, E., Johnston, N., Corney, S., Reid, K., 2018. Integrating Climate and Ecosystem Dynamics in the Southern Ocean (ICED) programme: Report of the ICED-CCAMLR Projections Workshop, 5-7 Apr 2018. SC-CAMLR-XXXVII, Hobart, Aus.   Modeling the krill transport pathways in the Scotia Sea: spatial and environmental connections generating the seasonal distribution of krill  E Murphy  S Thorpe  J Watkins  R Hewitt  Deep Sea Res. Part II  51 2004 Murphy, E., Thorpe, S., Watkins, J., Hewitt, R., 2004. Modeling the krill transport pathways in the Scotia Sea: spatial and environmental connections generating the seasonal distribution of krill. Deep Sea Res. Part II 51, 1435-1456.   Spatial and temporal operation of the Scotia Sea ecosystem: a review of large-scale links in a krill centred food web  E Murphy  J Watkins  P Trathan  K Reid  M Meredith  S Thorpe  N Johnston  A Clarke  G Tarling  M Collins  Philos. Trans. R. Soc. Lond. B Biol. Sci  362 2007 Murphy, E., Watkins, J., Trathan, P., Reid, K., Meredith, M., Thorpe, S., Johnston, N., Clarke, A., Tarling, G., Collins, M., 2007a. Spatial and temporal operation of the Scotia Sea ecosystem: a review of large-scale links in a krill centred food web. Philos. Trans. R. Soc. Lond. B Biol. Sci. 362, 113-148.   Understanding the structure and functioning of polar pelagic ecosystems to predict the impacts of change  E J Murphy  R D Cavanagh  K F Drinkwater  S M Grant  J Heymans  E E Hofmann  G L Hunt  N M Johnston  Proc. R. Soc. B  283 20161646 2016 Murphy, E.J., Cavanagh, R.D., Drinkwater, K.F., Grant, S.M., Heymans, J., Hofmann, E. E., Hunt, G.L., Johnston, N.M., 2016. Understanding the structure and functioning of polar pelagic ecosystems to predict the impacts of change. Proc. R. Soc. B 283, 20161646.   Variability of sea-ice in the northern Weddell Sea during the 20th century  E J Murphy  A Clarke  N J Abram  J Turner  J. Geophys. Res. Oceans  119 2014 Murphy, E.J., Clarke, A., Abram, N.J., Turner, J., 2014. Variability of sea-ice in the northern Weddell Sea during the 20th century. J. Geophys. Res. Oceans 119, 4549-4572.   Climatically driven fluctuations in Southern Ocean ecosystems  E J Murphy  P N Trathan  J L Watkins  K Reid  M P Meredith  J Forcada  S E Thorpe  N M Johnston  P Rothery  Proceedings of the Royal Soc. B: Biol. Sci  274 2007 Murphy, E.J., Trathan, P.N., Watkins, J.L., Reid, K., Meredith, M.P., Forcada, J., Thorpe, S.E., Johnston, N.M., Rothery, P., 2007b. Climatically driven fluctuations in Southern Ocean ecosystems. Proceedings of the Royal Soc. B: Biol. Sci. 274, 3057-3067.   Interannual variability of the South Georgia marine ecosystem: biological and physical sources of variation in the abundance of krill  E J Murphy  J Watkins  K Reid  P Trathan  I Everson  J Croxall  J Priddle  M Brandon  A Brierley  E Hofmann  Fish. Oceanogr  7 1998 Murphy, E.J., Watkins, J., Reid, K., Trathan, P., Everson, I., Croxall, J., Priddle, J., Brandon, M., Brierley, A., Hofmann, E., 1998. Interannual variability of the South Georgia marine ecosystem: biological and physical sources of variation in the abundance of krill. Fish. Oceanogr. 7, 381-390.   Krill, currents, and sea ice: Euphausia superba and its changing environment  S Nicol  AIBS Bulletin  56 2006 Nicol, S., 2006. Krill, currents, and sea ice: Euphausia superba and its changing environment. AIBS Bulletin 56, 111-120.   The fishery for Antarctic krill: Its current status and management regime  S Nicol  J Foster  Biology and Ecology of Antarctic krill  2016 Springer Nicol, S., Foster, J., 2016. The fishery for Antarctic krill: Its current status and management regime, Biology and Ecology of Antarctic krill. Springer 387-421.   Ocean circulation off east Antarctica affects ecosystem structure and sea-ice extent  S Nicol  T Pauly  N L Bindoff  S Wright  D Thiele  G W Hosie  P G Strutton  E Woehler  Nature  406 2000 Nicol, S., Pauly, T., Bindoff, N.L., Wright, S., Thiele, D., Hosie, G.W., Strutton, P.G., Woehler, E., 2000. Ocean circulation off east Antarctica affects ecosystem structure and sea-ice extent. Nature 406, 504-507.   Standard body length of Euphausia superba collected with a 2-m, 700-um net towed from surface to 120 m, collected aboard Palmer LTER annual cruises off the coast of the Western Antarctic Peninsula  Palmer Station Antarctica  Lter Steinberg  D Initiative, E.D.  2020. 2009 -2019 Environmental Data Initiative 5 ed Palmer Station Antarctica LTER, Steinberg, D., 2020. Standard body length of Euphausia superba collected with a 2-m, 700-um net towed from surface to 120 m, collected aboard Palmer LTER annual cruises off the coast of the Western Antarctic Peninsula, 2009 -2019, in: Initiative, E.D. (Ed.), 5 ed. Environmental Data Initiative.   Habitat partitioning in Antarctic krill: Spawning hotspots and nursery areas  F A Perry  A Atkinson  S F Sailley  G A Tarling  S L Hill  C H Lucas  D J Mayor  PLoS ONE  14 219325 2019 Perry, F.A., Atkinson, A., Sailley, S.F., Tarling, G.A., Hill, S.L., Lucas, C.H., Mayor, D.J., 2019. Habitat partitioning in Antarctic krill: Spawning hotspots and nursery areas. PLoS ONE 14, e0219325.   Projected changes of Antarctic krill habitat by the end of the 21st century  A Pi\\u00f1ones  A V Fedorov  Geophys. Res. Lett  43 2016 Pi\\u00f1ones, A., Fedorov, A.V., 2016. Projected changes of Antarctic krill habitat by the end of the 21st century. Geophys. Res. Lett. 43, 8580-8589.   Modeling the remote and local connectivity of Antarctic krill populations along the western Antarctic Peninsula  A Pi\\u00f1ones  E E Hofmann  K L Daly  M S Dinniman  J M Klinck  Mar. Ecol. Prog. Ser  481 2013 Pi\\u00f1ones, A., Hofmann, E.E., Daly, K.L., Dinniman, M.S., Klinck, J.M., 2013. Modeling the remote and local connectivity of Antarctic krill populations along the western Antarctic Peninsula. Mar. Ecol. Prog. Ser. 481, 69-92.   Episodic recruitment in Antarctic krill Euphausia superba in the Palmer LTER study region  L B Quetin  R M Ross  Mar. Ecol. Prog. Ser  259 2003 Quetin, L.B., Ross, R.M., 2003. Episodic recruitment in Antarctic krill Euphausia superba in the Palmer LTER study region. Mar. Ecol. Prog. Ser. 259, 185-200.   Ecological responses of Antarctic krill to environmental variability: can we predict the future?  L B Quetin  R M Ross  C H Fritsen  M Vernet  Antarct. Sci  19 2007 Quetin, L.B., Ross, R.M., Fritsen, C.H., Vernet, M., 2007. Ecological responses of Antarctic krill to environmental variability: can we predict the future? Antarct. Sci. 19, 253-266.   R: A language and environment for statistical computing  Team Core  R Foundation for Statistical computing Vienna, Austria  2018 R Core Team, 2018. R: A language and environment for statistical computing. R Foundation for Statistical computing, Vienna, Austria.   The influence of atmospheric zonal wave three on Antarctic sea ice variability  M N Raphael  Journal of Geophysical Research: Atmospheres  112 2007 Raphael, M.N., 2007. The influence of atmospheric zonal wave three on Antarctic sea ice variability. Journal of Geophysical Research: Atmospheres 112.   The Amundsen Sea low: Variability, change, and impact on Antarctic climate  M N Raphael  G Marshall  J Turner  R Fogt  D Schneider  D Dixon  J Hosking  J Jones  W R Hobbs  Bull. Am. Meteorol. Soc  97 2016 Raphael, M.N., Marshall, G., Turner, J., Fogt, R., Schneider, D., Dixon, D., Hosking, J., Jones, J., Hobbs, W.R., 2016. The Amundsen Sea low: Variability, change, and impact on Antarctic climate. Bull. Am. Meteorol. Soc. 97, 111-121.   Winter and spring controls on the summer food web of the coastal West Antarctic Peninsula  G K Saba  W R Fraser  V S Saba  R A Iannuzzi  K E Coleman  S C Doney  H W Ducklow  D G Martinson  T N Miles  D L Patterson-Fraser  Nat. Commun  5 4318 2014 Saba, G.K., Fraser, W.R., Saba, V.S., Iannuzzi, R.A., Coleman, K.E., Doney, S.C., Ducklow, H.W., Martinson, D.G., Miles, T.N., Patterson-Fraser, D.L., 2014. Winter and spring controls on the summer food web of the coastal West Antarctic Peninsula. Nat. Commun. 5, 4318.   Size and stage composition of age class 0 Antarctic krill (Euphausia superba) in the ice-water interface layer during winter/early spring  F Schaafsma  C David  E Pakhomov  B P Hunt  B Lange  H Flores  J A Van Franeker  Polar Biol  39 2016 Schaafsma, F., David, C., Pakhomov, E., Hunt, B.P., Lange, B., Flores, H., van Franeker, J. A., 2016. Size and stage composition of age class 0 Antarctic krill (Euphausia superba) in the ice-water interface layer during winter/early spring. Polar Biol. 39, 1515-1526.   Feeding and overwintering of Antarctic krill across its major habitats: The role of sea ice cover, water depth, and phytoplankton abundance  K Schmidt  A Atkinson  D W Pond  L C Ireland  Limnol. Oceanogr  59 2014 Schmidt, K., Atkinson, A., Pond, D.W., Ireland, L.C., 2014. Feeding and overwintering of Antarctic krill across its major habitats: The role of sea ice cover, water depth, and phytoplankton abundance. Limnol. Oceanogr. 59, 17-36.   Seabed foraging by Antarctic krill: Implications for stock assessment, bentho-pelagic coupling, and the vertical transfer of iron  K Schmidt  A Atkinson  S Steigenberger  S Fielding  M C Lindsay  D W Pond  G A Tarling  T A Klevjer  C S Allen  S Nicol  Limnol. Oceanogr  56 2011 Schmidt, K., Atkinson, A., Steigenberger, S., Fielding, S., Lindsay, M.C., Pond, D.W., Tarling, G.A., Klevjer, T.A., Allen, C.S., Nicol, S., 2011. Seabed foraging by Antarctic krill: Implications for stock assessment, bentho-pelagic coupling, and the vertical transfer of iron. Limnol. Oceanogr. 56, 1411-1428.   Krill (Euphausiacea) life history and aspects of population dynamics  V Siegel  Can. J. Fish. Aquat. Sci  57 2000 Siegel, V., 2000. Krill (Euphausiacea) life history and aspects of population dynamics. Can. J. Fish. Aquat. Sci. 57, 130-150.   Recruitment of Antarctic krill Euphausia superba and possible causes for its variability  V Siegel  V Loeb  Mar. Ecol. Prog. Ser  123 1995 Siegel, V., Loeb, V., 1995. Recruitment of Antarctic krill Euphausia superba and possible causes for its variability. Mar. Ecol. Prog. Ser. 123, 45-56.   Distribution, Biomass and Demography of Antarctic Krill, Euphausia superba  V Siegel  J L Watkins  Biology and Ecology of Antarctic Krill  V Siegel Cham  Springer International Publishing 2016 Siegel, V., Watkins, J.L., 2016. Distribution, Biomass and Demography of Antarctic Krill, Euphausia superba. In: Siegel, V. (Ed.), Biology and Ecology of Antarctic Krill. Springer International Publishing, Cham, pp. 21-100.   Trends in Antarctic annual sea ice retreat and advance and their relation to El Ni\\u00f1o-Southern Oscillation and Southern Annular Mode variability  S Stammerjohn  D Martinson  R Smith  X Yuan  D Rind  J. Geophys. Res. Oceans  113 2008 Stammerjohn, S., Martinson, D., Smith, R., Yuan, X., Rind, D., 2008. Trends in Antarctic annual sea ice retreat and advance and their relation to El Ni\\u00f1o-Southern Oscillation and Southern Annular Mode variability. J. Geophys. Res. Oceans 113.   An intercomparison of A rctic ice drift products to deduce uncertainty estimates  H Sumata  T Lavergne  F Girard-Ardhuin  N Kimura  M A Tschudi  F Kauker  M Karcher  R Gerdes  J. Geophys. Res. Oceans  119 2014 Sumata, H., Lavergne, T., Girard-Ardhuin, F., Kimura, N., Tschudi, M.A., Kauker, F., Karcher, M., Gerdes, R., 2014. An intercomparison of A rctic ice drift products to deduce uncertainty estimates. J. Geophys. Res. Oceans 119, 4887-4921.   Persistent artifacts in the NSIDC ice motion data set and their implications for analysis  S Szanyi  J V Lukovich  D Barber  G Haller  Geophys. Res. Lett  43 807 2016 Szanyi, S., Lukovich, J.V., Barber, D., Haller, G., 2016. Persistent artifacts in the NSIDC ice motion data set and their implications for analysis. Geophys. Res. Lett. 43, 10,800-810,807.   Helicopter-borne observations with portable microwave radiometer in the Southern Ocean and the Sea of Okhotsk  T Tamura  K I Ohshima  J L Lieser  T Toyota  K Tateyama  D Nomura  K Nakata  A D Fraser  P W Jansen  K B Newbery  Ann. Glaciol  56 2015 Tamura, T., Ohshima, K.I., Lieser, J.L., Toyota, T., Tateyama, K., Nomura, D., Nakata, K., Fraser, A.D., Jansen, P.W., Newbery, K.B., 2015. Helicopter-borne observations with portable microwave radiometer in the Southern Ocean and the Sea of Okhotsk. Ann. Glaciol. 56, 436-444.   Circumpolar connections between Antarctic krill (Euphausia superba Dana) populations: investigating the roles of ocean and sea ice transport  S Thorpe  E Murphy  J Watkins  Deep Sea Res. Part I  54 2007 Thorpe, S., Murphy, E., Watkins, J., 2007. Circumpolar connections between Antarctic krill (Euphausia superba Dana) populations: investigating the roles of ocean and sea ice transport. Deep Sea Res. Part I 54, 792-810.   Tracking passive drifters in a high resolution ocean model: implications for interannual variability of larval krill transport to South Georgia  S E Thorpe  K J Heywood  D P Stevens  M A Brandon  Deep Sea Res. Part I  51 2004 Thorpe, S.E., Heywood, K.J., Stevens, D.P., Brandon, M.A., 2004. Tracking passive drifters in a high resolution ocean model: implications for interannual variability of larval krill transport to South Georgia. Deep Sea Res. Part I 51, 909-920.   Circumpolar patterns in Antarctic krill larval recruitment: an environmentally driven model  S E Thorpe  G A Tarling  E J Murphy  Mar. Ecol. Prog. Ser  613 2019 Thorpe, S.E., Tarling, G.A., Murphy, E.J., 2019. Circumpolar patterns in Antarctic krill larval recruitment: an environmentally driven model. Mar. Ecol. Prog. Ser. 613, 77-96.   Physical and biological properties of early winter Antarctic sea ice in the Ross Sea  J.-L Tison  T Maksym  A D Fraser  M Corkill  N Kimura  Y Nosaka  D Nomura  M Vancoppenolle  S Ackley  S Stammerjohn  Ann. Glaciol  2020 Tison, J.-L., Maksym, T., Fraser, A.D., Corkill, M., Kimura, N., Nosaka, Y., Nomura, D., Vancoppenolle, M., Ackley, S., Stammerjohn, S., 2020. Physical and biological properties of early winter Antarctic sea ice in the Ross Sea. Ann. Glaciol. 1-19.   Retrieval of sea-ice thickness distribution in the Sea of Okhotsk from ALOS/PALSAR backscatter data  T Toyota  S Ono  K Cho  K I Ohshima  Ann. Glaciol  52 2011 Toyota, T., Ono, S., Cho, K., Ohshima, K.I., 2011. Retrieval of sea-ice thickness distribution in the Sea of Okhotsk from ALOS/PALSAR backscatter data. Ann. Glaciol. 52, 177-184.   The importance of krill predation in the Southern Ocean, Biology and ecology of Antarctic krill  P N Trathan  S L Hill  Springer  2016 Trathan, P.N., Hill, S.L., 2016. The importance of krill predation in the Southern Ocean, Biology and ecology of Antarctic krill. Springer 321-350.   Polar Pathfinder Weekly 25 km EASE-Grid Sea Ice Motion Vectors, Version 4  M Tschudi  W N Meier  J S Stewart  C Fowler  J Maslanik  NASA National Snow and Ice Data Center Distributed Active Archive Center  N N S A I D C D A A Center Boulder, Colorado, USA  2019 Tschudi, M., Meier, W.N., Stewart, J.S., Fowler, C., Maslanik, J., 2019. Polar Pathfinder Weekly 25 km EASE-Grid Sea Ice Motion Vectors, Version 4, in: Center, N.N.S.a.I.D. C.D.A.A. (Ed.). NASA National Snow and Ice Data Center Distributed Active Archive Center, Boulder, Colorado, USA.   The el nino-southern oscillation and antarctica  J Turner  Int. J. Climatol.: A J. Royal Meteorol. Soc  24 2004 Turner, J., 2004. The el nino-southern oscillation and antarctica. Int. J. Climatol.: A J. Royal Meteorol. Soc. 24, 1-31.   Circumpolar projections of Antarctic krill growth potential  D Veytia  S Corney  K M Meiners  S Kawaguchi  E J Murphy  S Bestley  Nat. Clim. Change  10 2020 Veytia, D., Corney, S., Meiners, K.M., Kawaguchi, S., Murphy, E.J., Bestley, S., 2020. Circumpolar projections of Antarctic krill growth potential. Nat. Clim. Change 10, 568-575.   Flexibility in Antarctic krill Euphausia superba decouples diet and recruitment from overwinter sea-ice conditions in the northern Antarctic Peninsula  J Walsh  C S Reiss  G M Watters  Mar. Ecol. Prog. Ser  642 2020 Walsh, J., Reiss, C.S., Watters, G.M., 2020. Flexibility in Antarctic krill Euphausia superba decouples diet and recruitment from overwinter sea-ice conditions in the northern Antarctic Peninsula. Mar. Ecol. Prog. Ser. 642, 1-19.   Connecting recruitment of Antarctic krill and sea ice  J Wiedenmann  K A Cresswell  M Mangel  Limnol. Oceanogr  54 2009 Wiedenmann, J., Cresswell, K.A., Mangel, M., 2009. Connecting recruitment of Antarctic krill and sea ice. Limnol. Oceanogr. 54, 799-811.   Estimation of Antarctic Land-Fast Sea Ice Algal Biomass and Snow Thickness From Under-Ice Radiance Spectra in Two Contrasting Areas  P Wongpan  K M Meiners  P J Langhorne  P Heil  I J Smith  G H Leonard  R A Massom  L A Clementson  T G Haskell  J. Geophys. Res. Oceans  123 2018 Wongpan, P., Meiners, K.M., Langhorne, P.J., Heil, P., Smith, I.J., Leonard, G.H., Massom, R.A., Clementson, L.A., Haskell, T.G., 2018. Estimation of Antarctic Land- Fast Sea Ice Algal Biomass and Snow Thickness From Under-Ice Radiance Spectra in Two Contrasting Areas. J. Geophys. Res. Oceans 123, 1907-1923.   Thickness distribution of Antarctic sea ice  A P Worby  C A Geiger  M J Paget  M L Van Woert  S F Ackley  T L Deliberty  J. Geophys. Res. Oceans  113 2008 Worby, A.P., Geiger, C.A., Paget, M.J., Van Woert, M.L., Ackley, S.F., DeLiberty, T.L., 2008. Thickness distribution of Antarctic sea ice. J. Geophys. Res. Oceans 113.   Structural changes in the digestive glands of larval Antarctic krill (Euphausia superba) during starvation  T Yoshida  S Kawaguchi  B Meyer  P Virtue  J Penschow  G Nash  Polar Biol  32 2009 Yoshida, T., Kawaguchi, S., Meyer, B., Virtue, P., Penschow, J., Nash, G., 2009. Structural changes in the digestive glands of larval Antarctic krill (Euphausia superba) during starvation. Polar Biol. 32, 503-507.   Climate modes in southern high latitudes and their impacts on Antarctic sea ice  X Yuan  C Li  J. Geophys. Res. Oceans  113 2008 Yuan, X., Li, C., 2008. Climate modes in southern high latitudes and their impacts on Antarctic sea ice. J. Geophys. Res. Oceans 113.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"label\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 3,\n        \"samples\": [\n          \"Primary\",\n          \"Missing\",\n          \"Secondary\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 40
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "data['label'].value_counts()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 209
        },
        "id": "fy97ozCbEA5j",
        "outputId": "5dcdbe6d-d578-44ae-cb3c-eb3155955dcc"
      },
      "execution_count": 41,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "label\n",
              "Secondary    423\n",
              "Missing      257\n",
              "Primary      222\n",
              "Name: count, dtype: int64"
            ],
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>count</th>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>label</th>\n",
              "      <th></th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>Secondary</th>\n",
              "      <td>423</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>Missing</th>\n",
              "      <td>257</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>Primary</th>\n",
              "      <td>222</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div><br><label><b>dtype:</b> int64</label>"
            ]
          },
          "metadata": {},
          "execution_count": 41
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "vectorizer = TfidfVectorizer()\n",
        "X = vectorizer.fit_transform(data['text'])  # Teks diubah ke bentuk numerik\n",
        "y = data['label']                           # Label sebagai target\n"
      ],
      "metadata": {
        "id": "4nG6c3aSE1Fq"
      },
      "execution_count": 42,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "X_train, X_test, y_train, y_test = train_test_split(\n",
        "    X, y, test_size=0.2, stratify=y, random_state=42)\n"
      ],
      "metadata": {
        "id": "2B0x4cO4GOHh"
      },
      "execution_count": 43,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "model = SVC(kernel='linear')\n",
        "model.fit(X_train, y_train)\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 80
        },
        "id": "VVl6OJQqGa3G",
        "outputId": "4d46c45e-a85a-4269-c093-d915d3b5b555"
      },
      "execution_count": 44,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "SVC(kernel='linear')"
            ],
            "text/html": [
              "<style>#sk-container-id-1 {\n",
              "  /* Definition of color scheme common for light and dark mode */\n",
              "  --sklearn-color-text: #000;\n",
              "  --sklearn-color-text-muted: #666;\n",
              "  --sklearn-color-line: gray;\n",
              "  /* Definition of color scheme for unfitted estimators */\n",
              "  --sklearn-color-unfitted-level-0: #fff5e6;\n",
              "  --sklearn-color-unfitted-level-1: #f6e4d2;\n",
              "  --sklearn-color-unfitted-level-2: #ffe0b3;\n",
              "  --sklearn-color-unfitted-level-3: chocolate;\n",
              "  /* Definition of color scheme for fitted estimators */\n",
              "  --sklearn-color-fitted-level-0: #f0f8ff;\n",
              "  --sklearn-color-fitted-level-1: #d4ebff;\n",
              "  --sklearn-color-fitted-level-2: #b3dbfd;\n",
              "  --sklearn-color-fitted-level-3: cornflowerblue;\n",
              "\n",
              "  /* Specific color for light theme */\n",
              "  --sklearn-color-text-on-default-background: var(--sg-text-color, var(--theme-code-foreground, var(--jp-content-font-color1, black)));\n",
              "  --sklearn-color-background: var(--sg-background-color, var(--theme-background, var(--jp-layout-color0, white)));\n",
              "  --sklearn-color-border-box: var(--sg-text-color, var(--theme-code-foreground, var(--jp-content-font-color1, black)));\n",
              "  --sklearn-color-icon: #696969;\n",
              "\n",
              "  @media (prefers-color-scheme: dark) {\n",
              "    /* Redefinition of color scheme for dark theme */\n",
              "    --sklearn-color-text-on-default-background: var(--sg-text-color, var(--theme-code-foreground, var(--jp-content-font-color1, white)));\n",
              "    --sklearn-color-background: var(--sg-background-color, var(--theme-background, var(--jp-layout-color0, #111)));\n",
              "    --sklearn-color-border-box: var(--sg-text-color, var(--theme-code-foreground, var(--jp-content-font-color1, white)));\n",
              "    --sklearn-color-icon: #878787;\n",
              "  }\n",
              "}\n",
              "\n",
              "#sk-container-id-1 {\n",
              "  color: var(--sklearn-color-text);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 pre {\n",
              "  padding: 0;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 input.sk-hidden--visually {\n",
              "  border: 0;\n",
              "  clip: rect(1px 1px 1px 1px);\n",
              "  clip: rect(1px, 1px, 1px, 1px);\n",
              "  height: 1px;\n",
              "  margin: -1px;\n",
              "  overflow: hidden;\n",
              "  padding: 0;\n",
              "  position: absolute;\n",
              "  width: 1px;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-dashed-wrapped {\n",
              "  border: 1px dashed var(--sklearn-color-line);\n",
              "  margin: 0 0.4em 0.5em 0.4em;\n",
              "  box-sizing: border-box;\n",
              "  padding-bottom: 0.4em;\n",
              "  background-color: var(--sklearn-color-background);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-container {\n",
              "  /* jupyter's `normalize.less` sets `[hidden] { display: none; }`\n",
              "     but bootstrap.min.css set `[hidden] { display: none !important; }`\n",
              "     so we also need the `!important` here to be able to override the\n",
              "     default hidden behavior on the sphinx rendered scikit-learn.org.\n",
              "     See: https://github.com/scikit-learn/scikit-learn/issues/21755 */\n",
              "  display: inline-block !important;\n",
              "  position: relative;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-text-repr-fallback {\n",
              "  display: none;\n",
              "}\n",
              "\n",
              "div.sk-parallel-item,\n",
              "div.sk-serial,\n",
              "div.sk-item {\n",
              "  /* draw centered vertical line to link estimators */\n",
              "  background-image: linear-gradient(var(--sklearn-color-text-on-default-background), var(--sklearn-color-text-on-default-background));\n",
              "  background-size: 2px 100%;\n",
              "  background-repeat: no-repeat;\n",
              "  background-position: center center;\n",
              "}\n",
              "\n",
              "/* Parallel-specific style estimator block */\n",
              "\n",
              "#sk-container-id-1 div.sk-parallel-item::after {\n",
              "  content: \"\";\n",
              "  width: 100%;\n",
              "  border-bottom: 2px solid var(--sklearn-color-text-on-default-background);\n",
              "  flex-grow: 1;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-parallel {\n",
              "  display: flex;\n",
              "  align-items: stretch;\n",
              "  justify-content: center;\n",
              "  background-color: var(--sklearn-color-background);\n",
              "  position: relative;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-parallel-item {\n",
              "  display: flex;\n",
              "  flex-direction: column;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-parallel-item:first-child::after {\n",
              "  align-self: flex-end;\n",
              "  width: 50%;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-parallel-item:last-child::after {\n",
              "  align-self: flex-start;\n",
              "  width: 50%;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-parallel-item:only-child::after {\n",
              "  width: 0;\n",
              "}\n",
              "\n",
              "/* Serial-specific style estimator block */\n",
              "\n",
              "#sk-container-id-1 div.sk-serial {\n",
              "  display: flex;\n",
              "  flex-direction: column;\n",
              "  align-items: center;\n",
              "  background-color: var(--sklearn-color-background);\n",
              "  padding-right: 1em;\n",
              "  padding-left: 1em;\n",
              "}\n",
              "\n",
              "\n",
              "/* Toggleable style: style used for estimator/Pipeline/ColumnTransformer box that is\n",
              "clickable and can be expanded/collapsed.\n",
              "- Pipeline and ColumnTransformer use this feature and define the default style\n",
              "- Estimators will overwrite some part of the style using the `sk-estimator` class\n",
              "*/\n",
              "\n",
              "/* Pipeline and ColumnTransformer style (default) */\n",
              "\n",
              "#sk-container-id-1 div.sk-toggleable {\n",
              "  /* Default theme specific background. It is overwritten whether we have a\n",
              "  specific estimator or a Pipeline/ColumnTransformer */\n",
              "  background-color: var(--sklearn-color-background);\n",
              "}\n",
              "\n",
              "/* Toggleable label */\n",
              "#sk-container-id-1 label.sk-toggleable__label {\n",
              "  cursor: pointer;\n",
              "  display: flex;\n",
              "  width: 100%;\n",
              "  margin-bottom: 0;\n",
              "  padding: 0.5em;\n",
              "  box-sizing: border-box;\n",
              "  text-align: center;\n",
              "  align-items: start;\n",
              "  justify-content: space-between;\n",
              "  gap: 0.5em;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 label.sk-toggleable__label .caption {\n",
              "  font-size: 0.6rem;\n",
              "  font-weight: lighter;\n",
              "  color: var(--sklearn-color-text-muted);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 label.sk-toggleable__label-arrow:before {\n",
              "  /* Arrow on the left of the label */\n",
              "  content: \"▸\";\n",
              "  float: left;\n",
              "  margin-right: 0.25em;\n",
              "  color: var(--sklearn-color-icon);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 label.sk-toggleable__label-arrow:hover:before {\n",
              "  color: var(--sklearn-color-text);\n",
              "}\n",
              "\n",
              "/* Toggleable content - dropdown */\n",
              "\n",
              "#sk-container-id-1 div.sk-toggleable__content {\n",
              "  max-height: 0;\n",
              "  max-width: 0;\n",
              "  overflow: hidden;\n",
              "  text-align: left;\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-unfitted-level-0);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-toggleable__content.fitted {\n",
              "  /* fitted */\n",
              "  background-color: var(--sklearn-color-fitted-level-0);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-toggleable__content pre {\n",
              "  margin: 0.2em;\n",
              "  border-radius: 0.25em;\n",
              "  color: var(--sklearn-color-text);\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-unfitted-level-0);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-toggleable__content.fitted pre {\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-fitted-level-0);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 input.sk-toggleable__control:checked~div.sk-toggleable__content {\n",
              "  /* Expand drop-down */\n",
              "  max-height: 200px;\n",
              "  max-width: 100%;\n",
              "  overflow: auto;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 input.sk-toggleable__control:checked~label.sk-toggleable__label-arrow:before {\n",
              "  content: \"▾\";\n",
              "}\n",
              "\n",
              "/* Pipeline/ColumnTransformer-specific style */\n",
              "\n",
              "#sk-container-id-1 div.sk-label input.sk-toggleable__control:checked~label.sk-toggleable__label {\n",
              "  color: var(--sklearn-color-text);\n",
              "  background-color: var(--sklearn-color-unfitted-level-2);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-label.fitted input.sk-toggleable__control:checked~label.sk-toggleable__label {\n",
              "  background-color: var(--sklearn-color-fitted-level-2);\n",
              "}\n",
              "\n",
              "/* Estimator-specific style */\n",
              "\n",
              "/* Colorize estimator box */\n",
              "#sk-container-id-1 div.sk-estimator input.sk-toggleable__control:checked~label.sk-toggleable__label {\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-unfitted-level-2);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-estimator.fitted input.sk-toggleable__control:checked~label.sk-toggleable__label {\n",
              "  /* fitted */\n",
              "  background-color: var(--sklearn-color-fitted-level-2);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-label label.sk-toggleable__label,\n",
              "#sk-container-id-1 div.sk-label label {\n",
              "  /* The background is the default theme color */\n",
              "  color: var(--sklearn-color-text-on-default-background);\n",
              "}\n",
              "\n",
              "/* On hover, darken the color of the background */\n",
              "#sk-container-id-1 div.sk-label:hover label.sk-toggleable__label {\n",
              "  color: var(--sklearn-color-text);\n",
              "  background-color: var(--sklearn-color-unfitted-level-2);\n",
              "}\n",
              "\n",
              "/* Label box, darken color on hover, fitted */\n",
              "#sk-container-id-1 div.sk-label.fitted:hover label.sk-toggleable__label.fitted {\n",
              "  color: var(--sklearn-color-text);\n",
              "  background-color: var(--sklearn-color-fitted-level-2);\n",
              "}\n",
              "\n",
              "/* Estimator label */\n",
              "\n",
              "#sk-container-id-1 div.sk-label label {\n",
              "  font-family: monospace;\n",
              "  font-weight: bold;\n",
              "  display: inline-block;\n",
              "  line-height: 1.2em;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-label-container {\n",
              "  text-align: center;\n",
              "}\n",
              "\n",
              "/* Estimator-specific */\n",
              "#sk-container-id-1 div.sk-estimator {\n",
              "  font-family: monospace;\n",
              "  border: 1px dotted var(--sklearn-color-border-box);\n",
              "  border-radius: 0.25em;\n",
              "  box-sizing: border-box;\n",
              "  margin-bottom: 0.5em;\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-unfitted-level-0);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-estimator.fitted {\n",
              "  /* fitted */\n",
              "  background-color: var(--sklearn-color-fitted-level-0);\n",
              "}\n",
              "\n",
              "/* on hover */\n",
              "#sk-container-id-1 div.sk-estimator:hover {\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-unfitted-level-2);\n",
              "}\n",
              "\n",
              "#sk-container-id-1 div.sk-estimator.fitted:hover {\n",
              "  /* fitted */\n",
              "  background-color: var(--sklearn-color-fitted-level-2);\n",
              "}\n",
              "\n",
              "/* Specification for estimator info (e.g. \"i\" and \"?\") */\n",
              "\n",
              "/* Common style for \"i\" and \"?\" */\n",
              "\n",
              ".sk-estimator-doc-link,\n",
              "a:link.sk-estimator-doc-link,\n",
              "a:visited.sk-estimator-doc-link {\n",
              "  float: right;\n",
              "  font-size: smaller;\n",
              "  line-height: 1em;\n",
              "  font-family: monospace;\n",
              "  background-color: var(--sklearn-color-background);\n",
              "  border-radius: 1em;\n",
              "  height: 1em;\n",
              "  width: 1em;\n",
              "  text-decoration: none !important;\n",
              "  margin-left: 0.5em;\n",
              "  text-align: center;\n",
              "  /* unfitted */\n",
              "  border: var(--sklearn-color-unfitted-level-1) 1pt solid;\n",
              "  color: var(--sklearn-color-unfitted-level-1);\n",
              "}\n",
              "\n",
              ".sk-estimator-doc-link.fitted,\n",
              "a:link.sk-estimator-doc-link.fitted,\n",
              "a:visited.sk-estimator-doc-link.fitted {\n",
              "  /* fitted */\n",
              "  border: var(--sklearn-color-fitted-level-1) 1pt solid;\n",
              "  color: var(--sklearn-color-fitted-level-1);\n",
              "}\n",
              "\n",
              "/* On hover */\n",
              "div.sk-estimator:hover .sk-estimator-doc-link:hover,\n",
              ".sk-estimator-doc-link:hover,\n",
              "div.sk-label-container:hover .sk-estimator-doc-link:hover,\n",
              ".sk-estimator-doc-link:hover {\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-unfitted-level-3);\n",
              "  color: var(--sklearn-color-background);\n",
              "  text-decoration: none;\n",
              "}\n",
              "\n",
              "div.sk-estimator.fitted:hover .sk-estimator-doc-link.fitted:hover,\n",
              ".sk-estimator-doc-link.fitted:hover,\n",
              "div.sk-label-container:hover .sk-estimator-doc-link.fitted:hover,\n",
              ".sk-estimator-doc-link.fitted:hover {\n",
              "  /* fitted */\n",
              "  background-color: var(--sklearn-color-fitted-level-3);\n",
              "  color: var(--sklearn-color-background);\n",
              "  text-decoration: none;\n",
              "}\n",
              "\n",
              "/* Span, style for the box shown on hovering the info icon */\n",
              ".sk-estimator-doc-link span {\n",
              "  display: none;\n",
              "  z-index: 9999;\n",
              "  position: relative;\n",
              "  font-weight: normal;\n",
              "  right: .2ex;\n",
              "  padding: .5ex;\n",
              "  margin: .5ex;\n",
              "  width: min-content;\n",
              "  min-width: 20ex;\n",
              "  max-width: 50ex;\n",
              "  color: var(--sklearn-color-text);\n",
              "  box-shadow: 2pt 2pt 4pt #999;\n",
              "  /* unfitted */\n",
              "  background: var(--sklearn-color-unfitted-level-0);\n",
              "  border: .5pt solid var(--sklearn-color-unfitted-level-3);\n",
              "}\n",
              "\n",
              ".sk-estimator-doc-link.fitted span {\n",
              "  /* fitted */\n",
              "  background: var(--sklearn-color-fitted-level-0);\n",
              "  border: var(--sklearn-color-fitted-level-3);\n",
              "}\n",
              "\n",
              ".sk-estimator-doc-link:hover span {\n",
              "  display: block;\n",
              "}\n",
              "\n",
              "/* \"?\"-specific style due to the `<a>` HTML tag */\n",
              "\n",
              "#sk-container-id-1 a.estimator_doc_link {\n",
              "  float: right;\n",
              "  font-size: 1rem;\n",
              "  line-height: 1em;\n",
              "  font-family: monospace;\n",
              "  background-color: var(--sklearn-color-background);\n",
              "  border-radius: 1rem;\n",
              "  height: 1rem;\n",
              "  width: 1rem;\n",
              "  text-decoration: none;\n",
              "  /* unfitted */\n",
              "  color: var(--sklearn-color-unfitted-level-1);\n",
              "  border: var(--sklearn-color-unfitted-level-1) 1pt solid;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 a.estimator_doc_link.fitted {\n",
              "  /* fitted */\n",
              "  border: var(--sklearn-color-fitted-level-1) 1pt solid;\n",
              "  color: var(--sklearn-color-fitted-level-1);\n",
              "}\n",
              "\n",
              "/* On hover */\n",
              "#sk-container-id-1 a.estimator_doc_link:hover {\n",
              "  /* unfitted */\n",
              "  background-color: var(--sklearn-color-unfitted-level-3);\n",
              "  color: var(--sklearn-color-background);\n",
              "  text-decoration: none;\n",
              "}\n",
              "\n",
              "#sk-container-id-1 a.estimator_doc_link.fitted:hover {\n",
              "  /* fitted */\n",
              "  background-color: var(--sklearn-color-fitted-level-3);\n",
              "}\n",
              "</style><div id=\"sk-container-id-1\" class=\"sk-top-container\"><div class=\"sk-text-repr-fallback\"><pre>SVC(kernel=&#x27;linear&#x27;)</pre><b>In a Jupyter environment, please rerun this cell to show the HTML representation or trust the notebook. <br />On GitHub, the HTML representation is unable to render, please try loading this page with nbviewer.org.</b></div><div class=\"sk-container\" hidden><div class=\"sk-item\"><div class=\"sk-estimator fitted sk-toggleable\"><input class=\"sk-toggleable__control sk-hidden--visually\" id=\"sk-estimator-id-1\" type=\"checkbox\" checked><label for=\"sk-estimator-id-1\" class=\"sk-toggleable__label fitted sk-toggleable__label-arrow\"><div><div>SVC</div></div><div><a class=\"sk-estimator-doc-link fitted\" rel=\"noreferrer\" target=\"_blank\" href=\"https://scikit-learn.org/1.6/modules/generated/sklearn.svm.SVC.html\">?<span>Documentation for SVC</span></a><span class=\"sk-estimator-doc-link fitted\">i<span>Fitted</span></span></div></label><div class=\"sk-toggleable__content fitted\"><pre>SVC(kernel=&#x27;linear&#x27;)</pre></div> </div></div></div></div>"
            ]
          },
          "metadata": {},
          "execution_count": 44
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "y_pred = model.predict(X_test)\n",
        "\n",
        "print(\"✅ Akurasi:\", accuracy_score(y_test, y_pred))\n",
        "print(\"\\n📊 Classification Report:\\n\", classification_report(y_test, y_pred))\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "BpfNDEwJIg4s",
        "outputId": "a020726e-1a40-4e6e-b4d4-fa6cb87994b8"
      },
      "execution_count": 45,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "✅ Akurasi: 0.9392265193370166\n",
            "\n",
            "📊 Classification Report:\n",
            "               precision    recall  f1-score   support\n",
            "\n",
            "     Missing       0.91      0.98      0.94        52\n",
            "     Primary       0.89      0.91      0.90        44\n",
            "   Secondary       0.99      0.93      0.96        85\n",
            "\n",
            "    accuracy                           0.94       181\n",
            "   macro avg       0.93      0.94      0.93       181\n",
            "weighted avg       0.94      0.94      0.94       181\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from sklearn.metrics import confusion_matrix, ConfusionMatrixDisplay\n",
        "import matplotlib.pyplot as plt\n",
        "\n",
        "# Buat confusion matrix\n",
        "cm = confusion_matrix(y_test, y_pred, labels=model.classes_)\n",
        "\n",
        "# Tampilkan confusion matrix\n",
        "disp = ConfusionMatrixDisplay(confusion_matrix=cm, display_labels=model.classes_)\n",
        "fig, ax = plt.subplots(figsize=(6, 6))\n",
        "disp.plot(ax=ax, cmap='Blues', values_format='d')\n",
        "plt.title(\"Confusion Matrix - SVM\")\n",
        "plt.show()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 507
        },
        "id": "RfRm5sChR3Pv",
        "outputId": "6f849304-2875-4e40-d41b-35dd97d47631"
      },
      "execution_count": 50,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 600x600 with 2 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAjgAAAHqCAYAAAAJazRPAAAAOnRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjEwLjAsIGh0dHBzOi8vbWF0cGxvdGxpYi5vcmcvlHJYcgAAAAlwSFlzAAAPYQAAD2EBqD+naQAAXy9JREFUeJzt3XdcU9f7B/BPWAkCAURlKKAoAiruhQsHintgtY5WQLELq5U6WwdqrS111dZdBbViq1ap2yLWjVStWm2RqlXRMhzIVGbu7w+/5NeIg0hiQu7n7eu+2px7cu4TcDw855x7JYIgCCAiIiIyIEa6DoCIiIhI05jgEBERkcFhgkNEREQGhwkOERERGRwmOERERGRwmOAQERGRwWGCQ0RERAaHCQ4REREZHBNdB0BERETal5+fj8LCQq2MbWZmBplMppWxXxUTHCIiIgOXn58Pcys7oPiRVsZ3cHDAjRs39CrJYYJDRERk4AoLC4HiR5A2DAaMzTQ7eEkh0v6MRGFhIRMcIiIi0gFjM0g0nODo6wMtmeAQERGJhQSARKL5MfUQd1ERERGRwWEFh4iISCwkRk8OTY+ph/QzKiIiIqIKYAWHiIhILCQSLazB0c9FOKzgEBERkcFhBYeIiEgsRLQGhwkOERGRWHCKioiIiKjyYgWHiIhINLQwRaWntRL9jIqIiIioAljBISIiEguuwSEiIiKqvFjBISIiEgsRbRPXz6iIiIiIKoAVHCIiIrEQ0RocJjhERERiwSkqIiIiosqLFRwiIiKxENEUFSs4REREZHCY4BBVMlevXkWPHj1gbW0NiUSCmJgYjY5/8+ZNSCQSREVFaXTcyqxz587o3LmzrsMgqrjSNTiaPvSQfkZFpOeuX7+Od999F25ubpDJZJDL5Wjfvj2+/vprPH78WKvXDgwMxKVLlzB//nxs2rQJLVu21Or1XqegoCBIJBLI5fJnfh2vXr0KiUQCiUSChQsXqj1+SkoKwsPDceHCBQ1E+3oUFhbi66+/RrNmzSCXy2FjY4OGDRvinXfewZUrVwAA/fv3R5UqVZCTk/PccUaOHAkzMzM8ePAAAJRfx5CQkGf2//TTT5V97t+/r/kPRqRlXINDpKa9e/diyJAhkEqlGDVqFBo1aoTCwkKcOHECkydPxp9//ok1a9Zo5dqPHz9GfHw8Pv30U4wbN04r13B1dcXjx49hamqqlfFfxsTEBI8ePcLu3bsxdOhQlXObN2+GTCZDfn7+K42dkpKCOXPmoHbt2mjatGm53/fLL7+80vU0YfDgwdi/fz+GDx+OsWPHoqioCFeuXMGePXvQrl07eHp6YuTIkdi9ezd27tyJUaNGlRnj0aNH+Pnnn9GzZ0/Y2dkp22UyGX766SesWLECZmZmKu/ZsmVLhb7WpKckEi3sotLPNThMcIjUcOPGDQwbNgyurq44fPgwHB0dledCQ0Nx7do17N27V2vXv3fvHgDAxsZGa9eQSCSQyWRaG/9lpFIp2rdvjy1btpRJcKKjo9GnTx/89NNPryWWR48eoUqVKmX+8X9dzpw5gz179mD+/Pn45JNPVM59++23yMzMBPCkgmNlZYXo6OhnJjg///wz8vLyMHLkSJX2nj17YteuXdi/fz8GDBigbD916hRu3LiBwYMHv7avNZGmcYqKSA0RERHIzc3FunXrVJKbUvXq1cOECROUr4uLizFv3jzUrVsXUqkUtWvXxieffIKCggKV99WuXRt9+/bFiRMn0Lp1a8hkMri5uWHjxo3KPuHh4XB1dQUATJ48GRKJBLVr1wbwZGqn9P//Kzw8HJKnfrqKjY1Fhw4dYGNjA0tLS3h4eKj84/m8NTiHDx9Gx44dYWFhARsbGwwYMACJiYnPvN61a9cQFBQEGxsbWFtbIzg4GI8ePXr+F/YpI0aMwP79+5X/gANP/rG/evUqRowYUaZ/RkYGJk2aBG9vb1haWkIul6NXr164ePGiss+RI0fQqlUrAEBwcLBy+qX0c3bu3BmNGjXCuXPn0KlTJ1SpUkX5dXl6DU5gYCBkMlmZz+/v7w9bW1ukpKSU+7O+yPXr1wEA7du3L3PO2NhYWY0xNzdHQEAA4uLicPfu3TJ9o6OjYWVlhf79+6u016xZE506dUJ0dLRK++bNm+Ht7Y1GjRpp5HOQHjGSaOfQQ0xwiNSwe/duuLm5oV27duXqHxISglmzZqF58+ZYsmQJfH19sWDBAgwbNqxM32vXruGNN95A9+7dsWjRItja2iIoKAh//vknACAgIABLliwBAAwfPhybNm3C0qVL1Yr/zz//RN++fVFQUIC5c+di0aJF6N+/P06ePPnC9x06dAj+/v64e/cuwsPDERYWhlOnTqF9+/a4efNmmf5Dhw5FTk4OFixYgKFDhyIqKgpz5swpd5wBAQGQSCTYsWOHsi06Ohqenp5o3rx5mf7//PMPYmJi0LdvXyxevBiTJ0/GpUuX4Ovrq0w2vLy8MHfuXADAO++8g02bNmHTpk3o1KmTcpwHDx6gV69eaNq0KZYuXYouXbo8M76vv/4a1atXR2BgIEpKSgAAq1evxi+//IJvvvkGTk5O5f6sL1Ka0G7evBnFxcUv7Dty5EgUFxdj69atKu0ZGRk4ePAgBg0aBHNz8zLvGzFiBHbv3o3c3FwAT5Lybdu2PTORJAMgokXGEIioXLKysgQAwoABA8rV/8KFCwIAISQkRKV90qRJAgDh8OHDyjZXV1cBgHDs2DFl2927dwWpVCp8/PHHyrYbN24IAISvvvpKZczAwEDB1dW1TAyzZ88W/vvHfMmSJQIA4d69e8+Nu/QakZGRyramTZsKNWrUEB48eKBsu3jxomBkZCSMGjWqzPVGjx6tMuagQYMEOzu7517zv5/DwsJCEARBeOONN4Ru3boJgiAIJSUlgoODgzBnzpxnfg3y8/OFkpKSMp9DKpUKc+fOVbadOXOmzGcr5evrKwAQVq1a9cxzvr6+Km0HDx4UAAifffaZ8M8//wiWlpbCwIEDX/oZ1aFQKJRx2dvbC8OHDxeWL18u3Lp1q0zf4uJiwdHRUfDx8VFpX7VqlQBAOHjwoEo7ACE0NFTIyMgQzMzMhE2bNgmCIAh79+4VJBKJcPPmTeX380W/X6hyKP37S9pxhiDr8plGD2nHGQIAISsrS9cfU4Wepl1E+ic7OxsAYGVlVa7++/btAwCEhYWptH/88ccAUGatToMGDdCxY0fl6+rVq8PDwwP//PPPK8f8tNK1Oz///DMUCkW53pOamooLFy4gKCgIVatWVbY3btwY3bt3V37O/3rvvfdUXnfs2BEPHjxQfg3LY8SIEThy5AjS0tJw+PBhpKWlPbeqIJVKYWT05K+zkpISPHjwQDn99vvvv5f7mlKpFMHBweXq26NHD7z77ruYO3cuAgICIJPJsHr16nJfqzwkEgkOHjyIzz77DLa2ttiyZQtCQ0Ph6uqKN998U2UKz9jYGMOGDUN8fLxKVS06Ohr29vbo1q3bM69ha2uLnj17YsuWLcr+7dq1U1aPyMCU3uhP04ceYoJDVE5yuRwAXrgV979u3boFIyMj1KtXT6XdwcEBNjY2uHXrlkq7i4tLmTFsbW3x8OHDV4y4rDfffBPt27dHSEgI7O3tMWzYMGzduvWFyU5pnB4eHmXOeXl54f79+8jLy1Npf/qz2NraAoBan6V3796wsrLCjz/+iM2bN6NVq1ZlvpalFAoFlixZAnd3d0ilUlSrVg3Vq1fHH3/8gaysrHJfs2bNmmotKF64cCGqVq2KCxcuYNmyZahRo8ZL33Pv3j2kpaUpj9KpoeeRSqX49NNPkZiYiJSUFGzZsgVt27bF1q1by+ykK11EXLqm5s6dOzh+/DiGDRsGY2Pj515jxIgRiI2NRXJyMmJiYjg9RQaBCQ5ROcnlcjg5OeHy5ctqve/pRb7P87x/gARBeOVrlK4PKWVubo5jx47h0KFDePvtt/HHH3/gzTffRPfu3cv0rYiKfJZSUqkUAQEB2LBhA3bu3PnCf3Q///xzhIWFoVOnTvj+++9x8OBBxMbGomHDhuWuVAF45hqVFzl//rxyUe+lS5fK9Z5WrVrB0dFReahzPx9HR0cMGzYMx44dg7u7O7Zu3aqyNqdFixbw9PRUVmO2bNkCQRDK7J56Wv/+/SGVShEYGIiCgoIyu9fIgIhoDY5+RkWkp/r27Yvr168jPj7+pX1dXV2hUChw9epVlfb09HRkZmZqdArA1tZWZbqi1NNVIgAwMjJCt27dsHjxYvz111+YP38+Dh8+jF9//fWZY5fGmZSUVObclStXUK1aNVhYWFTsAzzHiBEjcP78eeTk5DxzYXap7du3o0uXLli3bh2GDRuGHj16wM/Pr8zXpLzJZnnk5eUhODgYDRo0wDvvvIOIiAicOXPmpe/bvHkzYmNjlceztnW/jKmpKRo3boyioqIyN+EbOXIkLl++jD/++APR0dFwd3dX7h57HnNzcwwcOBBHjhxB9+7dUa1aNbVjItI3THCI1DBlyhRYWFggJCQE6enpZc5fv34dX3/9NYAnUywAyux0Wrx4MQCgT58+Gourbt26yMrKwh9//KFsS01Nxc6dO1X6ZWRklHlv6Q3vnt66XsrR0RFNmzbFhg0bVBKGy5cv45dfflF+Tm3o0qUL5s2bh2+//RYODg7P7WdsbFymOrRt2zb8+++/Km2lidizkkF1TZ06FcnJydiwYQMWL16M2rVrKysgL9K+fXv4+fkpDzc3t+f2vXr1KpKTk8u0Z2ZmIj4+Hra2tqhevbrKudJqzaxZs3DhwoWXVm9KTZo0CbNnz8bMmTPL1Z8qKRGtweGN/ojUULduXURHR+PNN9+El5eXyp2MT506hW3btiEoKAgA0KRJEwQGBmLNmjXIzMyEr68vfvvtN2zYsAEDBw587hbkVzFs2DBMnToVgwYNwvjx4/Ho0SOsXLkS9evXV1lkO3fuXBw7dgx9+vSBq6sr7t69ixUrVqBWrVro0KHDc8f/6quv0KtXL/j4+GDMmDF4/PgxvvnmG1hbWyM8PFxjn+NpRkZGmDFjxkv79e3bF3PnzkVwcDDatWuHS5cuYfPmzWWSh7p168LGxgarVq2ClZUVLCws0KZNG9SpU0etuA4fPowVK1Zg9uzZym3rkZGR6Ny5M2bOnImIiAi1xnueixcvYsSIEejVqxc6duyIqlWr4t9//8WGDRuQkpKCpUuXlpkOrFOnDtq1a4eff/4ZAMqd4DRp0gRNmjTRSNxE+oAJDpGa+vfvjz/++ANfffUVfv75Z6xcuRJSqRSNGzfGokWLMHbsWGXf7777Dm5uboiKisLOnTvh4OCA6dOnY/bs2RqNyc7ODjt37kRYWBimTJmCOnXqYMGCBbh69apKgtO/f3/cvHkT69evx/3791GtWjX4+vpizpw5sLa2fu74fn5+OHDgAGbPno1Zs2bB1NQUvr6++PLLL9VODrThk08+QV5eHqKjo/Hjjz+iefPm2Lt3L6ZNm6bSz9TUFBs2bMD06dPx3nvvobi4GJGRkWp9hpycHIwePRrNmjXDp59+qmzv2LEjJkyYgEWLFiEgIABt27at8Ofq1KkT5s2bh/3792Px4sW4d+8erKys0KxZM3z55ZcYPHjwM983cuRInDp1Cq1bt37uwmwSKW2smdHTNTgSQZ1Vf0RERFTpZGdnw9raGtKu8yAx0eyjWITifBQcnomsrCzlblN9oJ9pFxEREVEFcIqKiIhILEQ0RaWfURERERFVACs4REREYqGNbd16uk2cFRwiIiIyOKzgVHIKhQIpKSmwsrLS6F1aiYjo9RIEATk5OXByclI+PFbztPFoBf2slTDBqeRSUlLg7Oys6zCIiEhDbt++jVq1auk6jEqPCU4lZ2VlBQBoOeMnmMi08zwg0i873q34DeSo8jA2YmVWLHKys1GvjrPy73WtENEaHCY4lVzptJSJzIIJjkjo0420SPuY4IiPVpcbSCRa2Caun79H9XPijIiIiKgCmOAQERGJRemN/jR9qKF27dqQSCRljtDQUABAfn4+QkNDYWdnB0tLSwwePBjp6elqf1QmOERERPTanDlzBqmpqcojNjYWADBkyBAAwMSJE7F7925s27YNR48eRUpKCgICAtS+DtfgEBERiYUeLDKuXr26yusvvvgCdevWha+vL7KysrBu3TpER0eja9euAIDIyEh4eXnh9OnTaNu2/JssWMEhIiKiCsvOzlY5CgoKXvqewsJCfP/99xg9ejQkEgnOnTuHoqIi+Pn5Kft4enrCxcUF8fHxasXDBIeIiEgstLgGx9nZGdbW1spjwYIFLw0nJiYGmZmZCAoKAgCkpaXBzMwMNjY2Kv3s7e2Rlpam1kflFBURERFV2O3bt1VuYyGVSl/6nnXr1qFXr15wcnLSeDxMcIiIiMRCi2tw5HK5WvfpunXrFg4dOoQdO3Yo2xwcHFBYWIjMzEyVKk56ejocHBzUCotTVERERGKhB9vES0VGRqJGjRro06ePsq1FixYwNTVFXFycsi0pKQnJycnw8fFRa3xWcIiIiOi1UigUiIyMRGBgIExM/j8Vsba2xpgxYxAWFoaqVatCLpfjww8/hI+Pj1o7qAAmOEREROKhB9vEAeDQoUNITk7G6NGjy5xbsmQJjIyMMHjwYBQUFMDf3x8rVqxQ+xpMcIiIiOi16tGjBwRBeOY5mUyG5cuXY/ny5RW6BhMcIiIikSh9LIKGB9XseBrCRcZERERkcFjBISIiEglWcIiIiIgqMVZwiIiIxELyv0PTY+ohJjhEREQiwSkqIiIiokqMFRwiIiKRYAWHiIiIqBJjBYeIiEgkWMEhIiIiqsRYwSEiIhIJVnCIiIiIKjFWcIiIiMSCN/ojIiIiQ8MpKiIiIqJKjBUcIiIikZBIoIUKjmaH0xRWcIiIiMjgsIJDREQkEhJoYQ2OnpZwWMEhIiIig8MKDhERkUhwFxURERFRJcYKDhERkViI6EZ/rOAQERGRwWEFh4iISCy0sAZH0NM1OExwiIiIREIbi4w1v+1cMzhFRURERAaHFRwiIiKRYAWHiIiIqBJjBYeIiEgsuE2ciIiIqPJiBYeIiEgkuAaHiIiIqBJjBYeIiEgkxFTBYYJDREQkEmJKcDhFRURERAaHFRwiIiKRYAWHiIiIqBJjBYeIiEgseKM/IiIiosqLFRwiIiKR4BocIiIiokqMFRzSW2+3ccGoti4qbckZjzBm0+8AgN6N7NHVowbqVbeAhdQEA1fGI6+wRBehkpacOn8N334fh4tXkpF+PxsbI0LQ27eJrsMiLVq79Si++T4Odx9ko5F7TXw5eQhaNKyt67AMBis4ItK5c2d89NFHFR4nKCgIAwcOrPA4pOrG/TwMXZugPCZu+0N5TmpijDO3HmLL2Ts6jJC06dHjAjRyr4mIyUN1HQq9Bjt+OYcZS3diakgvHNk0FY3ca2Lwh8txLyNH16EZjNIER9OHPjLIBCcoKAgSiQTvvfdemXOhoaGQSCQICgoCAOzYsQPz5s2r8DW//vprREVFVXgcUqUQBDx8VKQ8svOLled2XkjBj2fvIDGVf/kZKr92DfHJe33RpzOrNmKwIvowRg1sh5H9feDp5ojF04ehiswM3++K13VoVAkZZIIDAM7Ozvjhhx/w+PFjZVt+fj6io6Ph4vL/0x5Vq1aFlZVVha9nbW0NGxubCo9DqpxszPHDmFbYGNQS0/zro7qVVNchEZEWFBYV48KV2+jc2kPZZmRkBN/WHjhz6YYOIzMwEi0deshgE5zmzZvD2dkZO3bsULbt2LEDLi4uaNasmbLt6SmqFStWwN3dHTKZDPb29njjjTeU57Zv3w5vb2+Ym5vDzs4Ofn5+yMvLA1B2iqpz584YP348pkyZgqpVq8LBwQHh4eEqMV65cgUdOnSATCZDgwYNcOjQIUgkEsTExGj0a1FZXUnLwcJf/sb0n//EssPX4CCXYckb3jA3NdZ1aESkYQ8yc1FSokD1qqo/cFavKsfdB9k6iooqM4NeZDx69GhERkZi5MiRAID169cjODgYR44ceWb/s2fPYvz48di0aRPatWuHjIwMHD9+HACQmpqK4cOHIyIiAoMGDUJOTg6OHz8OQRCee/0NGzYgLCwMCQkJiI+PR1BQENq3b4/u3bujpKQEAwcOhIuLCxISEpCTk4OPP/74pZ+poKAABQUFytfZ2Yb7B//MrYfK/7+BR0hMy8Hm0a3gW78aDvyZrsPIiIgqJy4yNhBvvfUWTpw4gVu3buHWrVs4efIk3nrrref2T05OhoWFBfr27QtXV1c0a9YM48ePB/AkwSkuLkZAQABq164Nb29vfPDBB7C0tHzueI0bN8bs2bPh7u6OUaNGoWXLloiLiwMAxMbG4vr169i4cSOaNGmCDh06YP78+S/9TAsWLIC1tbXycHZ2VvOrUnnlFZbgTuZjOFnLdB0KEWmYnY0ljI2NyiwovpeRjRp2ch1FRdry77//4q233oKdnR3Mzc3h7e2Ns2fPKs8LgoBZs2bB0dER5ubm8PPzw9WrV9W6hkEnONWrV0efPn0QFRWFyMhI9OnTB9WqVXtu/+7du8PV1RVubm54++23sXnzZjx69AgA0KRJE3Tr1g3e3t4YMmQI1q5di4cPHz53LOBJgvNfjo6OuHv3LgAgKSkJzs7OcHBwUJ5v3br1Sz/T9OnTkZWVpTxu37790vcYCpmpERytZcjIK9R1KESkYWamJmjq6YyjZ5KUbQqFAsfO/I1W3nV0GJlh0YddVA8fPkT79u1hamqK/fv346+//sKiRYtga2ur7BMREYFly5Zh1apVSEhIgIWFBfz9/ZGfn1/u6xj0FBXwZJpq3LhxAIDly5e/sK+VlRV+//13HDlyBL/88gtmzZqF8PBwnDlzBjY2NoiNjcWpU6fwyy+/4JtvvsGnn36KhIQE1Knz7D98pqamKq8lEgkUCkWFPo9UKoVUKo6Ftu90qI3TNzKQnl0AO0szjGrrAoUC+PXvewAA2yqmqFrFDDVtnlR06lSzwOPCEtzNKUBOQfGLhqZKIvdRAW7cuad8fSvlAS79fQe28iqo5VBVh5GRNnwwois+mLMJzbxc0Lxhbazc8ivyHhdgZL+2ug6NNOjLL7+Es7MzIiMjlW3//XdUEAQsXboUM2bMwIABAwAAGzduhL29PWJiYjBs2LByXcfgE5yePXuisLAQEokE/v7+L+1vYmICPz8/+Pn5Yfbs2bCxscHhw4cREBAAiUSC9u3bo3379pg1axZcXV2xc+dOhIWFqR2Xh4cHbt++jfT0dNjb2wMAzpw5o/Y4hqyapRSf9PSAlcwUWY+LcDklG+O3XkTW4yfJS19vR5UbAS4Z8qRi9tUvf+OXxLs6iZk060JiMgZ+sEz5eubSnQCAYX1a49tZb+sqLNKSgB4tcD8zF5+v3ou7D3LgXb8mti8L5RSVBkmghTU4/9tG9fSa0Of9QL5r1y74+/tjyJAhOHr0KGrWrIkPPvgAY8eOBQDcuHEDaWlp8PPzU77H2toabdq0QXx8PBOcUsbGxkhMTFT+/4vs2bMH//zzDzp16gRbW1vs27cPCoUCHh4eSEhIQFxcHHr06IEaNWogISEB9+7dg5eX1yvF1b17d9StWxeBgYGIiIhATk4OZsyYAUB/F2y9bp8fSHrh+U0JydiUkPyaoiFd6NDCHfcTvtF1GPQavTPUF+8M9dV1GAZLm4uMn14TOnv27DK7hwHgn3/+wcqVKxEWFoZPPvkEZ86cwfjx42FmZobAwECkpaUBgPKH/1L29vbKc+Vh8AkOAMjl5cv+bWxssGPHDoSHhyM/Px/u7u7YsmULGjZsiMTERBw7dgxLly5FdnY2XF1dsWjRIvTq1euVYjI2NkZMTAxCQkLQqlUruLm54auvvkK/fv0gk3ERLRERVS63b99W+ff2ecspFAoFWrZsic8//xwA0KxZM1y+fBmrVq1CYGCgxuIxyATnZXcU/u99Zv67ZbxDhw7P3ULu5eWFAwcOlPuazxrn6fvbeHp64sSJE8rXJ0+eBADUq1fvudchIiJ6Zdq4Md//xpPL5eUqKDg6OqJBgwYqbV5eXvjpp58AQLn5Jj09HY6Ojso+6enpaNq0abnDMuhdVPpu586diI2Nxc2bN3Ho0CG88847aN++PerWravr0IiIiLSiffv2SEpSXYLw999/w9XVFcCTBccODg7K26oAT9b3JCQkwMfHp9zXMcgKTmWRk5ODqVOnIjk5GdWqVYOfnx8WLVqk67CIiMhA6cON/iZOnIh27drh888/x9ChQ/Hbb79hzZo1WLNmjXK8jz76CJ999hnc3d1Rp04dzJw5E05OTmo91JoJjg6NGjUKo0aN0nUYREREr02rVq2wc+dOTJ8+HXPnzkWdOnWwdOlS5VMHAGDKlCnIy8vDO++8g8zMTHTo0AEHDhxQa40qExwiIiKR0IcKDgD07dsXffv2feGYc+fOxdy5c185Lq7BISIiIoPDCg4REZFISCRPDk2PqY+Y4BAREYnEkwRH01NUGh1OYzhFRURERAaHFRwiIiKx0MIUlcZvHKghrOAQERGRwWEFh4iISCT0ZZv468AKDhERERkcVnCIiIhEQkzbxFnBISIiIoPDCg4REZFIGBlJYGSk2ZKLoOHxNIUJDhERkUhwioqIiIioEmMFh4iISCS4TZyIiIioEmMFh4iISCS4BoeIiIioEmMFh4iISCS4BoeIiIioEmMFh4iISCTEVMFhgkNERCQSXGRMREREVImxgkNERCQSEmhhigr6WcJhBYeIiIgMDis4REREIsE1OERERESVGCs4REREIiGmbeKs4BAREZHBYQWHiIhIJMS0BocJDhERkUhwioqIiIioEmMFh4iISCTENEXFCg4REREZHFZwiIiIRIJrcIiIiIgqMVZwiIiIxEILa3D09FmbrOAQERGR4WEFh4iISCS4BoeIiIioEmMFh4iISCTEdB8cJjhEREQiwSkqIiIiokqMFRwiIiKRENMUFSs4REREZHBYwSEiIhIJrsEhIiIiqsRYwSEiIhIJVnCIiIiItCA8PFyZaJUenp6eyvP5+fkIDQ2FnZ0dLC0tMXjwYKSnp6t9HSY4REREIlG6i0rTh7oaNmyI1NRU5XHixAnluYkTJ2L37t3Ytm0bjh49ipSUFAQEBKh9DU5RERERiYS+TFGZmJjAwcGhTHtWVhbWrVuH6OhodO3aFQAQGRkJLy8vnD59Gm3bti33NVjBISIiogrLzs5WOQoKCp7b9+rVq3BycoKbmxtGjhyJ5ORkAMC5c+dQVFQEPz8/ZV9PT0+4uLggPj5erXhYwTEQP4a0hlwu13UY9BpM3p2o6xDoNfqqn5euQ6DXpEQhaP0a2rzRn7Ozs0r77NmzER4eXqZ/mzZtEBUVBQ8PD6SmpmLOnDno2LEjLl++jLS0NJiZmcHGxkblPfb29khLS1MrLiY4REREVGG3b99W+UFbKpU+s1+vXr2U/9+4cWO0adMGrq6u2Lp1K8zNzTUWD6eoiIiIROLp3UuaOgBALperHM9LcJ5mY2OD+vXr49q1a3BwcEBhYSEyMzNV+qSnpz9zzc6LMMEhIiIincnNzcX169fh6OiIFi1awNTUFHFxccrzSUlJSE5Oho+Pj1rjcoqKiIhIJCTQwhocNftPmjQJ/fr1g6urK1JSUjB79mwYGxtj+PDhsLa2xpgxYxAWFoaqVatCLpfjww8/hI+Pj1o7qAAmOERERPQa3blzB8OHD8eDBw9QvXp1dOjQAadPn0b16tUBAEuWLIGRkREGDx6MgoIC+Pv7Y8WKFWpfhwkOERGRSBhJJDDScAlH3fF++OGHF56XyWRYvnw5li9fXpGwmOAQERGJhTa3iesbLjImIiIig8MKDhERkUjoy6MaXgdWcIiIiMjgsIJDREQkEkaSJ4emx9RHrOAQERGRwWEFh4iISCwkWlgzwwoOERER0evBCg4REZFIiOk+OExwiIiIRELyv1+aHlMfcYqKiIiIDA4rOERERCLBbeJERERElRgrOERERCLBRzUQERERVWKs4BAREYmEmLaJs4JDREREBocVHCIiIpEwkkhgpOGSi6bH0xQmOERERCLBKSoiIiKiSowVHCIiIpHgNnEiIiKiSowVHCIiIpEQ0xqcciU4u3btKveA/fv3f+VgiIiIiDShXAnOwIEDyzWYRCJBSUlJReIhIiIiLeE28acoFAptx0FERESkMRVaZJyfn6+pOIiIiEjLJFo69JHaCU5JSQnmzZuHmjVrwtLSEv/88w8AYObMmVi3bp3GAyQiIiLNKN0mrulDH6md4MyfPx9RUVGIiIiAmZmZsr1Ro0b47rvvNBocERER0atQO8HZuHEj1qxZg5EjR8LY2FjZ3qRJE1y5ckWjwREREZHmGEm0c+gjtROcf//9F/Xq1SvTrlAoUFRUpJGgiIiIiCpC7QSnQYMGOH78eJn27du3o1mzZhoJioiIiDRPTGtw1L6T8axZsxAYGIh///0XCoUCO3bsQFJSEjZu3Ig9e/ZoI0YiIiIitahdwRkwYAB2796NQ4cOwcLCArNmzUJiYiJ2796N7t27ayNGIiIi0pDSxzVo6tBXr/Qsqo4dOyI2NlbTsRARERFpxCs/bPPs2bNITEwE8GRdTosWLTQWFBEREWmeNtbMGMwanDt37mD48OE4efIkbGxsAACZmZlo164dfvjhB9SqVUvTMRIREZEGaGNbt8FsEw8JCUFRURESExORkZGBjIwMJCYmQqFQICQkRBsxEhEREalF7QrO0aNHcerUKXh4eCjbPDw88M0336Bjx44aDY6IiIg0R0xTVGpXcJydnZ95Q7+SkhI4OTlpJCgiIiKiilA7wfnqq6/w4Ycf4uzZs8q2s2fPYsKECVi4cKFGgyMiIiLNEdPTxMs1RWVra6tSgsrLy0ObNm1gYvLk7cXFxTAxMcHo0aMxcOBArQRKREREVF7lSnCWLl2q5TCIiIhI24wkEhhpeM2MpsfTlHIlOIGBgdqOg4iIiEhjXvlGfwCQn5+PwsJClTa5XF6hgIiIiEg7tPF4BT0t4Kif4OTl5WHq1KnYunUrHjx4UOZ8SUmJRgIjIiIizeI28ReYMmUKDh8+jJUrV0IqleK7777DnDlz4OTkhI0bN2ojRiIiIiK1qF3B2b17NzZu3IjOnTsjODgYHTt2RL169eDq6orNmzdj5MiR2oiTiIiIKkhMU1RqV3AyMjLg5uYG4Ml6m4yMDABAhw4dcOzYMc1GR0RERAbtiy++gEQiwUcffaRsy8/PR2hoKOzs7GBpaYnBgwcjPT1drXHVruC4ubnhxo0bcHFxgaenJ7Zu3YrWrVtj9+7dyodvEmnLhh0nsGHnCdxOfZJYe9RxxMTR/ujm00DHkZGmdXW3Q98G9jh2/QFiLj/5i83ESIL+jezRrKYcJkZGSLqbi+1/pCK3gGv/DMGp89fw7fdxuHglGen3s7ExIgS9fZvoOiyDom/bxM+cOYPVq1ejcePGKu0TJ07E3r17sW3bNlhbW2PcuHEICAjAyZMnyx+XusEEBwfj4sWLAIBp06Zh+fLlkMlkmDhxIiZPnqzucDpXu3Zt3uenEnGsYYNP3++Hg5GTcGD9JLRv4Y7gqd8h6Z9UXYdGGuRsI4OPqy1SsvJV2gc0skdDeytsOHMHy0/chFxmguBWzjqKkjTt0eMCNHKviYjJQ3UdCr0Gubm5GDlyJNauXQtbW1tle1ZWFtatW4fFixeja9euaNGiBSIjI3Hq1CmcPn263OOrXcGZOHGi8v/9/Pxw5coVnDt3DvXq1SuTgb1uQUFB2LBhAwDA1NQULi4uGDVqFD755BPlXZefdubMGVhYWLzOMKkCenRopPJ6+nt9sXHnSZz78yY83Bx1FBVpkpmxBCNb1MTWi6noXr+asl1mYoQ2rrb4/uwdXLv/CADww/kUTOtWD6625rj18LGuQiYN8WvXEH7tGuo6DIOmzTU42dnZKu1SqRRSqfS57wsNDUWfPn3g5+eHzz77TNl+7tw5FBUVwc/PT9nm6ekJFxcXxMfHo23btuWKq0L3wQEAV1dXuLq6VnQYjenZsyciIyNRUFCAffv2ITQ0FKamppg+fbpKv8LCQpiZmaF69eo6ibP0+vTqSkoU2H34Ah7lF6BFozq6Doc0ZHBjRySm5+LqvTyVBKeWjQwmRhL8fS9P2XY3txAZjwqZ4BDpAWdn1Wrq7NmzER4e/sy+P/zwA37//XecOXOmzLm0tDSYmZmVWfZib2+PtLS0csdTrgRn2bJl5R5w/Pjx5e6rDVKpFA4ODgCA999/Hzt37sSuXbuQlJSEzMxMtGrVCsuXL4dUKsWNGzdQu3ZtfPTRR8rFTRKJBKtWrcLu3btx+PBhuLq6Yv369ahevTpCQkJw5swZNGnSBJs2bULdunUBANevX0dYWBhOnz6NvLw8eHl5YcGCBSrZZ+3atTFmzBhcvXoVMTExCAgIQHJyMho0aIBvv/1W2e/evXuoWbMm9u/fj27dur2+L1wlkng9BX3fWYKCwmJYmEuxfsEYeNRx0HVYpAFNa8pRy0aGJUdvlDknl5qguESB/GKFSntuQQnksgr/rEYkCtq8D87t27dVbvb7vOrN7du3MWHCBMTGxkImk2k0lv8q198KS5YsKddgEolE5wnO08zNzZU3JIyLi4NcLkdsbOwL3zNv3jwsXrwYixcvxtSpUzFixAi4ublh+vTpcHFxwejRozFu3Djs378fwJN5xN69e2P+/PmQSqXYuHEj+vXrh6SkJLi4uCjHXbhwIWbNmoXZs2cDABISEjBu3DgsWrRI+Rvh+++/R82aNdG1a9dnxlZQUICCggLl66dLgmJQ16UGDm2YguzcfOz59QLGf7YZO5aPZ5JTydnITDCokQNWxd9CsULQdThEpCa5XF6upxmcO3cOd+/eRfPmzZVtJSUlOHbsGL799lscPHgQhYWFyMzMVKnipKenKwsY5VGuBOfGjbI/Tek7QRAQFxeHgwcP4sMPP8S9e/dgYWGB77777qVTQ8HBwRg69Mkit6lTp8LHxwczZ86Ev78/AGDChAkIDg5W9m/SpAmaNPn/lf7z5s1TVo7GjRunbO/atSs+/vhj5euaNWti3Lhx+Pnnn5XXi4qKQlBQ0HMz7AULFmDOnDlqfjUMi5mpCerUejK12MTTGRcTk/Hd1qP4auqbOo6MKqKWjTmsZCYI83VTthkbSeBmVwXt61TFmvhkmBgbQWZipFLFsZQaIzu/WBchE1U6RniF3UXlGFMd3bp1w6VLl1TagoOD4enpialTp8LZ2RmmpqaIi4vD4MGDAQBJSUlITk6Gj49Pua9jcHXdPXv2wNLSEkVFRVAoFBgxYgTCw8MRGhoKb2/vcq17+e9iaXt7ewCAt7e3Slt+fj6ys7Mhl8uRm5uL8PBw7N27F6mpqSguLsbjx4+RnJysMm7Lli1VXstkMrz99ttYv349hg4dit9//x2XL1/Grl27nhvb9OnTERYWpnydnZ1dZt5TbBQKAYVF/Aeusrt6Pw8Rh6+rtA1r5oS7uQU4fPUBMh8XoVghoH51C/yRmgMAqG5phqpVzLj+hqic9OFRDVZWVmjUSHXDiIWFBezs7JTtY8aMQVhYGKpWrQq5XI4PP/wQPj4+5V5gDBhggtOlSxesXLkSZmZmcHJyUtk9Vd7dUqampsr/L/3GPatNoXjyU+SkSZMQGxuLhQsXol69ejA3N8cbb7xR5kGkz7p+SEgImjZtijt37iAyMhJdu3Z94aLtl61KN3TzV+5G17ZeqOVgi9xHBdjxyzmcOn8NW5a8p+vQqIIKihVIyylQaSssUeBRYYmyPeHWQ/RvZI9HRSXIL1JgUGMH3Mh4xATHQOQ+KsCNO/eUr2+lPMClv+/AVl4FtRyq6jAyet2WLFkCIyMjDB48GAUFBfD398eKFSvUGsPgEhwLCwvUq1fvtV7z5MmTCAoKwqBBgwA8WZNz8+bNcr3X29sbLVu2xNq1axEdHa2y4JjKevAwB+PnbcbdB1mwsjBHg3pO2LLkPfi29tR1aPQa/Hw5HQKAoFbOMDaSIOluLn76g/dAMhQXEpMx8IP/39Qyc+lOAMCwPq3x7ay3dRWWQZFIACM9fFTDkSNHVF7LZDIsX74cy5cvf+UxDS7B0QV3d3fs2LED/fr1g0QiwcyZM5XVnfIICQnBuHHjYGFhoUyS6NkWfzJC1yHQa7Ti5C2V18UKATv+SMOOP8q/VZQqjw4t3HE/4Rtdh0EGQtNrjURp8eLFsLW1Rbt27dCvXz/4+/urrA5/meHDh8PExATDhw/X6pY5IiISNyOJdg599EoVnOPHj2P16tW4fv06tm/fjpo1a2LTpk2oU6cOOnTooOkYyy0qKkrtc09PJQmC6vbU2rVrl2nr3LmzSlvt2rVx+PBhlT6hoaEvvM5/3b9/H/n5+RgzZsxz+xAREVH5qV3B+emnn+Dv7w9zc3OcP39eeU+WrKwsfP755xoP0JAVFRUhLS0NM2bMQNu2bdWq+hAREamrdBeVpg99pHaC89lnn2HVqlVYu3atys6i9u3b4/fff9docIbu5MmTcHR0xJkzZ7Bq1Spdh0NERGQw1J6iSkpKQqdOncq0W1tbIzMzUxMxicbTU11ERETapI01M/q6BkftCo6DgwOuXbtWpv3EiRNwc3N7xjuIiIhIH5Q+TVzThz5SO8EZO3YsJkyYgISEBEgkEqSkpGDz5s2YNGkS3n//fW3ESERERKQWtaeopk2bBoVCgW7duuHRo0fo1KkTpFIpJk2ahA8//FAbMRIREZEGGEkkMNJwyUXT42mK2gmORCLBp59+ismTJ+PatWvIzc1FgwYNYGlpqY34iIiIiNT2yncyNjMzQ4MGDTQZCxEREWmRPjxN/HVRO8Hp0qXLC/e8P33DOyIiIqLXTe0Ep2nTpiqvi4qKcOHCBVy+fBmBgYGaiouIiIg0TBu7nvR0CY76Cc6SJUue2R4eHo7c3NwKB0RERERUURqbOnvrrbewfv16TQ1HREREGmYEiXInlcYO6GcJ55UXGT8tPj6eT8ImIiLSY5yieoGAgACV14IgIDU1FWfPnsXMmTM1FhgRERHRq1I7wbG2tlZ5bWRkBA8PD8ydOxc9evTQWGBERESkWWJ6FpVaCU5JSQmCg4Ph7e0NW1tbbcVEREREVCFqLTI2NjZGjx49+NRwIiKiSkgigcYXGevrGhy1d1E1atQI//zzjzZiISIiItIItROczz77DJMmTcKePXuQmpqK7OxslYOIiIj0U+kuKk0f+qjca3Dmzp2Ljz/+GL179wYA9O/fX+WRDYIgQCKRoKSkRPNREhEREamh3AnOnDlz8N577+HXX3/VZjxERESkJdxF9QyCIAAAfH19tRYMERERaY/kf780PaY+UmsNzoueIk5ERESkL9S6D079+vVfmuRkZGRUKCAiIiLSDk5RPcecOXPK3MmYiIiISN+oleAMGzYMNWrU0FYsREREpEViquCUew0O198QERFRZaH2LioiIiKqnCQSicYLFvpaACl3gqNQKLQZBxEREZHGqLUGh4iIiCovMa3BYYJDREQkEtp4dpSezlCp/7BNIiIiIn3HCg4REZFIGEkkMNJwyUXT42kKKzhERERkcFjBISIiEgkxLTJmBYeIiIgMDis4REREYqGFXVRgBYeIiIjo9WAFh4iISCSMIIGRhksumh5PU5jgEBERiQRv9EdERERUibGCQ0REJBLcJk5ERERUibGCQ0REJBJ8VAMRERGRFqxcuRKNGzeGXC6HXC6Hj48P9u/frzyfn5+P0NBQ2NnZwdLSEoMHD0Z6erra12GCQ0REJBKlu6g0faijVq1a+OKLL3Du3DmcPXsWXbt2xYABA/Dnn38CACZOnIjdu3dj27ZtOHr0KFJSUhAQEKD2Z+UUFREREb02/fr1U3k9f/58rFy5EqdPn0atWrWwbt06REdHo2vXrgCAyMhIeHl54fTp02jbtm25r8MEh4iISCSMoIU1OP+70V92drZKu1QqhVQqfeF7S0pKsG3bNuTl5cHHxwfnzp1DUVER/Pz8lH08PT3h4uKC+Ph4tRIcTlERERGJhDanqJydnWFtba08FixY8Nw4Ll26BEtLS0ilUrz33nvYuXMnGjRogLS0NJiZmcHGxkalv729PdLS0tT6rKzgEBERUYXdvn0bcrlc+fpF1RsPDw9cuHABWVlZ2L59OwIDA3H06FGNxsMEh4iISCSMoPmpm9LxSndFlYeZmRnq1asHAGjRogXOnDmDr7/+Gm+++SYKCwuRmZmpUsVJT0+Hg4PDK8VFREREpBMKhQIFBQVo0aIFTE1NERcXpzyXlJSE5ORk+Pj4qDUmKzhEREQiIZFIINHwImN1x5s+fTp69eoFFxcX5OTkIDo6GkeOHMHBgwdhbW2NMWPGICwsDFWrVoVcLseHH34IHx8ftRYYA0xwiIiI6DW6e/cuRo0ahdTUVFhbW6Nx48Y4ePAgunfvDgBYsmQJjIyMMHjwYBQUFMDf3x8rVqxQ+zpMcIiIiERC8r9D02OqY926dS88L5PJsHz5cixfvvzVgwITHINhLJHAWE+fB0Ka9VnP+roOgV6jam0+1HUI9JoIJYW6DsGgMMEhIiISCTE9bJMJDhERkYjoZzqiedwmTkRERAaHFRwiIiKReJWnf5dnTH3ECg4REREZHFZwiIiIREIfbvT3urCCQ0RERAaHFRwiIiKR0ObDNvWNvsZFRERE9MpYwSEiIhIJMa3BYYJDREQkEvrwLKrXhVNUREREZHBYwSEiIhIJMU1RsYJDREREBocVHCIiIpHgNnEiIiKiSowVHCIiIpHgGhwiIiKiSowVHCIiIpHgfXCIiIiIKjFWcIiIiERCInlyaHpMfcQEh4iISCSMIIGRhieVND2epnCKioiIiAwOKzhEREQiIaYpKlZwiIiIyOCwgkNERCQSkv/90vSY+ogVHCIiIjI4rOAQERGJBNfgEBEREVVirOAQERGJhEQL98HR1zU4THCIiIhEglNURERERJUYKzhEREQiwQoOERERUSXGCg4REZFI8EZ/RERERJUYKzhEREQiYSR5cmh6TH3ECg4REREZHFZwiIiIREJMa3CY4BAREYkEt4kTERERVWKs4BAREYmEBJqfUtLTAg4rOERERGR4WMEhIiISCW4TJyIiIqrEWMEhIiISCTFtE2cFh4iIiAwOExwiIiKRKL0PjqYPdSxYsACtWrWClZUVatSogYEDByIpKUmlT35+PkJDQ2FnZwdLS0sMHjwY6enpal2HCQ4REZFISLR0qOPo0aMIDQ3F6dOnERsbi6KiIvTo0QN5eXnKPhMnTsTu3buxbds2HD16FCkpKQgICFDrOlyDQ0RERK/NgQMHVF5HRUWhRo0aOHfuHDp16oSsrCysW7cO0dHR6Nq1KwAgMjISXl5eOH36NNq2bVuu6zDBISIiEgkjSGCk4WcrGP2vhpOdna3SLpVKIZVKX/r+rKwsAEDVqlUBAOfOnUNRURH8/PyUfTw9PeHi4oL4+PhyJzicoiIiIqIKc3Z2hrW1tfJYsGDBS9+jUCjw0UcfoX379mjUqBEAIC0tDWZmZrCxsVHpa29vj7S0tHLHwwoOERGRSLzKmpnyjAkAt2/fhlwuV7aXp3oTGhqKy5cv48SJExqOigkOERERaYBcLldJcF5m3Lhx2LNnD44dO4ZatWop2x0cHFBYWIjMzEyVKk56ejocHBzKPT6nqIiIiMRCD7ZRCYKAcePGYefOnTh8+DDq1Kmjcr5FixYwNTVFXFycsi0pKQnJycnw8fEp93VYwaFK5esNv2Dv0T9w9VY6zKWmaOldB7M+6I96rva6Do207NtNh7Bg9R6MGdIJcyeot12U9M/Fn+fAxcmuTPt3245hcsRW1K5ZDfMmDELbpm4wMzVBXHwipi7chnsZOTqIljQpNDQU0dHR+Pnnn2FlZaVcV2NtbQ1zc3NYW1tjzJgxCAsLQ9WqVSGXy/Hhhx/Cx8en3AuMASY4ryQoKAiZmZmIiYnRdSiic+r8NYwe3BFNvVxQXKLA56t2Y+hHK3A8+hNYmL98vpcqpwuJyfh+1yl41XXSdSikIV0Dv4Kx8f//6O9V1wkxyz9EzKHzqCIzw45vQ3H56r8Y8P43AIBP3uuDLYvfRffgRRAEQVdhV3r68KiGlStXAgA6d+6s0h4ZGYmgoCAAwJIlS2BkZITBgwejoKAA/v7+WLFihVrX0ekU1b179/D+++/DxcUFUqkUDg4O8Pf3x8mTJ3UZFumxH5d+gGF92sDTzRGN3Gti2YyRuJP2EH9cua3r0EhL8h4VYNycTYiY8iZsrMx1HQ5pyIPMXNx9kKM8/Ds0wj+37+Hk71fRpokbXBztEDrne/x1PQV/XU/BB+Gb0MzLBZ1a1dd16JWbNu5i/ApTVM86SpMbAJDJZFi+fDkyMjKQl5eHHTt2qLX+BtBxgjN48GCcP38eGzZswN9//41du3ahc+fOePDggS7Dei2Kiop0HYJByM7NBwDYyKvoOBLSlk8Wb0e3dg3QqZWHrkMhLTE1McbQXq2weVc8AEBqZgJBEFBQWKzsk19YDIVCQNsmdXUVJlUyOktwMjMzcfz4cXz55Zfo0qULXF1d0bp1a0yfPh39+/dX9gkJCUH16tUhl8vRtWtXXLx4UWWc3bt3o1WrVpDJZKhWrRoGDRqkPPfw4UOMGjUKtra2qFKlCnr16oWrV68qz0dFRcHGxgYHDx6El5cXLC0t0bNnT6Smpir7lJSUICwsDDY2NrCzs8OUKVPKlEcPHDiADh06KPv07dsX169fV56/efMmJBIJfvzxR/j6+kImk2HNmjWQy+XYvn27ylgxMTGwsLBATg7nmV9GoVBg5tIdaN3YjVMXBurnQ7/j8t93MP3dvroOhbSoT+fGsLY0R/SeBADAmUs38Si/EOEfDoC51BRVZGaYN2EQTEyM4VCt/Lt0qCw9WGP82ugswbG0tISlpSViYmJQUFDwzD5DhgzB3bt3sX//fpw7dw7NmzdHt27dkJGRAQDYu3cvBg0ahN69e+P8+fOIi4tD69atle8PCgrC2bNnsWvXLsTHx0MQBPTu3VulevLo0SMsXLgQmzZtwrFjx5CcnIxJkyYpzy9atAhRUVFYv349Tpw4gYyMDOzcuVMlzry8PISFheHs2bOIi4uDkZERBg0aBIVCodJv2rRpmDBhAhITExEQEIBhw4YhMjJSpU9kZCTeeOMNWFlZPfNrUlBQgOzsbJVDrKYu3IYr/6RizbxAXYdCWvBv+kPM+noHvpn1NmRSU12HQ1r0Vv92OBT/F9LuP7mj7YPMXARNW4eeHRvhzrFFuPXrV7C2MseFxGQoFFx/Q+UjEXS4Wuunn37C2LFj8fjxYzRv3hy+vr4YNmwYGjdujBMnTqBPnz64e/euys2C6tWrhylTpuCdd95Bu3bt4Obmhu+//77M2FevXkX9+vVx8uRJtGvXDgDw4MEDODs7Y8OGDRgyZAiioqIQHByMa9euoW7dJ2XPFStWYO7cucpV3U5OTpg4cSImT54MACguLkadOnXQokWL5y4yvn//PqpXr45Lly6hUaNGuHnzJurUqYOlS5diwoQJyn6//fYb2rVrh9u3b8PR0RF3795FzZo1cejQIfj6+j5z7PDwcMyZM6dM+530h2rdf6Cym7ZwGw4cv4SfV06A6zN2YhiyohLFyzsZgAPH/sCYT9bD2Pj/fw4rKVFAIpHAyEiCG4cXqpwzVDU7fKTrELTK2cEW52Pm4O0pa7H/2KUy56taW6C4RIHs3Me4cuBzLP8+Dt98H/eMkSo/oaQQBZfWIisrS+N/n2dnZ8Pa2hqHLybD0kqzY+fmZKNrExetxF0ROl+Dk5KSgl27dqFnz544cuQImjdvjqioKFy8eBG5ubnKR6WXHjdu3FBO/1y4cAHdunV75tiJiYkwMTFBmzZtlG12dnbw8PBAYmKisq1KlSrK5AaAMtEAnjwfIzU1VWUMExMTtGzZUuVaV69exfDhw+Hm5ga5XI7atWsDAJKTk1X6Pf2+1q1bo2HDhtiwYQMA4Pvvv4erqys6der03K/Z9OnTkZWVpTxu3xbX4lpBEDBt4TbsO/oHdnw7TnTJjZh0aFkfcRun4pfIycqjiaczBvVogV8iJ4siuRGDEf18cO9hDn45+eczz2dk5SE79zE6tqyP6raW2H+8bBJE9Cw63yYuk8nQvXt3dO/eHTNnzkRISAhmz56NDz74AI6Ojjhy5EiZ95Te2dDcvOI7KkxNVUvfEolE7S2I/fr1g6urK9auXQsnJycoFAo0atQIhYWFKv0sLCzKvDckJATLly/HtGnTEBkZieDgYEhe8CC08j68zFBNXbgNO345h41fhsCiigzpD55M0cktZDCXmek4OtIkyyoyeLo5qrRVkZnBVl6lTDtVThKJBCP7tcUPexNQ8lRlckS/tvj7RhruP8xF68Z1sCDsDazY8iuu3bqro2gNgz5sE39ddJ7gPK1BgwaIiYlB8+bNkZaWBhMTE2VF5GmNGzdGXFwcgoODy5zz8vJCcXExEhISVKaokpKS0KBBg3LFYm1tDUdHRyQkJCirKsXFxcr1QP8dc+3atejYsSMAqPVMjbfeegtTpkzBsmXL8NdffyEwkOtJXiRqx5Ov7cDQb1Tal80YiWF92jzrLUSkpzq39oCzY1V8v+t0mXPurjUwK7Q/bOVVkJySgUWRB7Ei+rAOoqTKSmcJzoMHDzBkyBCMHj0ajRs3hpWVFc6ePYuIiAgMGDAAfn5+8PHxwcCBAxEREYH69esjJSVFubC4ZcuWmD17Nrp164a6deti2LBhKC4uxr59+zB16lS4u7tjwIABGDt2LFavXg0rKytMmzYNNWvWxIABA8od54QJE/DFF1/A3d0dnp6eWLx4MTIzM5XnbW1tYWdnhzVr1sDR0RHJycmYNm1auce3tbVFQEAAJk+ejB49eqg8j4PKuhu/TNchkA5t//ZDXYdAGvRrwhXYthr3zHNzvt2FOd/ues0RGT7lvWs0PKY+0ukuqjZt2mDJkiXo1KkTGjVqhJkzZ2Ls2LH49ttvIZFIsG/fPnTq1AnBwcGoX78+hg0bhlu3bsHe/slt+Tt37oxt27Zh165daNq0Kbp27YrffvtNeY3IyEi0aNECffv2hY+PDwRBwL59+8pMS73Ixx9/jLfffhuBgYHw8fGBlZWVylZ0IyMj/PDDDzh37hwaNWqEiRMn4quvvlLrazFmzBgUFhZi9OjRar2PiIhIHWLaJq7TXVT0xKZNmzBx4kSkpKTAzEy9dSSlK+PFtotKzMSyi4qeMPRdVPT/XscuqqN/3NbKLirfxs56t4tK79bgiMmjR4+QmpqKL774Au+++67ayQ0REZFatFFy0dMSDvdZ6lBERAQ8PT3h4OCA6dOn6zocIiIig8EER4fCw8NRVFSEuLg4WFpa6jocIiIycBIt/dJHTHCIiIjI4HANDhERkUhwmzgRERFRJcYKDhERkUiIaBMVExwiIiLREFGGwykqIiIiMjis4BAREYmEmJ4mzgoOERERGRxWcIiIiESC28SJiIiIKjFWcIiIiERCRJuoWMEhIiIiw8MKDhERkViIqITDBIeIiEgkuE2ciIiIqBJjBYeIiEgkuE2ciIiIqBJjBYeIiEgkRLTGmBUcIiIiMjys4BAREYmFiEo4rOAQERGRwWEFh4iISCR4HxwiIiKiSowVHCIiIpEQ031wmOAQERGJhIjWGHOKioiIiAwPKzhERERiIaISDis4REREZHBYwSEiIhIJbhMnIiIiqsRYwSEiIhILLWwT19MCDis4REREZHhYwSEiIhIJEW2iYoJDREQkGiLKcDhFRURERK/VsWPH0K9fPzg5OUEikSAmJkblvCAImDVrFhwdHWFubg4/Pz9cvXpVrWswwSEiIhIJiZZ+qSsvLw9NmjTB8uXLn3k+IiICy5Ytw6pVq5CQkAALCwv4+/sjPz+/3NfgFBURERG9Vr169UKvXr2eeU4QBCxduhQzZszAgAEDAAAbN26Evb09YmJiMGzYsHJdgxUcIiIikSh9mrimDwDIzs5WOQoKCl4pxhs3biAtLQ1+fn7KNmtra7Rp0wbx8fHlHocJDhEREVWYs7MzrK2tlceCBQteaZy0tDQAgL29vUq7vb298lx5cIqKiIhIJLS5ier27duQy+XKdqlUquErqYcVHCIiIqowuVyucrxqguPg4AAASE9PV2lPT09XnisPJjhERERiIdHSoUF16tSBg4MD4uLilG3Z2dlISEiAj49PucfhFBUREZFI6MvTxHNzc3Ht2jXl6xs3buDChQuoWrUqXFxc8NFHH+Gzzz6Du7s76tSpg5kzZ8LJyQkDBw4s9zWY4BAREdFrdfbsWXTp0kX5OiwsDAAQGBiIqKgoTJkyBXl5eXjnnXeQmZmJDh064MCBA5DJZOW+BhMcIiIikZBA808Tf5XhOnfuDEEQnj+mRIK5c+di7ty5rxwX1+AQERGRwWEFh4iISCRE9KxNVnCIiIjI8LCCQ0REJBL/fbSCJsfUR6zgEBERkcFhBYeIiEg0xLMKhwlOJVe6zS4nJ1vHkdDrUlSi0HUI9BoJJYW6DoFek9Lv9Yu2T1eUmKaomOBUcjk5OQAAr3quOo6EiIg0IScnB9bW1roOo9JjglPJOTk54fbt27CysoJEX9NoLcjOzoazs3OZp9eSYeL3W1zE+v0WBAE5OTlwcnLS2jXEM0HFBKfSMzIyQq1atXQdhs6UPrWWxIHfb3ER4/eblRvNYYJDREQkEmJag8Nt4kRERGRwWMGhSkkqlWL27NmQSqW6DoVeA36/xYXfb+2R/O+XpsfURxJBm/vRiIiISOeys7NhbW2Nv5Pvw0rD65pysrNR36UasrKy9GrNFCs4REREYiGibVRMcIiIiERCRPkNFxkTERGR4WGCQzrXuXNnfPTRRxUeJygoCAMHDqzwOKR9tWvXxtKlS3UdBhkI/tkvv9Jt4po+9BETHNKKoKAgSCQSvPfee2XOhYaGQiKRICgoCACwY8cOzJs3r8LX/PrrrxEVFVXhcUg9pd9riUQCMzMz1KtXD3PnzkVxcfFz33PmzBm88847rzFKKo979+7h/fffh4uLC6RSKRwcHODv74+TJ0/qOjQitXENDmmNs7MzfvjhByxZsgTm5uYAgPz8fERHR8PFxUXZr2rVqhq5Hu8Aqjs9e/ZEZGQkCgoKsG/fPoSGhsLU1BTTp09X6VdYWAgzMzNUr15dJ3GWXp+ebfDgwSgsLMSGDRvg5uaG9PR0xMXF4cGDB7oOTeuKiopgamqq6zC0TkzbxFnBIa1p3rw5nJ2dsWPHDmXbjh074OLigmbNminbnp6iWrFiBdzd3SGTyWBvb4833nhDeW779u3w9vaGubk57Ozs4Ofnh7y8PABly9SdO3fG+PHjMWXKFFStWhUODg4IDw9XifHKlSvo0KEDZDIZGjRogEOHDkEikSAmJkajXwtDV/rTvqurK95//334+flh165dyu/J/Pnz4eTkBA8PDwBlp6gkEglWr16Nvn37okqVKvDy8kJ8fDyuXbuGzp07w8LCAu3atcP169eV77l+/ToGDBgAe3t7WFpaolWrVjh06JBKXLVr18a8efMwatQoyOVyvPPOO+jatSvGjRun0u/evXswMzNDXFyc9r5Iei4zMxPHjx/Hl19+iS5dusDV1RWtW7fG9OnT0b9/f2WfkJAQVK9eHXK5HF27dsXFixdVxtm9ezdatWoFmUyGatWqYdCgQcpzDx8+xKhRo2Bra4sqVaqgV69euHr1qvJ8VFQUbGxscPDgQXh5ecHS0hI9e/ZEamqqsk9JSQnCwsJgY2MDOzs7TJkypczTtw8cOIAOHToo+/Tt21fl987NmzchkUjw448/wtfXFzKZDGvWrIFcLsf27dtVxoqJiYGFhYXywcZUeTDBIa0aPXo0IiMjla/Xr1+P4ODg5/Y/e/Ysxo8fj7lz5yIpKQkHDhxAp06dAACpqakYPnw4Ro8ejcTERBw5cgQBAQFl/nL7rw0bNsDCwgIJCQmIiIjA3LlzERsbC+DJX5QDBw5ElSpVkJCQgDVr1uDTTz/V0CcXN3NzcxQWFgIA4uLikJSUhNjYWOzZs+e57ylNRC5cuABPT0+MGDEC7777LqZPn46zZ89CEASVxCQ3Nxe9e/dGXFwczp8/j549e6Jfv35ITk5WGXfhwoVo0qQJzp8/j5kzZyIkJATR0dEoKChQ9vn+++9Rs2ZNdO3aVcNficrD0tISlpaWiImJUfna/NeQIUNw9+5d7N+/H+fOnUPz5s3RrVs3ZGRkAAD27t2LQYMGoXfv3jh//jzi4uLQunVr5fuDgoJw9uxZ7Nq1C/Hx8RAEAb1790ZRUZGyz6NHj7Bw4UJs2rQJx44dQ3JyMiZNmqQ8v2jRIkRFRWH9+vU4ceIEMjIysHPnTpU48/LyEBYWhrNnzyIuLg5GRkYYNGgQFAqFSr9p06ZhwoQJSExMREBAAIYNG6by9xUAREZG4o033oCVldWrfWH1jURLhz4SiLQgMDBQGDBggHD37l1BKpUKN2/eFG7evCnIZDLh3r17woABA4TAwEBBEATB19dXmDBhgiAIgvDTTz8JcrlcyM7OLjPmuXPnBADCzZs3X3jNUr6+vkKHDh1U+rRq1UqYOnWqIAiCsH//fsHExERITU1Vno+NjRUACDt37nz1Dy8y//26KxQKITY2VpBKpcKkSZOEwMBAwd7eXigoKFB5j6urq7BkyRLlawDCjBkzlK/j4+MFAMK6deuUbVu2bBFkMtkLY2nYsKHwzTffqFxn4MCBKn0eP34s2NraCj/++KOyrXHjxkJ4eHi5P7Oh2r59u2BrayvIZDKhXbt2wvTp04WLFy8KgiAIx48fF+RyuZCfn6/ynrp16wqrV68WBEEQfHx8hJEjRz5z7L///lsAIJw8eVLZdv/+fcHc3FzYunWrIAiCEBkZKQAQrl27puyzfPlywd7eXvna0dFRiIiIUL4uKioSatWqpfJn/2n37t0TAAiXLl0SBEEQbty4IQAQli5dqtIvISFBMDY2FlJSUgRBEIT09HTBxMREOHLkyHPHriyysrIEAML1fx8Id3OKNHpc//eBAEDIysrS9cdUwQoOaVX16tXRp08fREVFITIyEn369EG1atWe27979+5wdXWFm5sb3n77bWzevBmPHj0CADRp0gTdunWDt7c3hgwZgrVr1+Lhw4cvvH7jxo1VXjs6OuLu3bsAgKSkJDg7O8PBwUF5/r8/bVL57dmzB5aWlpDJZOjVqxfefPNN5XSgt7d3uda9/Pd7ZW9vr3zvf9vy8/ORnZ0N4EkFZ9KkSfDy8oKNjQ0sLS2RmJhYpoLTsmVLldcymQxvv/021q9fDwD4/fffcfnyZeWidzEbPHgwUlJSsGvXLvTs2RNHjhxB8+bNERUVhYsXLyI3Nxd2dnbKao+lpSVu3LihnP65cOECunXr9syxExMTYWJigjZt2ijb7Ozs4OHhgcTERGVblSpVULduXeXr//6ZzcrKQmpqqsoYJiYmZb7HV69exfDhw+Hm5ga5XI7atWsDwEt/b7Ru3RoNGzbEhg0bADyp7Lm6uiqryIZATAUcJjikdaNHj0ZUVBQ2bNiA0aNHv7CvlZUVfv/9d2zZsgWOjo6YNWsWmjRpgszMTBgbGyM2Nhb79+9HgwYN8M0338DDwwM3btx47nhPLxqUSCRlytRUcV26dMGFCxdw9epVPH78WDk1CED535f57/dK8r99p89qK/3+TZo0CTt37sTnn3+O48eP48KFC/D29lZOjZV61vVDQkIQGxuLO3fuIDIyEl27doWrq6san9hwyWQydO/eHTNnzsSpU6cQFBSE2bNnIzc3F46Ojrhw4YLKkZSUhMmTJwOAcjNBRTzrz6yg5hOF+vXrh4yMDKxduxYJCQlISEgAgHL/3ijdjRkZGYng4GDl7z1DwG3iRBrUs2dPFBYWoqioCP7+/i/tb2JiAj8/P0REROCPP/7AzZs3cfjwYQBP/rJr37495syZg/Pnz8PMzKzM/Ht5eXh44Pbt20hPT1e2nTlz5pXGEjsLCwvUq1cPLi4uMDF5PZszT548iaCgIAwaNAje3t5wcHDAzZs3y/Veb29vtGzZEmvXrkV0dPRLE28xa9CgAfLy8tC8eXOkpaXBxMQE9erVUzlKq7KNGzd+7kJtLy8vFBcXK5MNAHjw4AGSkpLQoEGDcsVibW0NR0dHlTGKi4tx7ty5MmPOmDED3bp1g5eX10srvf/11ltv4datW1i2bBn++usvBAYGlvu9pF+4TZy0ztjYWFmCNjY2fmHfPXv24J9//kGnTp1ga2uLffv2QaFQwMPDAwkJCYiLi0OPHj1Qo0YNJCQk4N69e/Dy8nqluLp37466desiMDAQERERyMnJwYwZMwDAoH5iM1Tu7u7YsWMH+vXrB4lEgpkzZ6pVnQsJCcG4ceNgYWGhstNHrB48eIAhQ4Zg9OjRaNy4MaysrHD27FlERERgwIAB8PPzg4+PDwYOHIiIiAjUr18fKSkpyoXFLVu2xOzZs9GtWzfUrVsXw4YNQ3FxMfbt24epU6fC3d0dAwYMwNixY7F69WpYWVlh2rRpqFmzJgYMGFDuOCdMmIAvvvgC7u7u8PT0xOLFi5GZmak8b2trCzs7O6xZswaOjo5ITk7GtGnTyj2+ra0tAgICMHnyZPTo0QO1atVS58tYCWh+m7i+TlKxgkOvhVwuL9dTZm1sbLBjxw507doVXl5eWLVqFbZs2YKGDRtCLpfj2LFj6N27N+rXr48ZM2Zg0aJF6NWr1yvFZGxsjJiYGOTm5qJVq1YICQlR7qKSyWSvNCa9PosXL4atrS3atWuHfv36wd/fH82bNy/3+4cPHw4TExMMHz6c32882UXVpk0bLFmyBJ06dUKjRo0wc+ZMjB07Ft9++y0kEgn27duHTp06ITg4GPXr18ewYcNw69Yt5Zqpzp07Y9u2bdi1axeaNm2Krl274rffflNeIzIyEi1atEDfvn3h4+MDQRCwb98+te4/8/HHH+Ptt99GYGAgfHx8YGVlpZKgGhkZ4YcffsC5c+fQqFEjTJw4EV999ZVaX4sxY8agsLCQlb1KTiKoO7lJZMBOnjyJDh064Nq1ayoLHcnw3Lx5E3Xr1sWZM2fUSozI8G3atAkTJ05ESkqKwdwYMjs7G9bW1riZmlGuHzbVHbu2Y1VkZWVpfOyK4BQVidrOnTthaWkJd3d3XLt2DRMmTED79u2Z3BiwoqIiPHjwADNmzEDbtm2Z3JDSo0ePkJqaii+++ALvvvuuwSQ3YsUpKhK1nJwchIaGwtPTE0FBQWjVqhV+/vlnXYdFWnTy5Ek4OjrizJkzWLVqla7DIT0SEREBT09PODg4lHnMCFU+nKIiIiIycJyiIiIiIoOljfvW6OumUyY4REREIsGniRMRERFVYqzgEBERiYSYpqhYwSEijQgKCsLAgQOVrzt37oyPPvrotcdx5MgRSCQSlbvbPk0ikSAmJqbcY4aHh6Np06YViuvmzZuQSCS4cOFChcYhovJhgkNkwIKCgiCRSCCRSGBmZoZ69eph7ty5KC4u1vq1d+zYgXnz5pWrb3mSEiKqODE9TZxTVEQGrmfPnoiMjERBQQH27duH0NBQmJqaPvM+H4WFhRq7uVnVqlU1Mg4R0atgBYfIwEmlUjg4OMDV1RXvv/8+/Pz8sGvXLgD/P600f/58ODk5wcPDAwBw+/ZtDB06FDY2NqhatSoGDBig8qTukpIShIWFwcbGBnZ2dpgyZQqevqXW01NUBQUFmDp1KpydnSGVSlGvXj2sW7cON2/eRJcuXQA8edChRCJBUFAQAEChUGDBggWoU6cOzM3N0aRJE2zfvl3lOvv27UP9+vVhbm6OLl26lPuJ4v81depU1K9fH1WqVIGbmxtmzpyJoqKiMv1Wr14NZ2dnVKlSBUOHDkVWVpbK+e+++w5eXl6QyWTw9PTEihUr1I6FSKtEVMJhgkMkMubm5igsLFS+jouLQ1JSEmJjY7Fnzx4UFRXB398fVlZWOH78OE6ePAlLS0v07NlT+b5FixYhKioK69evx4kTJ5CRkYGdO3e+8LqjRo3Cli1bsGzZMiQmJmL16tWwtLSEs7MzfvrpJwBAUlISUlNT8fXXXwMAFixYgI0bN2LVqlX4888/MXHiRLz11ls4evQogCeJWEBAAPr164cLFy4gJCRErSdHl7KyskJUVBT++usvfP3111i7di2WLFmi0ufatWvYunUrdu/ejQMHDuD8+fP44IMPlOc3b96MWbNmYf78+UhMTMTnn3+OmTNnYsOGDWrHQ0QaIBCRwQoMDBQGDBggCIIgKBQKITY2VpBKpcKkSZOU5+3t7YWCggLlezZt2iR4eHgICoVC2VZQUCCYm5sLBw8eFARBEBwdHYWIiAjl+aKiIqFWrVrKawmCIPj6+goTJkwQBEEQkpKSBABCbGzsM+P89ddfBQDCw4cPlW35+flClSpVhFOnTqn0HTNmjDB8+HBBEARh+vTpQoMGDVTOT506tcxYTwMg7Ny587nnv/rqK6FFixbK17NnzxaMjY2FO3fuKNv2798vGBkZCampqYIgCELdunWF6OholXHmzZsn+Pj4CIIgCDdu3BAACOfPn3/udYm0JSsrSwAg/Hs3U8jJV2j0+PdupgBAyMrK0vXHVME1OEQGbs+ePbC0tERRUREUCgVGjBiB8PBw5Xlvb2+VdTcXL17EtWvXYGVlpTJOfn4+rl+/jqysLKSmpqJNmzbKcyYmJmjZsmWZaapSFy5cgLGxMXx9fcsd97Vr1/Do0SN0795dpb2wsBDNmjUDACQmJqrEAQA+Pj7lvkapH3/8EcuWLcP169eRm5uL4uLiMrecd3FxQc2aNVWuo1AokJSUBCsrK1y/fh1jxozB2LFjlX2Ki4thbW2tdjxE2iKmbeJMcIgMXJcuXbBy5UqYmZnByckJJiaqf+wtLCxUXufm5qJFixbYvHlzmbGqV6/+SjGYm5ur/Z7c3FwAwN69e1USC+DJuiJNiY+Px8iRIzFnzhz4+/vD2toaP/zwAxYtWqR2rGvXri2TcBkbG2ssViIqPyY4RAbOwsIC9erVK3f/5s2b48cff0SNGjWe++A8R0dHJCQkoFOnTgCeVCrOnTuH5s2bP7O/t7c3FAoFjh49Cj8/vzLnSytIJSUlyrYGDRpAKpUiOTn5uZUfLy8v5YLpUqdPn375h/yPU6dOwdXVFZ9++qmy7datW2X6JScnIyUlBU5OTsrrGBkZwcPDA/b29nBycsI///yDkSNHqnV9otdJG2uC9bSAw0XGRKRq5MiRqFatGgYMGIDjx4/jxo0bOHLkCMaPH487d+4AACZMmIAvvvgCMTExuHLlCj744IMX3sOmdu3aCAwMxOjRoxETE6Mcc+vWrQAAV1dXSCQS7NmzB/fu3UNubi6srKwwadIkTJw4ERs2bMD169fx+++/45tvvlEu3H3vvfdw9epVTJ48GUlJSYiOjkZUVJRan9fd3R3Jycn44YcfcP36dSxbtuyZC6ZlMhkCAwNx8eJFHD9+HOPHj8fQoUPh4OAAAJgzZw4WLFiAZcuW4e+//8alS5cQGRmJxYsXqxUPEWkGExwiUlGlShUcO3YMLi4uCAgIgJeXF8aMGYP8/HxlRefjjz/G22+/jcDAQPj4+MDKygqDBg164bgrV67EG2+8gQ8++ACenp4YO3Ys8vLyAAA1a9bEnDlzMG3aNNjb22PcuHEAgHnz5mHmzJlYsGABvLy80LNnT+zduxd16tQB8GRdzE8//YSYmBg0adIEq1atwueff67W5+3fvz8mTpyIcePGoWnTpjh16hRmzpxZpl+9evUQEBCA3r17o0ePHmjcuLHKNvCQkBB89913iIyMhLe3N3x9fREVFaWMlUgviGibuER43qpAIiIiMgjZ2dmwtrZG6v3M5049V2Rsx2o2yMrK0vjYFcEKDhERkUhItPTrVSxfvhy1a9eGTCZDmzZt8Ntvv2n0szLBISIiotfqxx9/RFhYGGbPno3ff/8dTZo0gb+/P+7evauxazDBISIiEonS++Bo+lDX4sWLMXbsWAQHB6NBgwZYtWoVqlSpgvXr12vss3KbOBERkUhkZ2drbcynx5ZKpc+8Z1VhYSHOnTun8sBfIyMj+Pn5IT4+XmNxMcEhIiIycGZmZnBwcIB7HWetjF/6XLn/mj17tspd00vdv38fJSUlsLe3V2m3t7fHlStXNBYTExwiIiIDJ5PJcOPGDZUH7WqSIAiQPDVXpck7jr8KJjhEREQiIJPJIJPJdB0GqlWrBmNjY6Snp6u0p6enK2+cqQlcZExERESvjZmZGVq0aIG4uDhlm0KhQFxc3Cs9LPd5WMEhIiKi1yosLAyBgYFo2bIlWrdujaVLlyIvLw/BwcEauwYTHCIiInqt3nzzTdy7dw+zZs1CWloamjZtigMHDpRZeFwRfFQDERERGRyuwSEiIiKDwwSHiIiIDA4THCIiIjI4THCIiIjI4DDBISIiIoPDBIeIiIgMDhMcIiIiMjhMcIiIiMjgMMEhIiIig8MEh4iIiAwOExwiIiIyOP8HNT5OVO3aFYwAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import seaborn as sns\n",
        "import pandas as pd\n",
        "from sklearn.metrics import precision_recall_fscore_support\n",
        "\n",
        "# Ambil metrik per label\n",
        "precision, recall, f1, support = precision_recall_fscore_support(y_test, y_pred, labels=model.classes_)\n",
        "\n",
        "# Masukkan ke DataFrame\n",
        "metrics_df = pd.DataFrame({\n",
        "    'Label': model.classes_,\n",
        "    'Precision': precision,\n",
        "    'Recall': recall,\n",
        "    'F1-Score': f1,\n",
        "    'Support': support\n",
        "})\n",
        "\n",
        "# Plot\n",
        "plt.figure(figsize=(10, 6))\n",
        "sns.barplot(data=metrics_df.melt(id_vars='Label'), x='Label', y='value', hue='variable')\n",
        "plt.title('Evaluasi Metrik per Label (SVM)')\n",
        "plt.ylabel('Score')\n",
        "plt.ylim(0.8, 1.05)\n",
        "plt.legend(title='Metric')\n",
        "plt.show()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 564
        },
        "id": "AKvnLvwPTYc6",
        "outputId": "a4454aa0-3397-453b-d6e0-e181f5be8647"
      },
      "execution_count": 51,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 1000x600 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAA1cAAAIjCAYAAADvBuGTAAAAOnRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjEwLjAsIGh0dHBzOi8vbWF0cGxvdGxpYi5vcmcvlHJYcgAAAAlwSFlzAAAPYQAAD2EBqD+naQAAXPhJREFUeJzt3XlYVdX+x/HPEYSDIOCAoKaI85AjTqSGM46pmTmVgKnZjTK9VupVUSspb2pmpjYIZJqaqZljSjnhUGp0K4ccMK1wTFExJtm/P3o8v06gIm49Iu/X85zndtZee+3v5si+flx7r2MxDMMQAAAAAOC2FHJ0AQAAAABwPyBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBwD3GYrFowoQJji7jnqkjL44dOyaLxaKYmBhbW1hYmDw8PBxX1D0kJiZGFotFu3fvNm3MCRMmyGKx5Lp/p06dNHjwYNOOb7Zz587J3d1da9ascXQpAPIRwhUA5ODaXz6v99q5c6ejS7wnXAsxFotFr776ao59+vfvL4vFkudgs2bNmnwb8sx0P4XD+Ph4ffnll3r55Zft2o8dO6bw8HBVqlRJVqtVfn5+evjhhxUZGSlJOn36tJydnfXEE09cd+xLly7Jzc1Njz76qCT73+Vt27Zl628YhsqVKyeLxaIuXbrY2kuUKKFBgwZp3LhxZpwygALC2dEFAMC9bNKkSQoICMjWXrlyZQdUc3f9+eefcnbO3f9NWK1WffLJJxo7dqxde0pKij7//HNZrdY817FmzRrNmjXrlgKWv7+//vzzTxUuXDjPx8Wd89///ldt2rSx+z06fPiwGjVqJDc3Nw0cOFAVKlRQUlKS9u7dqzfeeEMTJ05UqVKl1K5dO33++ee6cuWKihQpkm3sZcuWKTU1NVsAs1qtWrhwoZo3b27XvnnzZv36669ydXXNNtbQoUP19ttv66uvvlLr1q1NOnsA9zPCFQDcQMeOHdWwYUNHl+EQtxKIOnXqpGXLlun7779X3bp1be2ff/650tPT1aFDB3311Vd3okw7mZmZysrKkouLy20FOkcwDEOpqalyc3NzdCl31OnTp7V69WrNmTPHrn369Om6fPmyEhIS5O/vn22fa/r3769169Zp5cqV6tOnT7bxFy5cKC8vL3Xu3NmuvVOnTvr000/19ttv2/2jwcKFCxUYGKizZ89mG6tGjRp68MEHFRMTQ7gCkCvcFggAeZSRkaHixYsrPDw827aLFy/KarVq5MiRkqT09HSNHz9egYGB8vLykru7u1q0aKGvv/76pscJCwtThQoVsrXn9IxLdHS0WrdurVKlSsnV1VU1a9bU7Nmzs+27e/duhYSEqGTJknJzc1NAQIAGDhxo1+dWnrkKCgpSQECAFi5caNe+YMECdejQQcWLF89xv7Vr16pFixZyd3dX0aJF1blzZ/3000925z5r1ixbPdde0v/fkvjmm2/qrbfeUqVKleTq6qp9+/bl+MxVThISEuTj46OWLVvq8uXL1+137Za8o0ePKiQkRO7u7ipTpowmTZokwzDs+mZlZemtt95SrVq1ZLVa5evrq6efflrnz5+361ehQgV16dJF69evV8OGDeXm5qa5c+fesN6b+eWXX/Svf/1L1apVk5ubm0qUKKFevXrp2LFjOfa/cuWKnn76aZUoUUKenp4aMGBAtjqlm39Ot2L16tXKzMxU27Zt7dqPHDmiBx54IFuwkqRSpUrZ/rtHjx5yd3fP9mdN+iuExcXF6bHHHss2E9W3b1+dO3dOGzZssLWlp6dr6dKl6tev33Xrbdeunb744otsnzMA5IRwBQA3kJycrLNnz9q9zp07J0kqXLiwevTooRUrVig9Pd1uvxUrVigtLc32L+sXL17UBx98oJYtW+qNN97QhAkTdObMGYWEhCghIcG0emfPni1/f3+NGTNGU6dOVbly5fSvf/3LFlCkv/4C2r59ex07dkyjRo3SzJkz1b9//9t+jqxv375atGiR7S+hZ8+e1Zdffnndv7jOnz9fnTt3loeHh9544w2NGzdO+/btU/PmzW1h4Omnn1a7du1s/a+9/i46OlozZ87UkCFDNHXq1OsGuX/69ttv1bp1a9WvX19r16696fNMV69eVYcOHeTr66spU6YoMDBQkZGRtueBrnn66af14osvqlmzZpoxY4bCw8O1YMEChYSEKCMjw67vwYMH1bdvX7Vr104zZsxQvXr1clX7jc5p+/bt6tOnj95++20NHTpUcXFxatmypa5cuZKtf0REhPbv368JEyZowIABWrBggbp3724XJHLzOd2K7du3q0SJEtlClL+/v06cOHHTGU53d3d169ZN69ev1x9//GG3bfHixbp69ar69++fbb8KFSooKChIn3zyia1t7dq1Sk5OznEG7JrAwEBduHAhz2ESQAFjAACyiY6ONiTl+HJ1dbX1W79+vSHJ+OKLL+z279Spk1GxYkXb+8zMTCMtLc2uz/nz5w1fX19j4MCBdu2SjMjISNv70NBQw9/fP1uNkZGRxj8v41euXMnWLyQkxK6W5cuXG5KMb7/99vo/gBzqyEliYqIhyfjvf/9r/Pjjj4YkY+vWrYZhGMasWbMMDw8PIyUlxQgNDTXc3d1t+126dMnw9vY2Bg8ebDfeyZMnDS8vL7v2Z599Ntt5/v3Ynp6exunTp3PcFh0dbWv7ew3btm0zPD09jc6dOxupqak3PMdr+0oynnvuOVtbVlaW0blzZ8PFxcU4c+aMYRiGsXXrVkOSsWDBArv9161bl63d39/fkGSsW7fupsf/Z/3Xk9Pnv2PHDkOS8dFHH9narv35DgwMNNLT023tU6ZMMSQZn3/+uWEYt/Y55fTnMSfNmzc3AgMDs7X/+OOPhpubmyHJqFevnjFs2DBjxYoVRkpKSra+q1evNiQZc+fOtWtv2rSpUbZsWePq1avZzvXbb7813nnnHaNo0aK2n1OvXr2MVq1aGYbx1+fRuXPnbMfavn27IclYvHjxTc8NAJi5AoAbmDVrljZs2GD3Wrt2rW1769atVbJkSS1evNjWdv78eW3YsEG9e/e2tTk5OcnFxUXSX7eN/fHHH8rMzFTDhg21d+9e0+r9+/M612bdgoODdfToUSUnJ0uSvL29JUmrVq3KNpNyO2rVqqU6derYZgYWLlyobt265bjowIYNG3ThwgX17dvXblbQyclJTZo0ydXtktf07NlTPj4+ue7/9ddfKyQkRG3atNGyZctyXMjgeiIiImz/bbFYFBERofT0dG3cuFGS9Omnn8rLy0vt2rWzO6/AwEB5eHhkO6+AgACFhITk+vg38/fPPyMjQ+fOnVPlypXl7e2d45+zIUOG2C368cwzz8jZ2dm2/LiZn9M1586dU7FixbK116pVSwkJCXriiSd07NgxzZgxQ927d5evr6/ef/99u77t27eXj4+P3a2BiYmJ2rlzp/r27atChXL+683jjz+uP//8U6tWrdKlS5e0atWqG94SKMlWa07PZAHAP7GgBQDcQOPGjW+4oIWzs7N69uyphQsXKi0tTa6urlq2bJkyMjLswpUkxcbGaurUqTpw4IBdqMlpNcK8io+PV2RkpHbs2JHtNrDk5GR5eXkpODhYPXv21MSJEzV9+nS1bNlS3bt3V79+/W4paOSkX79+mjp1qoYPH67t27drzJgxOfY7dOiQJF13kQBPT89cH/NWfn6pqanq3LmzAgMDtWTJklyvhihJhQoVUsWKFe3aqlatKkm22+MOHTqk5ORku2eE/u7vCzPcau258eeffyoqKkrR0dH67bff7G7vuxau/65KlSp27z08PFS6dGm785HM+Zz+zrjO80tVq1bV/PnzdfXqVe3bt0+rVq3SlClTNGTIEAUEBNie03J2dlbv3r317rvv6rffflPZsmVtQSunWwKv8fHxUdu2bbVw4UJduXJFV69e1WOPPZarWm/lO7wAFFyEKwC4TX369NHcuXO1du1ade/eXUuWLFH16tXtVs37+OOPFRYWpu7du+vFF19UqVKl5OTkpKioKB05cuSG41/vL3VXr161e3/kyBG1adNG1atX17Rp01SuXDm5uLhozZo1mj59urKysmzjLV26VDt37tQXX3yh9evXa+DAgZo6dap27tx5W9+l1LdvX40ePVqDBw9WiRIl1L59+xz7Xatl/vz58vPzy7b9VkLPrayu5+rqqk6dOunzzz/XunXr7L7XyAxZWVkqVaqUFixYkOP2f86wmb0y4HPPPafo6Gi98MILCgoKkpeXlywWi/r06WP7md8KMz+na0qUKJHjohl/5+TkpNq1a6t27doKCgpSq1attGDBArtFMJ544gm98847+uSTTzRy5Eh98sknqlmz5k2fW+vXr58GDx6skydPqmPHjraZ3Ou5VmvJkiVzdX4ACjbCFQDcpocfflilS5fW4sWL1bx5c3311Vf6z3/+Y9dn6dKlqlixopYtW2YXlv65GEJOihUrpgsXLmRr/+WXX+zef/HFF0pLS9PKlStVvnx5W/v1bt1q2rSpmjZtqtdee00LFy5U//79tWjRIg0aNOimNV1P+fLl1axZM23atMl2i1lOKlWqJOmvVeD+uWrcP5k5Y2CxWLRgwQJ169ZNvXr10tq1a9WyZctc7ZuVlaWjR4/aZqsk6eeff5Yk22qOlSpV0saNG9WsWTOHLKm+dOlShYaGaurUqba21NTUHP/8SH/NTLVq1cr2/vLly0pKSlKnTp0k3drnlFvVq1fXZ599luv+12aOk5KS7NqbNGmiSpUqaeHChWrXrp1++uknvfbaazcdr0ePHnr66ae1c+dOu9t5rycxMVHSX8uyA8DN8MwVANymQoUK6bHHHtMXX3yh+fPnKzMzM9stgU5OTpLsb4fatWuXduzYcdPxK1WqpOTkZP3vf/+ztSUlJWn58uU3PUZycrKio6Pt+p0/fz7bbVnX/rU/LS3tpvXczKuvvqrIyEg999xz1+0TEhIiT09PTZ48Ocfnvs6cOWP7b3d3d0m6bkC4VS4uLlq2bJkaNWqkrl276ptvvsn1vu+8847tvw3D0DvvvKPChQurTZs2kv56pufq1at65ZVXsu2bmZlp2jlcj5OTU7bPdubMmdlmOa9577337H7+s2fPVmZmpjp27Cjp1j6n3AoKCtL58+d19OhRu/atW7fmeIxrz39Vq1Yt27b+/fvru+++U2RkpCwWy02fn5L+uvVx9uzZmjBhgrp27XrT/nv27JGXl5dq1ap1074AwMwVANzA2rVrdeDAgWztDz30kN3zN71799bMmTMVGRmp2rVrZ/tX7i5dumjZsmXq0aOHOnfurMTERM2ZM0c1a9a84fcrSX/ddvjyyy+rR48eev7553XlyhXNnj1bVatWtVukoH379nJxcVHXrl319NNP6/Lly3r//fdVqlQpu3/1j42N1bvvvqsePXqoUqVKunTpkt5//315enraZixuR3BwsIKDg2/Yx9PTU7Nnz9aTTz6pBg0aqE+fPvLx8dHx48e1evVqNWvWzBZkAgMDJUnPP/+8QkJC5OTkdMOls3PDzc1Nq1atUuvWrdWxY0dt3rxZDz744A33sVqtWrdunUJDQ9WkSROtXbtWq1ev1pgxY2y3+wUHB+vpp59WVFSUEhIS1L59exUuXFiHDh3Sp59+qhkzZtz0GZ8bycjI0KuvvpqtvXjx4vrXv/6lLl26aP78+fLy8lLNmjW1Y8cObdy4USVKlMhxvPT0dLVp00aPP/64Dh48qHfffVfNmzfXI488IunWPqfc6ty5s5ydnbVx40YNGTLE1v7GG29oz549evTRR1WnTh1J0t69e/XRRx+pePHieuGFF7KN9cQTT2jSpEn6/PPP1axZsxy/Dy4noaGhua53w4YN6tq1K89cAcgdxy1UCAD3rhstxa5/LPFtGH8ty12uXDlDkvHqq69mGy8rK8uYPHmy4e/vb7i6uhr169c3Vq1aleMy68phCfQvv/zSePDBBw0XFxejWrVqxscff5zj0tcrV6406tSpY1itVqNChQrGG2+8YcybN8+QZCQmJhqGYRh79+41+vbta5QvX95wdXU1SpUqZXTp0sXYvXv3Tev4p78vxX4j11tG/OuvvzZCQkIMLy8vw2q1GpUqVTLCwsLsasnMzDSee+45w8fHx7BYLLZzvtGxb7YU+zVnz541atasafj5+RmHDh26af1Hjhwx2rdvbxQpUsTw9fU1IiMj7Zb9vua9994zAgMDDTc3N6No0aJG7dq1jZdeesn4/fffbX2ut/T3jWq43p/HSpUqGYbx1/L+4eHhRsmSJQ0PDw8jJCTEOHDggOHv72+Ehobaxrr253vz5s3GkCFDjGLFihkeHh5G//79jXPnzmU7dm4+p9wuxW4YhvHII48Ybdq0sWuLj483nn32WePBBx80vLy8jMKFCxvly5c3wsLCjCNHjlx3rEaNGhmSjHfffTfH7X9fiv1Gcvo89u/fb0gyNm7cmKvzAgCLYfCV4wAA3EhYWJiWLl1601lG5M7WrVvVsmVLHThwINuKhfeSF154QVu2bNGePXuYuQKQKzxzBQAA7qoWLVqoffv2mjJliqNLua5z587pgw8+0KuvvkqwApBrPHMFAADuur9/Gfe9qESJEsxUArhlzFwBAAAAgAkcGq62bNmirl27qkyZMrJYLFqxYsVN99m0aZMaNGggV1dXVa5cWTExMXbbJ0yYIIvFYveqXr36nTkBAECBEBMTwywGAOCmHBquUlJSVLduXc2aNStX/RMTE9W5c2e1atVKCQkJeuGFFzRo0CCtX7/erl+tWrWUlJRke23btu1OlA8AAAAANg595qpjx462LyrMjTlz5iggIMD2zfM1atTQtm3bNH36dIWEhNj6OTs7y8/Pz/R6AQAAAOB68tWCFjt27FDbtm3t2kJCQrJ9seChQ4dUpkwZWa1WBQUFKSoqSuXLl7/uuGlpaUpLS7O9z8rK0h9//KESJUqwQhAAAABQgBmGoUuXLqlMmTIqVOjGN/7lq3B18uRJ+fr62rX5+vrq4sWL+vPPP+Xm5qYmTZooJiZG1apVU1JSkiZOnKgWLVroxx9/VNGiRXMcNyoqShMnTrwbpwAAAAAgHzpx4oQeeOCBG/bJV+EqN/5+m2GdOnXUpEkT+fv7a8mSJXrqqady3Gf06NEaMWKE7X1ycrLKly+vEydOyNPT847XDNwN2zrk/hbc/KD5unt7GWfgXsR1AIB0f10L7sZ14OLFiypXrtx1J2r+Ll+FKz8/P506dcqu7dSpU/L09JSbm1uO+3h7e6tq1ao6fPjwdcd1dXWVq6trtnZPT0/CFe4b7s756tf9pvjdBG4d1wEA0v11Lbib14HcPC6Ur77nKigoSHFxcXZtGzZsUFBQ0HX3uXz5so4cOaLSpUvf6fIAAAAAFGAODVeXL19WQkKCEhISJP211HpCQoKOHz8u6a/b9QYMGGDrP3ToUB09elQvvfSSDhw4oHfffVdLlizR8OHDbX1GjhypzZs369ixY9q+fbt69OghJycn9e3b966eGwAAAICCxaFzgrt371arVq1s76899xQaGqqYmBglJSXZgpYkBQQEaPXq1Ro+fLhmzJihBx54QB988IHdMuy//vqr+vbtq3PnzsnHx0fNmzfXzp075ePjc/dODAAAAECB49Bw1bJlSxmGcd3tMTExOe7z3XffXXefRYsWmVHaTRmGoczMTF29evWuHA+3z8nJSc7OziyvDwAAgDvi/nma7S5KT09XUlKSrly54uhScIuKFCmi0qVLy8XFxdGlAAAA4D5DuLpFWVlZSkxMlJOTk8qUKSMXFxdmQvIBwzCUnp6uM2fOKDExUVWqVLnpl8ABAAAAt4JwdYvS09OVlZWlcuXKqUiRIo4uB7fAzc1NhQsX1i+//KL09HRZrVZHlwQAAID7CP90n0fMeuRPfG4AAAC4U/ibJgAAAACYgHAFAAAAACYgXOG2WCwWrVixwtFlAAAAAA5HuLoPhIWFyWKxaOjQodm2Pfvss7JYLAoLC8vVWJs2bZLFYtGFCxdy1T8pKUkdO3a8hWoBAACA+xPh6j5Rrlw5LVq0SH/++aetLTU1VQsXLlT58uVNP156erokyc/PT66urqaPDwAAAOQ3hKv7RIMGDVSuXDktW7bM1rZs2TKVL19e9evXt7VlZWUpKipKAQEBcnNzU926dbV06VJJ0rFjx9SqVStJUrFixexmvFq2bKmIiAi98MILKlmypEJCQiRlvy3w119/Vd++fVW8eHG5u7urYcOG2rVr1x0+ewAAAMDx+J6r+8jAgQMVHR2t/v37S5LmzZun8PBwbdq0ydYnKipKH3/8sebMmaMqVapoy5YteuKJJ+Tj46PmzZvrs88+U8+ePXXw4EF5enrKzc3Ntm9sbKyeeeYZxcfH53j8y5cvKzg4WGXLltXKlSvl5+envXv3Kisr646eNwAAAHAvIFzdR5544gmNHj1av/zyiyQpPj5eixYtsoWrtLQ0TZ48WRs3blRQUJAkqWLFitq2bZvmzp2r4OBgFS9eXJJUqlQpeXt7241fpUoVTZky5brHX7hwoc6cOaNvv/3WNk7lypVNPksAAADg3kS4uo/4+Pioc+fOiomJkWEY6ty5s0qWLGnbfvjwYV25ckXt2rWz2y89Pd3u1sHrCQwMvOH2hIQE1a9f3xasAAAAgIKEcHWfGThwoCIiIiRJs2bNstt2+fJlSdLq1atVtmxZu225WZTC3d39htv/fgshAAAAUNAQru4zHTp0UHp6uiwWi23RiWtq1qwpV1dXHT9+XMHBwTnu7+LiIkm6evXqLR+7Tp06+uCDD/THH38wewUAAIACh9UC7zNOTk7av3+/9u3bJycnJ7ttRYsW1ciRIzV8+HDFxsbqyJEj2rt3r2bOnKnY2FhJkr+/vywWi1atWqUzZ87YZrtyo2/fvvLz81P37t0VHx+vo0eP6rPPPtOOHTtMPUcAAADgXkS4ug95enrK09Mzx22vvPKKxo0bp6ioKNWoUUMdOnTQ6tWrFRAQIEkqW7asJk6cqFGjRsnX19d2i2FuuLi46Msvv1SpUqXUqVMn1a5dW6+//nq2kAcAAADcjyyGYRiOLuJec/HiRXl5eSk5OTlbSElNTVViYqICAgJktVodVCHyqiB/fpsfzvlW0PwqeMtmR5cA5DtcBwBI99e14G5cB26UDf6JmSsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwgbOjC7ifBL740V071p7/Drhrx7odFotFy5cvV/fu3U3tCwAAANxrmLkqQMLCwmSxWGSxWOTi4qLKlStr0qRJyszMvGPHTEpKUseOHU3vCwAAANxrmLkqYDp06KDo6GilpaVpzZo1evbZZ1W4cGGNHj3arl96erpcXFxu+3h+fn53pC8AAABwr2HmqoBxdXWVn5+f/P399cwzz6ht27ZauXKlwsLC1L17d7322msqU6aMqlWrJkk6ceKEHn/8cXl7e6t48eLq1q2bjh07ZjfmvHnzVKtWLbm6uqp06dKKiIiwbbNYLFqxYoWkvwJbRESESpcuLavVKn9/f0VFReXYV5J++OEHtW7dWm5ubipRooSGDBmiy5cv27Zfq/nNN99U6dKlVaJECT377LPKyMgw/wcHAAAA3AThqoBzc3NTenq6JCkuLk4HDx7Uhg0btGrVKmVkZCgkJERFixbV1q1bFR8fLw8PD3Xo0MG2z+zZs/Xss89qyJAh+uGHH7Ry5UpVrlw5x2O9/fbbWrlypZYsWaKDBw9qwYIFqlChQo59U1JSFBISomLFiunbb7/Vp59+qo0bN9oFN0n6+uuvdeTIEX399deKjY1VTEyMYmJiTPv5AAAAALnFbYEFlGEYiouL0/r16/Xcc8/pzJkzcnd31wcffGC7HfDjjz9WVlaWPvjgA1ksFklSdHS0vL29tWnTJrVv316vvvqq/v3vf2vYsGG2sRs1apTjMY8fP64qVaqoefPmslgs8vf3v259CxcuVGpqqj766CO5u7tLkt555x117dpVb7zxhnx9fSVJxYoV0zvvvCMnJydVr15dnTt3VlxcnAYPHmzKzwkAAADILWauCphVq1bJw8NDVqtVHTt2VO/evTVhwgRJUu3ate2es/r+++91+PBhFS1aVB4eHvLw8FDx4sWVmpqqI0eO6PTp0/r999/Vpk2bXB07LCxMCQkJqlatmp5//nl9+eWX1+27f/9+1a1b1xasJKlZs2bKysrSwYMHbW21atWSk5OT7X3p0qV1+vTp3P44AAAAANMwc1XAtGrVSrNnz5aLi4vKlCkjZ+f//yPw9yAjSZcvX1ZgYKAWLFiQbRwfHx8VKnRr2bxBgwZKTEzU2rVrtXHjRj3++ONq27atli5dmreTkVS4cGG79xaLRVlZWXkeDwAAAMgrwlUB4+7uft1nov6pQYMGWrx4sUqVKiVPT88c+1SoUEFxcXFq1apVrsb09PRU79691bt3bz322GPq0KGD/vjjDxUvXtyuX40aNRQTE6OUlBRb6IuPj1ehQoVsi20AAAAA9xJuC8R19e/fXyVLllS3bt20detWJSYmatOmTXr++ef166+/SpImTJigqVOn6u2339ahQ4e0d+9ezZw5M8fxpk2bpk8++UQHDhzQzz//rE8//VR+fn7y9vbO8dhWq1WhoaH68ccf9fXXX+u5557Tk08+aXveCgAAALiXMHNloj3/HeDoEkxVpEgRbdmyRS+//LIeffRRXbp0SWXLllWbNm1sM1mhoaFKTU3V9OnTNXLkSJUsWVKPPfZYjuMVLVpUU6ZM0aFDh+Tk5KRGjRppzZo1Od5eWKRIEa1fv17Dhg1To0aNVKRIEfXs2VPTpk27o+cMAAAA5JXFMAzD0UXcay5evCgvLy8lJydnux0uNTVViYmJCggIkNVqdVCFyKuC/PltfjjY0SWYKnjLZkeXAOQ7XAcASPfXteBuXAdulA3+idsCAQAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwATOji7gfnJ8Uu27dqzy43+4a8cyk8Vi0fLly9W9e3cdO3ZMAQEB+u6771SvXj1HlwYAAADcFmauCpCwsDBZLBZZLBYVLlxYAQEBeumll5Samuro0gAAAIB8j5mrAqZDhw6Kjo5WRkaG9uzZo9DQUFksFr3xxhuOLg0AAADI15i5KmBcXV3l5+encuXKqXv37mrbtq02bNggScrKylJUVJQCAgLk5uamunXraunSpXb7//TTT+rSpYs8PT1VtGhRtWjRQkeOHJEkffvtt2rXrp1KliwpLy8vBQcHa+/evXf9HAEAAABHIFwVYD/++KO2b98uFxcXSVJUVJQ++ugjzZkzRz/99JOGDx+uJ554Qps3b5Yk/fbbb3r44Yfl6uqqr776Snv27NHAgQOVmZkpSbp06ZJCQ0O1bds27dy5U1WqVFGnTp106dIlh50jAAAAcLdwW2ABs2rVKnl4eCgzM1NpaWkqVKiQ3nnnHaWlpWny5MnauHGjgoKCJEkVK1bUtm3bNHfuXAUHB2vWrFny8vLSokWLVLhwYUlS1apVbWO3bt3a7ljvvfeevL29tXnzZnXp0uXunSQAAADgAISrAqZVq1aaPXu2UlJSNH36dDk7O6tnz5766aefdOXKFbVr186uf3p6uurXry9JSkhIUIsWLWzB6p9OnTqlsWPHatOmTTp9+rSuXr2qK1eu6Pjx43f8vAAAAABHI1wVMO7u7qpcubIkad68eapbt64+/PBDPfjgg5Kk1atXq2zZsnb7uLq6SpLc3NxuOHZoaKjOnTunGTNmyN/fX66urgoKClJ6evodOBMAAADg3kK4KsAKFSqkMWPGaMSIEfr555/l6uqq48ePKzg4OMf+derUUWxsrDIyMnKcvYqPj9e7776rTp06SZJOnDihs2fP3tFzAAAAAO4VLGhRwPXq1UtOTk6aO3euRo4cqeHDhys2NlZHjhzR3r17NXPmTMXGxkqSIiIidPHiRfXp00e7d+/WoUOHNH/+fB08eFCSVKVKFc2fP1/79+/Xrl271L9//5vOdgEAAAD3C2auTFR+/A+OLuGWOTs7KyIiQlOmTFFiYqJ8fHwUFRWlo0ePytvbWw0aNNCYMWMkSSVKlNBXX32lF198UcHBwXJyclK9evXUrFkzSdKHH36oIUOGqEGDBipXrpwmT56skSNHOvL0AAAAgLvGYhiG4egi7jUXL16Ul5eXkpOT5enpabctNTVViYmJCggIkNVqdVCFyKuC/Pltfjjn2z3zq+Atmx1dApDvcB0AIN1f14K7cR24UTb4J24LBAAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABM4O7qA+0mzmc3u2rHin4u/a8cCAAAAcHPMXBUgYWFhslgs2V6HDx/Wli1b1LVrV5UpU0YWi0UrVqzI1Zjff/+9HnnkEZUqVUpWq1UVKlRQ7969dfr06Tt7MgAAAMA9hnBVwHTo0EFJSUl2r4CAAKWkpKhu3bqaNWtWrsc6c+aM2rRpo+LFi2v9+vXav3+/oqOjVaZMGaWkpNyxc8jIyLhjYwMAAAB5RbgqYFxdXeXn52f3cnJyUseOHfXqq6+qR48euR4rPj5eycnJ+uCDD1S/fn0FBASoVatWmj59ugICAmz9fvrpJ3Xp0kWenp4qWrSoWrRooSNHjkiSsrKyNGnSJD3wwANydXVVvXr1tG7dOtu+x44dk8Vi0eLFixUcHCyr1aoFCxZIkj744APVqFFDVqtV1atX17vvvmvSTwkAAAC4dTxzhTzz8/NTZmamli9frscee0wWiyVbn99++00PP/ywWrZsqa+++kqenp6Kj49XZmamJGnGjBmaOnWq5s6dq/r162vevHl65JFH9NNPP6lKlSq2cUaNGqWpU6eqfv36toA1fvx4vfPOO6pfv76+++47DR48WO7u7goNDb1rPwMAAADgGsJVAbNq1Sp5eHjY3nfs2FGffvppnsZq2rSpxowZo379+mno0KFq3LixWrdurQEDBsjX11eSNGvWLHl5eWnRokUqXLiwJKlq1aq2Md588029/PLL6tOnjyTpjTfe0Ndff6233nrL7hbFF154QY8++qjtfWRkpKZOnWprCwgI0L59+zR37lzCFQAAAByC2wILmFatWikhIcH2evvtt3O13+TJk+Xh4WF7HT9+XJL02muv6eTJk5ozZ45q1aqlOXPmqHr16vrhhx8kSQkJCWrRooUtWP3dxYsX9fvvv6tZM/tVFps1a6b9+/fbtTVs2ND23ykpKTpy5Iieeuopu5peffVV2+2GAAAAwN3GzFUB4+7ursqVK9/yfkOHDtXjjz9ue1+mTBnbf5coUUK9evVSr169NHnyZNWvX19vvvmmYmNj5ebmZlrd11y+fFmS9P7776tJkyZ2/ZycnEw5HgAAAHCrCFfIleLFi6t48eI37efi4qJKlSrZVgusU6eOYmNjlZGRkW32ytPTU2XKlFF8fLyCg4Nt7fHx8WrcuPF1j+Hr66syZcro6NGj6t+/fx7PCAAAADAX4QqS/poNOnz4sO19YmKiEhISVLx4cZUvXz7HfVatWqVFixapT58+qlq1qgzD0BdffKE1a9YoOjpakhQREaGZM2eqT58+Gj16tLy8vLRz5041btxY1apV04svvqjIyEhVqlRJ9erVU3R0tBISEmwrAl7PxIkT9fzzz8vLy0sdOnRQWlqadu/erfPnz2vEiBHm/WAAAACAXCJcmSj+uXhHl5Bnu3fvVqtWrWzvrwWU0NBQxcTE5LhPzZo1VaRIEf373//WiRMn5OrqqipVquiDDz7Qk08+KemvWwa/+uorvfjiiwoODpaTk5Pq1atne87q+eefV3Jysv7973/r9OnTqlmzplauXGm3UmBOBg0apCJFiui///2vXnzxRbm7u6t27dp64YUXbv+HAQAAAOSBxTAMw9FF3GsuXrwoLy8vJScny9PT025bamqqEhMTFRAQIKvV6qAKkVcF+fPb/HDwzTvlI8FbNju6BCDf4ToAQLq/rgV34zpwo2zwTw5dLXDLli3q2rWrypQpI4vFohUrVtx0n02bNqlBgwZydXVV5cqVc5xVmTVrlipUqCCr1aomTZrom2++Mb94AAAAAPgbh4arlJQU1a1b1+77jG4kMTFRnTt3ti0n/sILL2jQoEFav369rc/ixYs1YsQIRUZGau/evapbt65CQkJ0+vTpO3UaAAAAAODYZ646duyojh075rr/nDlzFBAQoKlTp0qSatSooW3btmn69OkKCQmRJE2bNk2DBw9WeHi4bZ/Vq1dr3rx5GjVqlPknAQAAAADKZ18ivGPHDrVt29auLSQkRDt27JAkpaena8+ePXZ9ChUqpLZt29r65CQtLU0XL160ewEAAADArchX4erkyZPy9fW1a/P19dXFixf1559/6uzZs7p69WqOfU6ePHndcaOiouTl5WV7lStX7o7UDwAAAOD+la/C1Z0yevRoJScn214nTpxwdEkAAAAA8pl89T1Xfn5+OnXqlF3bqVOn5OnpKTc3Nzk5OcnJySnHPn5+ftcd19XVVa6urnekZgAAAAAFQ76auQoKClJcXJxd24YNGxQUFCRJcnFxUWBgoF2frKwsxcXF2foAAAAAwJ3g0HB1+fJlJSQkKCEhQdJfS60nJCTo+PHjkv66XW/AgAG2/kOHDtXRo0f10ksv6cCBA3r33Xe1ZMkSDR8+3NZnxIgRev/99xUbG6v9+/frmWeeUUpKim31QAAAAAC4Exx6W+Du3bvVqlUr2/sRI0ZIkkJDQxUTE6OkpCRb0JKkgIAArV69WsOHD9eMGTP0wAMP6IMPPrAtwy5JvXv31pkzZzR+/HidPHlS9erV07p167ItcnEn3M1vu+Zb6QEAAIB7i0PDVcuWLWUYxnW3x8TE5LjPd999d8NxIyIiFBERcbvl3Xeuhc7Vq1fr1KlTKlasmOrWravx48erWbNmji7vpjZt2qRWrVrp/Pnz8vb2dnQ5AAAAgJ18taAFbk/Pnj2Vnp6u2NhYVaxYUadOnVJcXJzOnTvn6NJuKiMjw9ElAAAAADeUrxa0QN5duHBBW7du1RtvvKFWrVrJ399fjRs31ujRo/XII4/o2LFjslgstuffru1jsVi0adMmSX/NHFksFq1evVp16tSR1WpV06ZN9eOPP9r2iYmJkbe3t1asWKEqVarIarUqJCQk2/L2s2fPVqVKleTi4qJq1app/vz5dtstFotmz56tRx55RO7u7ho8eLDtFtJixYrJYrEoLCzsjvysAAAAgLwgXBUQHh4e8vDw0IoVK5SWlnZbY7344ouaOnWqvv32W/n4+Khr1652M0tXrlzRa6+9po8++kjx8fG6cOGC+vTpY9u+fPlyDRs2TP/+97/1448/6umnn1Z4eLi+/vpru+NMmDBBPXr00A8//KCJEyfqs88+kyQdPHhQSUlJmjFjxm2dBwAAAGAmwlUB4ezsrJiYGMXGxsrb21vNmjXTmDFj9L///e+Wx4qMjFS7du1Uu3ZtxcbG6tSpU1q+fLlte0ZGht555x0FBQUpMDBQsbGx2r59u7755htJ0ptvvqmwsDD961//UtWqVTVixAg9+uijevPNN+2O069fP4WHh6tixYry9/dX8eLFJUmlSpWSn5+fvLy8buMnAgAAAJiLcFWA9OzZU7///rtWrlypDh06aNOmTWrQoEGOC4fcyN+/M6x48eKqVq2a9u/fb2tzdnZWo0aNbO+rV68ub29vW5/9+/dnW0CjWbNmdmNIUsOGDW+pLgAAAMCRCFcFjNVqVbt27TRu3Dht375dYWFhioyMVKFCf/1R+PvqjY5eRMLd3d2hxwcAAABuBeGqgKtZs6ZSUlLk4+MjSUpKSrJt+/viFn+3c+dO23+fP39eP//8s2rUqGFry8zM1O7du23vDx48qAsXLtj61KhRQ/Hx8XZjxsfHq2bNmjes1cXFRZJ09erVXJwZAAAAcHexFHsBce7cOfXq1UsDBw5UnTp1VLRoUe3evVtTpkxRt27d5ObmpqZNm+r1119XQECATp8+rbFjx+Y41qRJk1SiRAn5+vrqP//5j0qWLKnu3bvbthcuXFjPPfec3n77bTk7OysiIkJNmzZV48aNJf21IMbjjz+u+vXrq23btvriiy+0bNkybdy48Ybn4O/vL4vFolWrVqlTp05yc3OTh4eHaT8jAAAA4HYQrkwUvGWzo0u4Lg8PDzVp0kTTp0/XkSNHlJGRoXLlymnw4MEaM2aMJGnevHl66qmnFBgYqGrVqmnKlClq3759trFef/11DRs2TIcOHVK9evX0xRdf2GaVJKlIkSJ6+eWX1a9fP/32229q0aKFPvzwQ9v27t27a8aMGXrzzTc1bNgwBQQEKDo6Wi1btrzhOZQtW1YTJ07UqFGjFB4ergEDBtzy82IAAADAnWIx/v6QDSRJFy9elJeXl5KTk+Xp6Wm3LTU1VYmJiQoICJDVanVQhY6xadMmtWrVSufPn5e3t3eOfWJiYvTCCy/owoULd7W23CrIn9/mh4MdXYKp7uV/zADuVVwHAEj317XgblwHbpQN/olnrgAAAADABIQrAAAAADAB4Qq51rJlSxmGcd1bAiUpLCzsnr0lEAAAALiTCFcAAAAAYALCVR6xDkj+xOcGAACAO4VwdYsKFy4sSbpy5YqDK0FeXPvcrn2OAAAAgFn4nqtb5OTkJG9vb50+fVrSX9/pZLFYHFwVbsYwDF25ckWnT5+Wt7e3nJycHF0SAAB2Al/8yNElmGbPfwc4ugTAIQhXeeDn5ydJtoCF/MPb29v2+QEAAABmIlzlgcViUenSpVWqVCllZGQ4uhzkUuHChZmxAgAAwB1DuLoNTk5O/GUdAAAAgCQWtAAAAAAAUxCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATODu6ANxfjk+q7egSTFN+/A+OLgEAAAD5CDNXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAAAAJiAcAUAAAAAJiBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAAAAJiAcAUAAAAAJiBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAAAAJiAcAUAAAAAJiBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAAAAJjA2dEFAAAAAPeyZjObOboEU00mAtwxzFwBAAAAgAkIVwAAAABgAsIVAAAAAJiAcAUAAAAAJiBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACRwermbNmqUKFSrIarWqSZMm+uabb67bNyMjQ5MmTVKlSpVktVpVt25drVu3zq7PhAkTZLFY7F7Vq1e/06cBAAAAoIBzaLhavHixRowYocjISO3du1d169ZVSEiITp8+nWP/sWPHau7cuZo5c6b27dunoUOHqkePHvruu+/s+tWqVUtJSUm217Zt2+7G6QAAAAAowBwarqZNm6bBgwcrPDxcNWvW1Jw5c1SkSBHNmzcvx/7z58/XmDFj1KlTJ1WsWFHPPPOMOnXqpKlTp9r1c3Z2lp+fn+1VsmTJu3E6AAAAAAowh4Wr9PR07dmzR23btv3/YgoVUtu2bbVjx44c90lLS5PVarVrc3NzyzYzdejQIZUpU0YVK1ZU//79dfz48RvWkpaWposXL9q9AAAAAOBWOCxcnT17VlevXpWvr69du6+vr06ePJnjPiEhIZo2bZoOHTqkrKwsbdiwQcuWLVNSUpKtT5MmTRQTE6N169Zp9uzZSkxMVIsWLXTp0qXr1hIVFSUvLy/bq1y5cuacJAAAAIACw+ELWtyKGTNmqEqVKqpevbpcXFwUERGh8PBwFSr0/6fRsWNH9erVS3Xq1FFISIjWrFmjCxcuaMmSJdcdd/To0UpOTra9Tpw4cTdOBwAAAMB9xGHhqmTJknJyctKpU6fs2k+dOiU/P78c9/Hx8dGKFSuUkpKiX375RQcOHJCHh4cqVqx43eN4e3uratWqOnz48HX7uLq6ytPT0+4FAAAAALfCYeHKxcVFgYGBiouLs7VlZWUpLi5OQUFBN9zXarWqbNmyyszM1GeffaZu3bpdt+/ly5d15MgRlS5d2rTaAQAAAOCfHHpb4IgRI/T+++8rNjZW+/fv1zPPPKOUlBSFh4dLkgYMGKDRo0fb+u/atUvLli3T0aNHtXXrVnXo0EFZWVl66aWXbH1GjhypzZs369ixY9q+fbt69OghJycn9e3b966fHwAAAICCw9mRB+/du7fOnDmj8ePH6+TJk6pXr57WrVtnW+Ti+PHjds9TpaamauzYsTp69Kg8PDzUqVMnzZ8/X97e3rY+v/76q/r27atz587Jx8dHzZs3186dO+Xj43O3Tw8AAABAAeLQcCVJERERioiIyHHbpk2b7N4HBwdr3759Nxxv0aJFZpUGAAAAALmWr1YLBAAAAIB7FeEKAAAAAExAuAIAAAAAEzj8mSvgXtVsZjNHl2Cqyfy6AwAA3FHMXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAAAAJiAcAUAAAAAJiBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABggtsKV+np6Tp48KAyMzPNqgcAAAAA8qU8hasrV67oqaeeUpEiRVSrVi0dP35ckvTcc8/p9ddfN7VAAAAAAMgP8hSuRo8ere+//16bNm2S1Wq1tbdt21aLFy82rTgAAAAAyC+c87LTihUrtHjxYjVt2lQWi8XWXqtWLR05csS04gAAAAAgv8jTzNWZM2dUqlSpbO0pKSl2YQsAAAAACoo8hauGDRtq9erVtvfXAtUHH3ygoKAgcyoDAAAAgHwkT7cFTp48WR07dtS+ffuUmZmpGTNmaN++fdq+fbs2b95sdo0AAAAAcM/L08xV8+bN9f333yszM1O1a9fWl19+qVKlSmnHjh0KDAw0u0YAAAAAuOfd8sxVRkaGnn76aY0bN07vv//+nagJAAAAAPKdWw5XhQsX1meffaZx48bdiXoAAACQzx2fVNvRJZirmKejK0A+kafbArt3764VK1aYXAoAAAAA5F95WtCiSpUqmjRpkuLj4xUYGCh3d3e77c8//7wpxQEAAABAfpGncPXhhx/K29tbe/bs0Z49e+y2WSwWwhUAAACAAidP4SoxMdHsOgAAAAAgX8vTM1d/ZxiGDMMwoxYAAAAAyLfyHK4++ugj1a5dW25ubnJzc1OdOnU0f/58M2sDAAAAgHwjT7cFTps2TePGjVNERISaNWsmSdq2bZuGDh2qs2fPavjw4aYWCQAAAAD3ujyFq5kzZ2r27NkaMGCAre2RRx5RrVq1NGHCBMIVAAAAgAInT7cFJiUl6aGHHsrW/tBDDykpKem2iwIAAACA/CZP4apy5cpasmRJtvbFixerSpUqt10UAAAAAOQ3ebotcOLEierdu7e2bNlie+YqPj5ecXFxOYYuAAAAALjf5WnmqmfPntq1a5dKliypFStWaMWKFSpZsqS++eYb9ejRw+waAQAAAOCel6eZK0kKDAzUxx9/bGYtAAAAAJBv5Wnmas2aNVq/fn229vXr12vt2rW3XRQAAAAA5Dd5ClejRo3S1atXs7UbhqFRo0bddlEAAAAAkN/kKVwdOnRINWvWzNZevXp1HT58+LaLAgAAAID8Jk/hysvLS0ePHs3WfvjwYbm7u992UQAAAACQ3+QpXHXr1k0vvPCCjhw5Yms7fPiw/v3vf+uRRx4xrTgAAAAAyC/yFK6mTJkid3d3Va9eXQEBAQoICFD16tVVokQJvfnmm2bXCAAAAAD3vDwtxe7l5aXt27drw4YN+v777+Xm5qa6deuqRYsWZtcHAAAAAPnCLc1c7dixQ6tWrZIkWSwWtW/fXqVKldKbb76pnj17asiQIUpLS7sjhQIAAADAveyWwtWkSZP0008/2d7/8MMPGjx4sNq1a6dRo0bpiy++UFRUlOlFAgAAAMC97pbCVUJCgtq0aWN7v2jRIjVu3Fjvv/++RowYobfffltLliwxvUgAAAAAuNfdUrg6f/68fH19be83b96sjh072t43atRIJ06cMK86AAAAAMgnbilc+fr6KjExUZKUnp6uvXv3qmnTprbtly5dUuHChc2tEAAAAADygVsKV506ddKoUaO0detWjR49WkWKFLFbIfB///ufKlWqZHqRAAAAAHCvu6Wl2F955RU9+uijCg4OloeHh2JjY+Xi4mLbPm/ePLVv3970IgEAAADgXndL4apkyZLasmWLkpOT5eHhIScnJ7vtn376qTw8PEwtEAAAAADygzx/iXBOihcvflvFAAAAAEB+dUvPXAEAAAAAcka4AgAAAAATEK4AAAAAwASEKwAAAAAwQZ4WtIB5Al/8yNElmGp5UUdXAAAAADgGM1cAAAAAYALCFQAAAACYgNsCAQCmOj6ptqNLMFX58T84ugQAQD7BzBUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAAAAJiAcAUAAAAAJiBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACh4erWbNmqUKFCrJarWrSpIm++eab6/bNyMjQpEmTVKlSJVmtVtWtW1fr1q27rTEBAAAAwAwODVeLFy/WiBEjFBkZqb1796pu3boKCQnR6dOnc+w/duxYzZ07VzNnztS+ffs0dOhQ9ejRQ999912exwQAAAAAMzg0XE2bNk2DBw9WeHi4atasqTlz5qhIkSKaN29ejv3nz5+vMWPGqFOnTqpYsaKeeeYZderUSVOnTs3zmAAAAABgBoeFq/T0dO3Zs0dt27b9/2IKFVLbtm21Y8eOHPdJS0uT1Wq1a3Nzc9O2bdvyPOa1cS9evGj3AgAAAIBb4bBwdfbsWV29elW+vr527b6+vjp58mSO+4SEhGjatGk6dOiQsrKytGHDBi1btkxJSUl5HlOSoqKi5OXlZXuVK1fuNs8OAAAAQEHj8AUtbsWMGTNUpUoVVa9eXS4uLoqIiFB4eLgKFbq90xg9erSSk5NtrxMnTphUMQAAAICCwtlRBy5ZsqScnJx06tQpu/ZTp07Jz88vx318fHy0YsUKpaam6ty5cypTpoxGjRqlihUr5nlMSXJ1dZWrq+ttnhEA4H7UbGYzR5dgmsmO+799ACgQHDZz5eLiosDAQMXFxdnasrKyFBcXp6CgoBvua7VaVbZsWWVmZuqzzz5Tt27dbntMAAAAALgdDv0nrBEjRig0NFQNGzZU48aN9dZbbyklJUXh4eGSpAEDBqhs2bKKioqSJO3atUu//fab6tWrp99++00TJkxQVlaWXnrppVyPCQAAAAB3gkPDVe/evXXmzBmNHz9eJ0+eVL169bRu3TrbghTHjx+3e54qNTVVY8eO1dGjR+Xh4aFOnTpp/vz58vb2zvWYAAAAAHAnOPzm64iICEVEROS4bdOmTXbvg4ODtW/fvtsaEwAAAADuhHy1WiAAAAAA3KsIVwAAAABgAsIVAAAAAJiAcAUAAAAAJiBcAQAAAIAJCFcAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAmcHV0AABR0gS9+5OgSTLW8qKMrAADAMZi5AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMIHDw9WsWbNUoUIFWa1WNWnSRN98880N+7/11luqVq2a3NzcVK5cOQ0fPlypqam27RMmTJDFYrF7Va9e/U6fBgAAAIACztmRB1+8eLFGjBihOXPmqEmTJnrrrbcUEhKigwcPqlSpUtn6L1y4UKNGjdK8efP00EMP6eeff1ZYWJgsFoumTZtm61erVi1t3LjR9t7Z2aGnCQAAAKAAcOjM1bRp0zR48GCFh4erZs2amjNnjooUKaJ58+bl2H/79u1q1qyZ+vXrpwoVKqh9+/bq27dvttkuZ2dn+fn52V4lS5a8G6cDAAAAoABzWLhKT0/Xnj171LZt2/8vplAhtW3bVjt27Mhxn4ceekh79uyxhamjR49qzZo16tSpk12/Q4cOqUyZMqpYsaL69++v48eP37CWtLQ0Xbx40e4FAAAAALfCYffLnT17VlevXpWvr69du6+vrw4cOJDjPv369dPZs2fVvHlzGYahzMxMDR06VGPGjLH1adKkiWJiYlStWjUlJSVp4sSJatGihX788UcVLVo0x3GjoqI0ceJE804OAAAAQIHj8AUtbsWmTZs0efJkvfvuu9q7d6+WLVum1atX65VXXrH16dixo3r16qU6deooJCREa9as0YULF7RkyZLrjjt69GglJyfbXidOnLgbpwMAAADgPuKwmauSJUvKyclJp06dsms/deqU/Pz8ctxn3LhxevLJJzVo0CBJUu3atZWSkqIhQ4boP//5jwoVyp4Vvb29VbVqVR0+fPi6tbi6usrV1fU2zgYAAABAQeewmSsXFxcFBgYqLi7O1paVlaW4uDgFBQXluM+VK1eyBSgnJydJkmEYOe5z+fJlHTlyRKVLlzapcgAAAADIzqFrlI8YMUKhoaFq2LChGjdurLfeekspKSkKDw+XJA0YMEBly5ZVVFSUJKlr166aNm2a6tevryZNmujw4cMaN26cunbtagtZI0eOVNeuXeXv76/ff/9dkZGRcnJyUt++fR12ngAAAADufw4NV71799aZM2c0fvx4nTx5UvXq1dO6detsi1wcP37cbqZq7NixslgsGjt2rH777Tf5+Pioa9eueu2112x9fv31V/Xt21fnzp2Tj4+Pmjdvrp07d8rHx+eunx8AAACAgsPh364bERGhiIiIHLdt2rTJ7r2zs7MiIyMVGRl53fEWLVpkZnkAAAAAkCv5arVAAAAAALhXEa4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABIQrAAAAADAB4QoAAAAATEC4AgAAAAATEK4AAAAAwASEKwAAAAAwAeEKAAAAAExAuAIAAAAAExCuAAAAAMAEhCsAAAAAMAHhCgAAAABMQLgCAAAAABMQrgAAAADABA4PV7NmzVKFChVktVrVpEkTffPNNzfs/9Zbb6latWpyc3NTuXLlNHz4cKWmpt7WmAAAAABwuxwarhYvXqwRI0YoMjJSe/fuVd26dRUSEqLTp0/n2H/hwoUaNWqUIiMjtX//fn344YdavHixxowZk+cxAQAAAMAMDg1X06ZN0+DBgxUeHq6aNWtqzpw5KlKkiObNm5dj/+3bt6tZs2bq16+fKlSooPbt26tv3752M1O3OiYAAAAAmMHZUQdOT0/Xnj17NHr0aFtboUKF1LZtW+3YsSPHfR566CF9/PHH+uabb9S4cWMdPXpUa9as0ZNPPpnnMSUpLS1NaWlptvfJycmSpIsXL97WOebG1bQ/7/gx7qZLha86ugTTZP6Z6egSTJVyf53OXfn9vFu4Dtzb7qdrAdeBe9v9dC3gOnBvu5+uBXfjOnDtGIZh3LSvw8LV2bNndfXqVfn6+tq1+/r66sCBAznu069fP509e1bNmzeXYRjKzMzU0KFDbbcF5mVMSYqKitLEiROztZcrV+5WT6vAe9DRBeC6Oju6ALN5eTm6AlwH14F7F9cB3C1cB+5t99W14C5eBy5duiSvmxzPYeEqLzZt2qTJkyfr3XffVZMmTXT48GENGzZMr7zyisaNG5fncUePHq0RI0bY3mdlZemPP/5QiRIlZLFYzCgd+czFixdVrlw5nThxQp6eno4uB4CDcC0AwHUAhmHo0qVLKlOmzE37OixclSxZUk5OTjp16pRd+6lTp+Tn55fjPuPGjdOTTz6pQYMGSZJq166tlJQUDRkyRP/5z3/yNKYkubq6ytXV1a7N29s7D2eF+42npycXUgBcCwBwHSjgbjZjdY3DFrRwcXFRYGCg4uLibG1ZWVmKi4tTUFBQjvtcuXJFhQrZl+zk5CTpr0SZlzEBAAAAwAwOvS1wxIgRCg0NVcOGDdW4cWO99dZbSklJUXh4uCRpwIABKlu2rKKioiRJXbt21bRp01S/fn3bbYHjxo1T165dbSHrZmMCAAAAwJ3g0HDVu3dvnTlzRuPHj9fJkydVr149rVu3zrYgxfHjx+1mqsaOHSuLxaKxY8fqt99+k4+Pj7p27arXXnst12MCueHq6qrIyMhst4sCKFi4FgDgOoBbYTFys6YgAAAAAOCGHPolwgAAAABwvyBcAQAAAIAJCFcAAAAAYALCFQqEli1b6oUXXrjtccLCwtS9e/fbHgeAY1SoUEFvvfWWo8sAcB/j7woFG+EK+VZYWJgsFouGDh2abduzzz4ri8WisLAwSdKyZcv0yiuv3PYxZ8yYoZiYmNseB8Dtu3YNsFgscnFxUeXKlTVp0iRlZmZed59vv/1WQ4YMuYtVAsirM2fO6JlnnlH58uXl6uoqPz8/hYSEKD4+3tGlAdfl0KXYgdtVrlw5LVq0SNOnT5ebm5skKTU1VQsXLlT58uVt/YoXL27K8XL77dwA7o4OHTooOjpaaWlpWrNmjZ599lkVLlxYo0ePtuuXnp4uFxcX+fj4OKTOa8cHkHs9e/ZUenq6YmNjVbFiRZ06dUpxcXE6d+6co0u74zIyMlS4cGFHl4E8YOYK+VqDBg1Urlw5LVu2zNa2bNkylS9fXvXr17e1/fO2wHfffVdVqlSR1WqVr6+vHnvsMdu2pUuXqnbt2nJzc1OJEiXUtm1bpaSkSMo+1d+yZUs9//zzeumll1S8eHH5+flpwoQJdjUeOHBAzZs3l9VqVc2aNbVx40ZZLBatWLHC1J8FUBBd+9dsf39/PfPMM2rbtq1Wrlxp+1197bXXVKZMGVWrVk1S9tsCLRaL5s6dqy5duqhIkSKqUaOGduzYocOHD6tly5Zyd3fXQw89pCNHjtj2OXLkiLp16yZfX195eHioUaNG2rhxo11dFSpU0CuvvKIBAwbI09NTQ4YMUevWrRUREWHX78yZM3JxcVFcXNyd+yEB+dCFCxe0detWvfHGG2rVqpX8/f3VuHFjjR49Wo888oitz6BBg+Tj4yNPT0+1bt1a33//vd04X3zxhRo1aiSr1aqSJUuqR48etm3nz5/XgAEDVKxYMRUpUkQdO3bUoUOHbNtjYmLk7e2t9evXq0aNGvLw8FCHDh2UlJRk63P16lWNGDFC3t7eKlGihF566SX981uO1q1bp+bNm9v6dOnSxe6acuzYMVksFi1evFjBwcGyWq1677335OnpqaVLl9qNtWLFCrm7u+vSpUu3/0PGHUG4Qr43cOBARUdH297PmzdP4eHh1+2/e/duPf/885o0aZIOHjyodevW6eGHH5YkJSUlqW/fvho4cKD279+vTZs26dFHH812ofy72NhYubu7a9euXZoyZYomTZqkDRs2SPrrotu9e3cVKVJEu3bt0nvvvaf//Oc/Jp05gH9yc3NTenq6JCkuLk4HDx7Uhg0btGrVquvucy0EJSQkqHr16urXr5+efvppjR49Wrt375ZhGHah6PLly+rUqZPi4uL03XffqUOHDuratauOHz9uN+6bb76punXr6rvvvtO4ceM0aNAgLVy4UGlpabY+H3/8scqWLavWrVub/JMA8jcPDw95eHhoxYoVdr8zf9erVy+dPn1aa9eu1Z49e9SgQQO1adNGf/zxhyRp9erV6tGjhzp16qTvvvtOcXFxaty4sW3/sLAw7d69WytXrtSOHTtkGIY6deqkjIwMW58rV67ozTff1Pz587VlyxYdP35cI0eOtG2fOnWqYmJiNG/ePG3btk1//PGHli9fbldnSkqKRowYod27dysuLk6FChVSjx49lJWVZddv1KhRGjZsmPbv369HH31Uffr0sfv7jSRFR0frscceU9GiRfP2g8WdZwD5VGhoqNGtWzfj9OnThqurq3Hs2DHj2LFjhtVqNc6cOWN069bNCA0NNQzDMIKDg41hw4YZhmEYn332meHp6WlcvHgx25h79uwxJBnHjh274TGvCQ4ONpo3b27Xp1GjRsbLL79sGIZhrF271nB2djaSkpJs2zds2GBIMpYvX573kwdg9/uYlZVlbNiwwXB1dTVGjhxphIaGGr6+vkZaWprdPv7+/sb06dNt7yUZY8eOtb3fsWOHIcn48MMPbW2ffPKJYbVab1hLrVq1jJkzZ9odp3v37nZ9/vzzT6NYsWLG4sWLbW116tQxJkyYkOtzBgqSpUuXGsWKFTOsVqvx0EMPGaNHjza+//57wzAMY+vWrYanp6eRmppqt0+lSpWMuXPnGoZhGEFBQUb//v1zHPvnn382JBnx8fG2trNnzxpubm7GkiVLDMMwjOjoaEOScfjwYVufWbNmGb6+vrb3pUuXNqZMmWJ7n5GRYTzwwAN2f1f4pzNnzhiSjB9++MEwDMNITEw0JBlvvfWWXb9du3YZTk5Oxu+//24YhmGcOnXKcHZ2NjZt2nTdseF4zFwh3/Px8VHnzp0VExOj6Ohode7cWSVLlrxu/3bt2snf318VK1bUk08+qQULFujKlSuSpLp166pNmzaqXbu2evXqpffff1/nz5+/4fHr1Klj97506dI6ffq0JOngwYMqV66c/Pz8bNv//q9mAG7PqlWr5OHhIavVqo4dO6p37962W3Nr166dq+ec/v477Ovra9v3722pqam6ePGipL9mrkaOHKkaNWrI29tbHh4e2r9/f7aZq4YNG9q9t1qtevLJJzVv3jxJ0t69e/Xjjz/aFt4BYK9nz576/ffftXLlSnXo0EGbNm1SgwYNFBMTo++//16XL19WiRIlbLNcHh4eSkxMtN1yl5CQoDZt2uQ49v79++Xs7KwmTZrY2kqUKKFq1app//79trYiRYqoUqVKtvd////45ORkJSUl2Y3h7Oyc7Xf/0KFD6tu3rypWrChPT09VqFBBkm56zWjcuLFq1aql2NhYSX/NdPv7+9vutsG9iXCF+8LAgQMVExOj2NhYDRw48IZ9ixYtqr179+qTTz5R6dKlNX78eNWtW1cXLlyQk5OTNmzYoLVr16pmzZqaOXOmqlWrpsTExOuO988HTi0WS7apfgB3RqtWrZSQkKBDhw7pzz//tN2mK8n2vzfz999hi8Vy3bZrv9cjR47U8uXLNXnyZG3dulUJCQmqXbu27XbEa3I6/qBBg7Rhwwb9+uuvio6OVuvWreXv738LZwwULFarVe3atdO4ceO0fft2hYWFKTIyUpcvX1bp0qWVkJBg9zp48KBefPFFSbItdHU7cvr/eOMGjwrkpGvXrvrjjz/0/vvva9euXdq1a5ck5fqacW2V4ujoaIWHh9uuSbg3Ea5wX+jQoYPS09OVkZGhkJCQm/Z3dnZW27ZtNWXKFP3vf//TsWPH9NVXX0n668LZrFkzTZw4Ud99951cXFyy3T+dW9WqVdOJEyd06tQpW9u3336bp7EAZOfu7q7KlSurfPnycna+OwvgxsfHKywsTD169FDt2rXl5+enY8eO5Wrf2rVrq2HDhnr//fe1cOHCm/5jEAB7NWvWVEpKiho0aKCTJ0/K2dlZlStXtntdu3ulTp06110spkaNGsrMzLQFHUk6d+6cDh48qJo1a+aqFi8vL5UuXdpujMzMTO3ZsyfbmGPHjlWbNm1Uo0aNm94R83dPPPGEfvnlF7399tvat2+fQkNDc70vHIOl2HFfcHJysk3jOzk53bDvqlWrdPToUT388MMqVqyY1qxZo6ysLFWrVk27du1SXFyc2rdvr1KlSmnXrl06c+aMatSokae62rVrp0qVKik0NFRTpkzRpUuXNHbsWEniX56AfKpKlSpatmyZunbtKovFonHjxt3SbPWgQYMUEREhd3d3u5XLAPy/c+fOqVevXho4cKDq1KmjokWLavfu3ZoyZYq6deumtm3bKigoSN27d9eUKVNUtWpV/f7777ZFLBo2bKjIyEi1adNGlSpVUp8+fZSZmak1a9bo5ZdfVpUqVdStWzcNHjxYc+fOVdGiRTVq1CiVLVtW3bp1y3Wdw4YN0+uvv64qVaqoevXqmjZtmi5cuGDbXqxYMZUoUULvvfeeSpcurePHj2vUqFG5Hr9YsWJ69NFH9eKLL6p9+/Z64IEHbuXHCAdg5gr3DU9PT3l6et60n7e3t5YtW6bWrVurRo0amjNnjj755BPVqlVLnp6e2rJlizp16qSqVatq7Nixmjp1qjp27JinmpycnLRixQpdvnxZjRo10qBBg2yrBVqt1jyNCcCxpk2bpmLFiumhhx5S165dFRISogYNGuR6/759+8rZ2Vl9+/blOgBch4eHh5o0aaLp06fr4Ycf1oMPPqhx48Zp8ODBeuedd2SxWLRmzRo9/PDDCg8PV9WqVdWnTx/98ssvtmcnW7ZsqU8//VQrV65UvXr11Lp1a33zzTe2Y0RHRyswMFBdunRRUFCQDMPQmjVrbun7pf7973/rySefVGhoqIKCglS0aFG7fzQpVKiQFi1apD179ujBBx/U8OHD9d///veWfhZPPfWU0tPTmenOJyzGrd44CuC2xMfHq3nz5jp8+LDdQ7IACoZjx46pUqVK+vbbb28plAEomObPn6/hw4fr999/58vI8wFuCwTusOXLl8vDw0NVqlTR4cOHNWzYMDVr1oxgBRQwGRkZOnfunMaOHaumTZsSrADc0JUrV5SUlKTXX39dTz/9NMEqn+C2QOAOu3Tpkp599llVr15dYWFhatSokT7//HNHlwXgLouPj1fp0qX17bffas6cOY4uB8A9bsqUKapevbr8/Pw0evRoR5eDXOK2QAAAAAAwATNXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAD/EBMTI29v79sex2KxaMWKFbc9DgAgfyBcAQDuS2FhYerevbujywAAFCCEKwAAAAAwAeEKAFDgTJs2TbVr15a7u7vKlSunf/3rX7p8+XK2fitWrFCVKlVktVoVEhKiEydO2G3//PPP1aBBA1mtVlWsWFETJ05UZmbm3ToNAMA9hnAFAChwChUqpLfffls//fSTYmNj9dVXX+mll16y63PlyhW99tpr+uijjxQfH68LFy6oT58+tu1bt27VgAEDNGzYMO3bt09z585VTEyMXnvttbt9OgCAe4TFMAzD0UUAAGC2sLAwXbhwIVcLSixdulRDhw7V2bNnJf21oEV4eLh27typJk2aSJIOHDigGjVqaNeuXWrcuLHatm2rNm3aaPTo0bZxPv74Y7300kv6/fffJf21oMXy5ct59gsACghnRxcAAMDdtnHjRkVFRenAgQO6ePGiMjMzlZqaqitXrqhIkSKSJGdnZzVq1Mi2T/Xq1eXt7a39+/ercePG+v777xUfH283U3X16tVs4wAACg7CFQCgQDl27Ji6dOmiZ555Rq+99pqKFy+ubdu26amnnlJ6enquQ9Hly5c1ceJEPfroo9m2Wa1Ws8sGAOQDhCsAQIGyZ88eZWVlaerUqSpU6K9Hj5csWZKtX2Zmpnbv3q3GjRtLkg4ePKgLFy6oRo0akqQGDRro4MGDqly58t0rHgBwTyNcAQDuW8nJyUpISLBrK1mypDIyMjRz5kx17dpV8fHxmjNnTrZ9CxcurOeee05vv/22nJ2dFRERoaZNm9rC1vjx49WlSxeVL19ejz32mAoVKqTvv/9eP/74o1599dW7cXoAgHsMqwUCAO5bmzZtUv369e1e8+fP17Rp0/TGG2/owQcf1IIFCxQVFZVt3yJFiujll19Wv3791KxZM3l4eGjx4sW27SEhIVq1apW+/PJLNWrUSE2bNtX06dPl7+9/N08RAHAPYbVAAAAAADABM1cAAAAAYALCFQAAAACYgHAFAAAAACYgXAEAAACACQhXAAAAAGACwhUAAAAAmIBwBQAAAAAmIFwBAAAAgAkIVwAAAABgAsIVAAAAAJiAcAUAAAAAJvg/BTY0S4w48l8AAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    }
  ]
}